icons/llvm/llvm-original.svg

```diff
@@ -0,0 +1 @@
9 0 0 1 .16 1.016c.051.508.07.555.199.516.059-.02.457-.078.887-.129zm-24.086-1.383c.609-.195 1.117-.871 1.117-1.477 0-.254-.309-.895-.535-1.105-.633-.594-1.77-.602-2.348-.016-.293.297-.426.621-.422 1.039.004.625.344 1.164.941 1.48.375.195.793.223 1.246.078zm51.285-.059c.66-.32 1.016-.836 1.02-1.48.004-.73-.383-1.227-1.133-1.445-1.293-.375-2.547.93-2.039 2.125a1.66 1.66 0 0 0 2.152.801zm-14.937-.426c-.117-.141-1.035-.715-1.422-.887-.516-.23-1.516-.586-1.551-.555-.016.02.043.125.129.238a4.5 4.5 0 0 1 .352.594c.105.215.211.391.238.395s.508.07 1.07.148c1.324.184 1.277.18 1.184.066zm-21.492-.199c.16-.078.262-.191.367-.422.195-.418.195-.492-.023-.445-.273.066-1.023.457-1.258.664-.18.152-.203.195-.125.246.152.098.801.07 1.039-.043zm-1.535-.062c.086-.055.238-.18.34-.273.316-.297.91-.617 1.434-.777.816-.25 1.328-.672 2.199-1.832 1.445-1.926 3.426-3.535 4.941-4.016 1.148-.363-.57-.133-2.316.309a19.42 19.42 0 0 0-2.957 1.012c-1.336.555-3.484 1.684-3.703 1.945a7.2 7.2 0 0 1-.312.25c-.773.582-1.391 1.383-1.512 1.965-.086.41.008.852.227 1.074.363.363 1.328.566 1.66.344zM38.836 30.52c.02-.016-.133-.352-.332-.742-.906-1.789-2.516-3.422-4.488-4.559l-.641-.371c-.148-.082-.285-.137-.305-.117s.195.254.48.523a45.3 45.3 0 0 1 2.004 2.039c.617.68 1.5 1.938 1.836 2.617l.336.691.539-.027c.297-.016.555-.039.57-.055zm51.578-.262c.168-.379.727-1.297 1.102-1.805.57-.773 1.238-1.52 2.375-2.648l1.184-1.176-.555.297c-1.141.617-2.375 1.492-2.992 2.125-.941.961-1.727 2.035-2.145 2.918-.312.664-.336.625.32.617l.57-.008zm-27.77-1.004a17.74 17.74 0 0 1 .977-.117c.492-.047.563-.07.82-.281.32-.266 1.32-.828 2.094-1.184a9.39 9.39 0 0 0 .648-.324c.109-.074.105-.082-.059-.125-.527-.129-2.824.328-3.723.746-.449.207-1.191.766-1.598 1.195l-.215.23.309-.039c.172-.02.508-.066.746-.102zm23.293 74.707c.113-.016.309-.016.438 0s.039.023-.203.023-.344-.012-.234-.023zm39.305-56.984c-.039-.078-.242-.574-.453-1.105-.465-1.172-1.426-3.051-2.223-4.355-.633-1.035-.699-1.16-.543-.965.406.496 1.633 2.676 2.32 4.121.391.828 1.027 2.387.992 2.426-.012.012-.055-.043-.094-.121zm-3.449-6.691c-.055-.059-.285-.387-.516-.727-.336-.5-1.695-2.273-1.836-2.398-.02-.02-.289-.328-.602-.687s-.629-.707-.703-.777-.254-.266-.391-.434c-.152-.195-.047-.109.289.234.992 1.012 2.512 2.883 3.484 4.293.398.57.492.742.273.496zm-74.16-3.469c.02-.07.066-.285.105-.48.125-.629.48-.973.84-.809.316.145.145.543-.312.734-.113.047-.285.191-.379.32-.246.336-.293.379-.254.234zm.762-.969c.07-.109-.004-.125-.129-.027-.105.078-.109.09-.016.09.059.004.121-.027.145-.062zm67.762-2.168c-.641-.613-1.383-1.293-1.648-1.504s-.469-.406-.453-.422c.055-.055 3.332 2.91 3.332 3.012 0 .078-.16-.062-1.23-1.086zM66.18 29.398a.5.5 0 0 1 .215 0c.063.016.012.027-.105.027s-.172-.012-.109-.027zm6.039-.5c.059-.016.145-.016.184.004s-.008.027-.109.027-.137-.016-.074-.031zm-57.328-.687a3.13 3.13 0 0 1 .469 0c.129.016.023.027-.234.027s-.363-.012-.234-.027zm1.246-.062a7.27 7.27 0 0 1 .715 0c.195.012.035.023-.359.023s-.555-.012-.355-.023zm0 0" fill="rgb(0%,0%,0%)"/></svg>
```

Comment:
Changed the color codes so they are the same as those in the .png.

#### New version
<img title="new version (new color codes)"  alt="new version (new color codes)" src="https://user-images.githubusercontent.com/43886029/194524137-876c8d7c-a236-485b-a8aa-e9b7c8eb539d.svg" width="500px"></img> 

#### Original PNG Version
<img  title="original PNG version"  alt="original PNG version" src="https://llvm.org/img/LLVMWyvernSmall.png" width="500px"></img> 

#### Old Version
<img  title="old version"  alt="old version" src="https://user-images.githubusercontent.com/43886029/194524577-2addf9fe-aac0-43db-9ad7-45b4bbb76977.svg" width="500px"></img>


```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path d="M82.625 103.785c-1.48-.203-3.551-.723-5.164-1.301a22.87 22.87 0 0 1-1.992-.875c-3.031-1.496-4.984-3.203-8.324-7.281-.609-.746-1.066-1.254-1.141-1.266s-.531.18-1.027.418c-1.711.824-5.727 2.488-7.871 3.262-6.113 2.203-10.492 3.074-14.324 2.848-2.918-.172-5.246-.676-6.93-1.5-1.246-.609-2-1.258-2.605-2.242-.375-.609-.684-1.402-.582-1.5.074-.074.168-.016 1.047.668.754.59 1.383.973 1.969 1.207.91.363 2.332.688 3.906.887.883.109 3.887.109 4.637-.004 2.254-.336 6.117-1.441 8.777-2.512 1.09-.441 4.414-2.039 7.375-3.547 2.387-1.219 3.113-1.617 3.383-1.859.191-.168.191-.176.117-.457-.043-.16-.367-1.023-.727-1.922-.691-1.746-1.523-4.035-2.324-6.391-1.023-3.008-1.309-3.734-1.465-3.734-.172 0-.734.738-2.133 2.797-1.051 1.543-1.227 1.746-1.543 1.746-.324 0-.758-.184-1.355-.574-.859-.559-1.625-1.234-3.234-2.859-.973-.98-1.578-1.547-1.656-1.547-.359 0-.582.699-.625 1.945-.027.836.008 1.199.145 1.563.023.063.008.102-.039.102-.141 0-.852-.5-1.109-.777-.324-.352-.73-1.215-.91-1.934-.078-.316-.16-.609-.184-.645-.086-.141-.281-.055-.551.246-.52.574-.852 1.66-.852 2.766 0 .688-.047.73-.41.367-.508-.508-.965-1.457-1.09-2.27-.191-1.215-.27-1.309-.687-.855-.387.418-.789 1.285-.902 1.949-.051.309-.117.586-.148.617-.062.066-.402-.391-.594-.801-.5-1.074-.445-2.586.141-3.727.496-.973.57-1.148.699-1.625.32-1.18.895-1.785 2.117-2.23.293-.105.547-.129 1.645-.152 1.992-.047 3.23.109 4.012.508.363.184 1.055.898 2.371 2.453 1.402 1.66 1.738 2.012 1.898 1.988.152-.02.621-.777.914-1.48.168-.395.207-.59.25-1.246.031-.426.098-.93.152-1.117.266-.914.906-1.676 1.914-2.27l.602-.355-.027-.227a18.65 18.65 0 0 1-.066-.75c-.051-.75-.141-1.109-.309-1.242-.109-.086-.348-.133-.973-.187a19.47 19.47 0 0 1-3.414-.676c-1.93-.582-5.559-2.359-10.211-5-3.344-1.898-4.945-2.59-6.902-2.984-1.305-.266-2.648-.328-3.559-.164-.922.164-2.328.836-2.895 1.383-.359.348-.445.539-.57 1.266-.109.652-.203.813-.203.359 0-.145-.027-.445-.062-.668-.055-.363-.09-.43-.324-.645-.402-.367-1.285-.766-1.941-.879-.719-.125-2.703-.129-3.273-.008-1.656.355-4.082 2.102-5.82 4.195-.582.695-.641.793-.68 1.09-.117.902-.242 1.598-.293 1.598-.027 0-.055-.387-.055-.859 0-1 .055-.902-.809-1.461-2.102-1.355-4.074-1.59-5.789-.691-2.684 1.406-4.75 5.344-6.223 11.855-.633 2.805-.652 4.008-.09 6.465l.246 1.047c.016.043.004.078-.02.078-.105 0-.754-1.965-1.293-3.922C1.66 74.691.719 69.727.348 66.164c-.25-2.371-.285-7.445-.07-9.336.352-3.094.676-4.531 1.719-7.672 1.184-3.574 2.297-5.937 4.027-8.574 2.074-3.164 4.801-6.199 7.898-8.789a61.3 61.3 0 0 1 1.988-1.547c.723-.543 1.348-1.031 1.387-1.082s.07-.156.059-.234c-.02-.125-.07-.148-.426-.195-.422-.051-3.945-.008-4.996.063-.402.027-.605.02-.605-.023 0-.082.145-.109 1.398-.266 1.656-.203 2.512-.254 4.918-.293 3.125-.047 5.746.094 7.875.426 1.328.207 1.742.215 2.148.051.281-.113.402-.129.699-.094.387.051 3.801.895 6.238 1.551 1.813.484 2.707.688 2.836.637.219-.082.051-.543-.512-1.426-.648-1.016-1.199-1.652-2.941-3.418-.699-.711-1.25-1.312-1.223-1.34.059-.059 1.121.5 1.84.965 1.574 1.016 2.914 2.395 3.707 3.813.652 1.16.797 1.391 1.184 1.848.602.727.734 1.289.441 1.918-.145.309-.254.422-.895.906-.281.211-.371.34-.637.883-.777 1.609-2.16 6.383-3.047 10.512-.176.809-.316 1.52-.316 1.578s.113.234.258.395c.332.375.785 1.309.859 1.773.035.203.09.402.129.449s.215.113.391.152c.512.109 7.863 1.309 8.035 1.309.125 0 .195-.074.379-.398.121-.219.262-.434.309-.473s1.047-.191 2.223-.344c2.867-.371 5.738-.816 9.266-1.437 1.664-.297 1.988-.336 2.16-.277.16.059.234.055.402-.031.238-.121 3.094-2.977 4.594-4.594 1.215-1.312 1.625-1.84 2.043-2.645.625-1.211.848-2.105.805-3.273-.031-.937-.172-1.348-.609-1.785-.418-.422-.598-.461-2.082-.457-1.391.004-2.047.082-3.531.43-1.285.301-1.406.363-1.762.898-.344.527-.59.746-1.285 1.152-.812.477-1.316.883-2.473 1.977-1.277 1.211-1.875 1.715-2.227 1.867-.25.113-.266.113-.652-.008-.457-.145-.801-.336-.801-.453 0-.105.258-.344.996-.93 1.027-.816 1.875-1.746 2.336-2.57.34-.613.465-1.07.465-1.715 0-.52-.012-.574-.16-.723-.141-.141-.187-.152-.445-.117-.387.059-1.281.387-1.477.543-.215.168-.199.316.066.609.121.137.215.273.203.305s-.219.004-.457-.059c-.602-.16-1.09-.156-1.301.008-.09.07-.164.184-.164.246s.141.301.313.523.313.426.313.445c0 .086-.582-.039-1.035-.223-.715-.285-1.02-.242-1.02.141 0 .238.086.398.387.707.156.164.223.273.176.301-.09.059-.66-.113-1.34-.402-.309-.133-.754-.273-.988-.309s-.512-.113-.613-.164c-.32-.164-.711-.609-.93-1.059-.187-.387-.207-.48-.207-.977 0-.523.008-.559.168-.695.098-.086.32-.176.527-.215 1.461-.273 2.098-.543 2.723-1.145.227-.219.418-.461.445-.562s.008-.371-.039-.602c-.109-.582-.051-1.094.168-1.504.922-1.727 5.48-4.035 9.613-4.871.773-.156 1.609-.211 1.609-.105 0 .031-.227.203-.508.387-1.492.969-3.535 3.113-3.535 3.711 0 .137.035.223.102.25.219.082 2.469-.461 3.109-.754.18-.082.543-.359.84-.637.809-.762 1.371-1.105 2.328-1.414 1.219-.395 3.473-.633 3.297-.348-.02.031-.406.238-.859.453-1.352.645-2.098 1.113-2.098 1.32 0 .156.191.211 1.063.301 1.969.203 2 .203 2.891-.09.855-.277 1.469-.367 2.559-.367.816-.004 1.77.066 1.77.125 0 .051-.434.223-1.543.609-.879.309-1.023.398-1 .621.016.129.172.277.707.68a8.24 8.24 0 0 0 2.195 1.223c1.129.434 2.469 1.215 2.328 1.355-.039.043-.316.031-.832-.039a8.54 8.54 0 0 0-1.141-.086l-.375.02-.02.262c-.008.145.039.492.105.777s.141.66.164.828.055.395.074.496c.023.125.352.504 1.086 1.258.914.941 1.172 1.266 1.008 1.266-.023 0-.387-.109-.805-.246-.836-.273-.969-.293-1.078-.164-.082.102-.211.801-.32 1.762l-.078.695.344.578c.395.668.609 1.078.609 1.172 0 .117-.121.066-.547-.219-.461-.309-.539-.328-.625-.168-.082.148-1.094 3.766-1.32 4.711-.219.918-.227 1.066-.059 1.152.426.23 6.484 1.313 10.082 1.805l2.086.289c.832.113 1.098.227 1.164.504.027.109.129.258.223.332.152.125.219.133.73.102.566-.031 4.191-.594 4.961-.766.68-.152.777-.254 1.047-1.062.266-.797.379-1 .773-1.387.152-.152.281-.316.281-.363 0-.164-.93-4.187-1.305-5.645-.434-1.68-.93-3.375-1.359-4.637-.687-2.02-.871-2.332-1.805-3.102-.414-.34-.594-.68-.598-1.133-.008-.477.168-.883.578-1.336.195-.215.531-.711.754-1.117 1.188-2.141 2.359-3.422 4.195-4.578.742-.465 1.828-1.031 1.883-.977.02.02-.281.34-.668.707-.914.871-1.789 1.777-2.375 2.465-.953 1.113-1.805 2.465-1.773 2.816.027.344.211.316 3.293-.504 2.484-.66 5.074-1.301 5.664-1.402.426-.07.477-.066.762.059.383.172.797.172 1.832.012 2.117-.328 4.672-.5 7.488-.5 2.637 0 5.074.16 6.68.441.645.109.508.207-.234.16-1.207-.074-4.309-.121-4.855-.074-.395.031-.559.07-.586.137-.055.145.348.516 1.57 1.441 4.609 3.5 7.984 7.176 10.637 11.582 1.563 2.598 2.676 5.32 3.863 9.434.547 1.895.82 3.367 1.07 5.758.137 1.309.16 6.234.035 7.688-.359 4.141-.984 7.84-2.043 12.074-.684 2.734-1.691 5.914-1.703 5.375 0-.059.195-.961.438-2.008l.438-1.902-.223-1.262c-.547-3.086-1.375-6.18-2.254-8.418-1.215-3.082-3.094-5.434-4.902-6.137-.637-.246-1.184-.324-2.02-.289-.844.035-1.324.176-2.402.699-.828.402-1.418.777-1.559.992-.082.125-.113.359-.141 1-.016.465-.051.84-.078.84s-.066-.145-.09-.324c-.098-.777-.297-1.59-.449-1.852-.34-.582-2.469-2.629-3.512-3.375-.668-.477-2.113-1.195-2.664-1.32-.523-.121-2.855-.141-3.379-.031-.418.09-1.484.555-1.773.777s-.434.57-.484 1.191c-.023.285-.066.516-.098.516s-.07-.133-.094-.297c-.117-.863-.215-1.125-.535-1.477-.617-.676-1.613-1.191-2.789-1.441-.734-.156-2.539-.156-3.332 0-2.109.414-4.043 1.223-7.094 2.965-2.074 1.184-4.062 2.266-5.418 2.945-1.664.828-3.812 1.801-4.496 2.027a17.58 17.58 0 0 1-4.211.84c-.949.078-1.066.098-1.164.219-.137.168-.203.66-.207 1.5l-.004.652.594.047c1.215.09 2.051.395 2.676.965.668.613.984 1.391 1.113 2.711.078.813.223 1.27.641 2.035.289.535.461.75.609.75.105 0 .676-.625 1.988-2.18 1.375-1.625 1.832-2.094 2.262-2.297.781-.367 1.766-.504 3.586-.504 1.781 0 2.527.199 3.258.867.438.402.691.848.887 1.559.102.371.277.789.469 1.121.477.832.633 1.379.668 2.305.023.598.008.867-.07 1.18-.129.496-.348.984-.574 1.281-.16.211-.18.219-.254.113-.043-.059-.082-.187-.082-.277 0-.262-.238-.98-.477-1.437-.555-1.059-1.078-1.242-1.078-.379 0 .754-.453 1.871-1.066 2.613-.492.602-.523.57-.574-.605-.043-1.059-.355-1.93-.898-2.504-.254-.273-.574-.203-.574.125 0 .305-.199.934-.496 1.539-.227.469-.383.699-.633.938-.344.328-.902.691-.965.629-.02-.02.004-.184.051-.367.129-.508.098-1.84-.059-2.398s-.281-.762-.473-.762c-.176 0-.836.598-2.426 2.191-1.309 1.309-2.242 2.078-2.945 2.426-.633.316-.996.367-1.258.18-.102-.07-.523-.633-.937-1.242-.676-1-1.797-2.543-2.195-3.02-.176-.211-.359-.187-.359.039 0 .242.281 1.891.684 4 .645 3.371.961 4.754 1.098 4.805.047.02.441-.121.879-.309 2.055-.883 3.941-1.418 6.52-1.848 1.555-.258 2.469-.324 4.449-.324 2.43 0 3.746.133 5.387.547 4.281 1.082 7.035 3.156 8.285 6.242.684 1.695.84 3.219.516 5.145-.137.832-.336 1.402-.785 2.277-1.621 3.141-5.523 5.785-9.387 6.367-.812.125-4.008.121-4.918 0zm3.641-6.66c1.176-.254 2.336-.918 3.301-1.891 1.063-1.07 1.461-1.824 1.422-2.711-.027-.598-.203-.996-.723-1.621-1.027-1.25-2.824-2.117-5.273-2.547-1.379-.242-4.699-.242-6.801 0-1.887.219-4.621 1.234-4.621 1.715 0 .16 1.129 1.641 1.922 2.523 1.047 1.164 2.047 2.035 3.293 2.867 1.305.871 3.012 1.613 4.047 1.758.641.09 2.879.027 3.434-.094zm0 0" fill="#fefefe"/><path d="M82.781 103.938c-.613-.07-1.898-.328-2.832-.559-1.746-.434-3.465-1.07-4.852-1.793-2.949-1.535-4.895-3.289-8.289-7.469l-.797-.98-.937.449c-2.023.973-6.32 2.73-8.723 3.566-5.309 1.848-8.828 2.578-12.41 2.574-4.352-.004-8.031-.977-9.824-2.602-.828-.746-1.727-2.359-1.598-2.867.02-.082.074-.148.121-.148s.316.176.602.391l.984.746c.934.703 2.207 1.199 3.887 1.504 1.453.266 2.098.316 3.93.313 2 0 2.359-.047 4.566-.59 3.34-.816 5.887-1.715 8.691-3.062 2.883-1.383 8.305-4.129 8.465-4.285.094-.09-.043-.508-.773-2.359-.77-1.949-1.258-3.293-2.23-6.133-.969-2.816-1.34-3.828-1.414-3.824-.086 0-.82.996-1.949 2.648-.516.758-1.055 1.492-1.191 1.633-.352.363-.602.367-1.34.02-.973-.461-2.109-1.422-4.113-3.477-.785-.805-1.281-1.27-1.336-1.25-.277.09-.496 1.008-.496 2.074 0 .617.023.793.16 1.191.09.258.148.48.133.496-.086.086-1.164-.605-1.484-.953-.371-.395-.77-1.258-.984-2.113-.066-.262-.145-.504-.18-.535-.066-.07-.48.453-.691.879-.27.531-.437 1.387-.437 2.191 0 .902-.023.918-.621.32-.684-.687-1-1.418-1.238-2.871-.039-.219-.082-.41-.102-.43-.121-.125-.77.883-.992 1.551a4.95 4.95 0 0 0-.199 1.023l-.043.551-.332-.367c-.672-.734-.93-1.559-.875-2.766.039-.879.168-1.32.672-2.309a7.46 7.46 0 0 0 .496-1.203c.066-.27.191-.621.277-.785.336-.664 1.078-1.246 1.93-1.516.41-.129.543-.137 2.238-.133 1.996.004 2.605.082 3.387.434.461.207 1.016.766 2.473 2.488 1.367 1.613 1.723 2.012 1.809 2.016.094.008.449-.578.777-1.277.238-.504.258-.598.313-1.324.031-.434.098-.949.148-1.152.246-.984.98-1.824 2.156-2.465l.328-.18-.02-.453c-.02-.527-.109-1.18-.203-1.461-.074-.234-.121-.25-1.035-.328-.898-.078-1.66-.211-2.691-.473-1.078-.27-1.281-.344-2.742-.969-2.133-.914-4.68-2.227-8.027-4.133-1.922-1.094-4.023-2.152-4.934-2.488-.691-.254-1.742-.535-2.602-.695-.434-.082-.91-.113-1.801-.113H32.77l-.684.234c-.789.273-1.504.637-1.984 1.016s-.625.707-.789 1.859c-.199 1.41-.242 1.395-.371-.156-.09-1.059-.109-1.105-.582-1.418-.742-.488-1.316-.668-2.426-.758-1.051-.086-2.246.004-2.875.211-1.668.559-3.719 2.109-5.316 4.027l-.602.719-.137.918c-.336 2.258-.508 2.367-.512.324v-1.066l-.574-.387c-.703-.469-1.719-.961-2.348-1.137-.684-.191-1.59-.23-2.242-.098-1.977.398-3.68 2.109-5.137 5.16-.785 1.645-1.352 3.324-2.016 5.984-.59 2.352-.781 3.625-.734 4.855.035.973.141 1.555.641 3.641.184.77.336 1.453.336 1.523.004.566-1.031-2.215-1.734-4.668C1.613 75.164.617 69.988.219 66.133c-.207-2.02-.277-5.773-.156-8.031.109-1.961.477-4.41.91-6.066.391-1.504 1.555-4.941 2.145-6.34.371-.875 1.402-2.914 1.895-3.746 2.098-3.535 4.914-6.859 8.309-9.809.781-.676 2.266-1.852 2.957-2.336.633-.449 1.07-.836 1.008-.898-.137-.141-4.707-.074-6.695.102-1.465.125-1.535.129-1.441.07.129-.086 1.383-.359 2.445-.539 3.648-.613 10.008-.625 13.93-.027 1.406.215 1.641.223 2.121.055.398-.141.449-.145.898-.066.527.09 3.48.824 5.969 1.488 1.879.5 2.828.723 2.875.676.16-.16-.93-1.906-1.82-2.906-.227-.254-1.07-1.129-1.875-1.937s-1.445-1.504-1.422-1.539c.051-.086.18-.035 1.156.461.914.465 1.793 1.039 2.492 1.621 1.105.922 2.199 2.262 2.77 3.398.297.59.582 1.023.922 1.398.594.664.758 1.395.453 2.047-.148.316-.359.531-.867.883-.359.246-.531.52-.887 1.387-.82 1.996-2.117 6.633-2.891 10.309l-.266 1.254.297.363c.391.484.77 1.273.836 1.766.031.215.07.402.086.422.043.039 2.441.449 5.75.984 2.992.484 2.648.5 2.973-.137.094-.18.211-.34.27-.363s.973-.152 2.035-.285c2.789-.355 6.313-.902 9.824-1.531 1.445-.258 1.633-.277 1.848-.207l.238.078.363-.312c.676-.594 3.68-3.672 4.898-5.023.988-1.094 1.629-2.191 2.004-3.426.156-.504.172-.656.176-1.523.004-.914-.004-.984-.156-1.312-.215-.457-.5-.766-.82-.902-.391-.16-1.883-.195-2.988-.066-1.223.145-3.211.605-3.457.805-.047.035-.203.254-.355.484-.305.477-.66.781-1.41 1.211-.738.422-1.422.973-2.469 1.988-1.98 1.922-2.207 2.047-3.07 1.719-.445-.172-.676-.348-.676-.527 0-.164.188-.348 1.027-1.004 1.402-1.102 2.348-2.301 2.672-3.387.133-.453.137-1.328 0-1.461s-.254-.125-.91.086c-.617.199-.98.387-.98.508 0 .043.188.266.422.5.32.316.395.426.305.426-.066 0-.488-.086-.934-.191-.797-.187-.816-.187-1.051-.086-.129.059-.238.141-.238.176 0 .141.273.512.641.871l.379.371-.43-.043c-.473-.043-.832-.141-1.32-.352-.672-.289-.906-.016-.48.563.125.168.246.309.266.313s.156.102.293.223l.25.219-.406-.039c-.473-.047-1.121-.254-1.77-.562-.285-.133-.555-.219-.703-.219-1.004 0-1.945-1.152-1.945-2.371 0-.672.254-.906 1.137-1.047.977-.156 1.855-.551 2.391-1.074.395-.387.453-.543.348-.937-.133-.492-.078-1.199.125-1.609.793-1.609 4.586-3.695 8.633-4.75a20.11 20.11 0 0 1 1.395-.312c.574-.102 1.75-.133 1.805-.047.016.023-.277.234-.648.469-1.059.656-1.98 1.438-2.773 2.348-.879 1.008-1.168 1.555-.816 1.555.176 0 1.52-.324 2.223-.539.742-.223.953-.348 1.496-.867.871-.824 1.754-1.301 2.965-1.59.645-.152 1.727-.301 2.211-.301.355-.004.984.113.98.18 0 .02-.637.348-1.414.73-1.262.621-1.977 1.063-1.879 1.164.055.059 1.883.23 2.566.242.598.012.676 0 1.289-.223.594-.219.746-.246 1.625-.305.953-.066 2.926-.004 3.023.098s-.199.25-1.168.594c-1.492.527-1.684.617-1.613.754.066.125 1.262 1.023 1.68 1.262.156.09.617.293 1.027.449.906.352 1.727.766 2.215 1.129.344.25.609.551.547.613-.016.016-.461-.023-.992-.09l-1.285-.148-.328-.035v.211c0 .113.07.531.156.93s.156.813.156.922c0 .281.078.383 1.09 1.418.82.836 1.273 1.375 1.273 1.504 0 .102-.258.047-1.09-.23-.465-.152-.895-.277-.957-.277-.137 0-.195.254-.352 1.523l-.105.871.359.59c.586.965.793 1.5.574 1.492-.043 0-.316-.152-.605-.34s-.543-.316-.566-.293c-.102.117-1.488 5.297-1.488 5.563 0 .145-.008.145.809.324 2.203.492 7.508 1.379 10.457 1.754l1.566.223c.348.074.613.266.613.438 0 .066.07.195.16.281.156.156.18.16.699.125a68.84 68.84 0 0 0 4.957-.789c.273-.07.543-.176.602-.234s.18-.344.273-.637c.25-.77.48-1.195.816-1.488l.293-.258-.328-1.445c-1.031-4.57-1.926-7.797-2.715-9.824-.449-1.164-.656-1.5-1.09-1.797-.504-.344-.785-.684-.902-1.094-.184-.633-.012-1.207.547-1.816.188-.199.5-.66.699-1.016.914-1.637 1.375-2.273 2.313-3.172 1.102-1.055 2.035-1.676 3.789-2.523.582-.285 1.066-.504 1.078-.492s-.082.113-.215.227c-2.086 1.801-3.648 3.453-4.625 4.879-.457.664-.812 1.316-.812 1.484 0 .066.055.098.176.098.191 0 1.461-.312 4.023-.992 2.234-.598 4.402-1.117 4.824-1.16a1.09 1.09 0 0 1 .645.094c.375.164.73.164 1.758.008 1.176-.18 3.039-.359 4.621-.441 1.566-.086 5.691-.047 6.887.063 1.973.18 4.605.617 5.098.848.215.102.184.098-1.34-.031-1.957-.172-6.492-.246-6.629-.109-.062.066.203.297 1.441 1.238 5.531 4.215 9.434 8.773 12.012 14.023.867 1.762 1.641 3.832 2.457 6.566.992 3.316 1.371 6.141 1.371 10.27 0 3.195-.266 6.242-.844 9.68-.793 4.691-2.238 10.184-3.238 12.285-.187.391-.348.699-.363.688s.254-1.23.594-2.699l.617-2.672-.223-1.25c-.691-3.871-1.797-7.617-2.906-9.832-.844-1.687-1.785-2.934-2.871-3.809a4.78 4.78 0 0 0-1.902-.965c-.691-.176-1.801-.145-2.465.066-.973.313-2.609 1.199-2.773 1.508-.047.09-.094.59-.121 1.344-.023.66-.062 1.219-.086 1.242-.074.074-.152-.121-.223-.574-.395-2.461-.32-2.301-1.574-3.57-1.367-1.391-2.133-2.02-3.172-2.617-1.469-.844-1.852-.945-3.594-.945-1.531-.004-1.707.031-2.66.473-.961.453-1.016.531-1.133 1.715-.066.684-.105.867-.195.867-.07 0-.105-.133-.215-.871-.148-1.016-.223-1.18-.723-1.645-1.012-.937-2.602-1.402-4.504-1.324-2.211.094-4.625.965-8.031 2.91-3.617 2.063-5.652 3.133-8.09 4.25-.855.391-1.879.816-2.273.941-1.375.445-2.926.738-4.305.816-.402.02-.773.07-.828.113-.145.102-.215.457-.258 1.289l-.043.723.637.043c1.258.086 2.063.406 2.742 1.094.66.664.922 1.32 1.055 2.676.086.883.281 1.426.863 2.395l.223.367.309-.336c.168-.187.828-.953 1.465-1.703 1.621-1.91 2.121-2.406 2.594-2.586.887-.336 1.418-.398 3.328-.402 1.551-.004 1.82.012 2.145.109.621.191 1.098.473 1.496.875.418.426.641.832.832 1.547.074.266.301.797.504 1.184.465.879.625 1.492.625 2.418 0 1.094-.246 1.828-.863 2.566l-.289.348-.043-.41c-.098-.977-.477-1.934-.973-2.465-.293-.312-.344-.246-.41.523-.078.859-.508 1.801-1.141 2.484-.219.238-.453.434-.516.434-.133 0-.164-.172-.207-1.184-.043-.965-.363-1.867-.867-2.41-.203-.223-.355-.187-.355.082 0 .285-.316 1.242-.578 1.734-.27.52-.902 1.133-1.461 1.426-.289.148-.449.203-.449.152s.059-.266.129-.488c.184-.582.219-1.395.09-2.082-.105-.566-.316-1.152-.41-1.152-.082 0-.758.633-2.301 2.148-1.555 1.527-1.863 1.801-2.57 2.258-.84.547-1.43.695-1.824.453-.117-.07-.477-.535-.93-1.195-1.398-2.059-2.348-3.316-2.379-3.16-.066.34 1.523 8.523 1.688 8.688.012.012.441-.16.949-.379 1.957-.836 3.879-1.367 6.566-1.809 1.488-.246 2.527-.301 4.902-.266 2.348.039 2.988.102 4.43.43 5.031 1.148 8.141 3.742 9.152 7.637a9.58 9.58 0 0 1-.172 5.176c-.66 1.934-2.391 3.992-4.562 5.434-1.324.879-3.258 1.695-4.824 2.039-.562.125-.824.137-2.77.152-1.18.012-2.355-.008-2.613-.039zm.742-.145c-.016-.016-.301-.113-.633-.219-.59-.191-.609-.195-.809-.086-.23.121-.254.113.574.23.637.094.91.117.867.074zm-2.113-.52l-.004-.23-.887-.43-.887-.426-.465.203c-.254.113-.504.242-.551.285-.078.066-.062.09.082.145a1.02 1.02 0 0 0 .246.066c.043 0 .102.023.137.055.066.063 1.988.543 2.254.563.047.004.074-.078.074-.23zm-3.062-1.066a2.74 2.74 0 0 0 .176-.492c.035-.172.016-.199-.262-.359-.168-.098-.547-.336-.84-.531l-.535-.355-.523.25c-.285.141-.547.297-.574.348-.047.074-.152.082-.594.047-.348-.027-.516-.023-.473.016.133.133 1.758.918 2.355 1.141.348.129.742.238.879.238.238.004.258-.008.391-.301zm11.031.09c1.059-.129 1.934-.328 2.457-.562.105-.047.336-.141.512-.211.617-.242 1.359-.645 1.855-1.012.637-.465.875-.664.836-.699-.016-.016-.258.035-.535.117-2.695.781-3.922 1.016-5.598 1.07-1.258.039-2.105-.027-3.309-.266-.906-.18-3.004-.758-3.75-1.035-.566-.207-1.766-.805-2.434-1.211a16.47 16.47 0 0 1-2.09-1.602c-.223-.211-.43-.391-.461-.402-.109-.039-1.344-1.434-1.801-2.031-1.594-2.09-2.727-4.184-3.844-7.098-.492-1.293-1.242-4.125-1.539-5.82-.68-3.895-.965-9.289-.652-12.355.137-1.359.238-2.035.465-3.172.082-.41.184-.973.223-1.246.082-.609.328-1.664.457-1.996.148-.375.492-1.77.84-3.418l.375-1.68c.074-.227.254-1.301.473-2.789l.16-1.109-.285-.426c-1.02-1.531-1.289-3.484-.719-5.168a16.24 16.24 0 0 0 .25-.809 3.31 3.31 0 0 1 .125-.406c.059-.125.219-.687.84-2.957.418-1.512.523-2.434.527-4.48.004-1.836-.141-2.973-.531-4.219-.543-1.727-1.246-2.754-2.449-3.574-.777-.531-1.52-.816-2.742-1.055-.652-.129-.902-.141-2.77-.145-1.523 0-2.152.02-2.43.082-2.191.5-3.023.762-4.383 1.375-1.168.523-1.508.891-1.41 1.508.039.234.023.262-.242.508-.281.262-.625.852-.535.914.074.047.191.793.191 1.207 0 .504-.246 1.195-.703 1.961-.488.816-1.016 1.383-2.078 2.23l-.977.777c-.113.094-.117.117-.039.172.262.172 1.063-.059 1.605-.465.57-.43 1.102-.973 2.578-2.652.078-.09.316-.277.527-.422.52-.344 1.219-1.059 1.398-1.422.125-.254.145-.395.145-.922 0-.598-.008-.641-.238-1.105-.133-.266-.227-.5-.207-.52s.324-.059.676-.086.758-.094.898-.152c.609-.246 2.289-.727 3.168-.906a14.46 14.46 0 0 1 3.059-.352c1.297-.039 2.203.035 2.387.195.051.043.207.113.344.156a5.59 5.59 0 0 1 2.223 1.523c.359.449.879 1.426.895 1.684.004.102.074.48.152.84a6.33 6.33 0 0 1 .141 1.492c0 .82-.133 1.785-.258 1.863-.074.051-.152.473-.473 2.578-.18 1.176-.305 1.828-.367 1.898-.051.059-.145.383-.207.727-.137.75-.16 3.121-.035 3.727.043.211.051.398.023.438s-.02.355.027.723c.16 1.309.223 2.738.203 4.523-.02 1.621-.035 1.828-.129 1.898-.082.063-.121.254-.172.828a24.36 24.36 0 0 1-.648 3.758 27.19 27.19 0 0 0-.348 1.625c-.105.633-.156.801-.27.887-.074.063-.168.137-.203.164-.078.063-.281 1.055-.434 2.094-.055.375-.125 1.34-.156 2.141-.031.848-.082 1.484-.117 1.523-.102.098-.074.719.113 2.746.152 1.633.164 1.887.086 2.004s-.07.266.043 1.164a85.14 85.14 0 0 0 .352 2.355c.344 2.063.359 2.176.285 2.297-.113.18.195 2.102.555 3.422.188.703.297 1.219.273 1.309-.043.172.016.34 1.031 2.961.133.348.246.727.246.84s.035.266.082.332.129.234.184.371a20.98 20.98 0 0 0 1.117 2.336c.258.461.5.945.535 1.074.129.465 1.301 2.129 2.152 3.051.984 1.063 2.855 2.941 3.242 3.254.207.164.441.371.527.457.266.262 1.734 1.191 2.328 1.469a15.55 15.55 0 0 0 1.254.496c.375.125.727.254.785.281.273.145 2.461.5 3.539.578.742.055 2.156.031 2.801-.047zm-13.699-2.012c.199-.492.23-.637.145-.637-.027 0-.219-.148-.422-.332l-.852-.75-.48-.414-.566.32-.57.32-.879-.129c-.484-.07-.887-.117-.898-.105-.059.055 1.563 1.336 2.289 1.813.492.32.605.348 1.719.387l.32.012zm-28.719-1.562c1.816-.258 3.574-.699 5.109-1.289 2.289-.871 5.852-3.082 8.242-5.113 1.277-1.086 2.508-2.199 2.469-2.234-.031-.031-1.598.719-3.34 1.598l-3.766 1.926c-.035.031-.523.254-1.09.496l-1.309.578c-.355.164-3.066 1.117-3.887 1.363-1.852.559-4.266 1.113-5.512 1.273-1.781.223-5.191-.016-6.75-.477-2.098-.621-2.59-.863-3.602-1.766l-.676-.586-.184-.148.102.238c.133.324.344.598 1.09 1.418.391.43 1.375 1.152 1.902 1.391.762.352 3.781 1.105 5.285 1.32 1.234.176 1.242.176 3.207.16 1.418-.012 1.961-.043 2.707-.148zm26.168-.785c.18-.258.301-.469.273-.469-.086 0-1.23-1.379-1.508-1.82-.145-.23-.285-.418-.309-.418-.086 0-.848.266-.918.32-.059.043-2.301-.508-2.723-.668-.059-.023-.102-.016-.102.016 0 .117 1.438 1.813 2.305 2.719l.41.43 1.086.172 1.125.18c.02.004.184-.203.359-.461zm12.605-.848c1.574-.203 3.172-1.113 4.203-2.395.758-.945 1.078-1.754.941-2.395-.102-.492-.461-1.066-1-1.617-.555-.562-1.055-.91-1.832-1.277-1.957-.918-4.273-1.25-7.598-1.09-2.488.121-3.078.215-4.641.73-1.02.336-1.957.77-2.074.961s1.656 2.391 2.859 3.543c1.781 1.703 3.953 2.992 5.816 3.449.855.211 2.109.246 3.324.09zm8.18-2.793c.746-1.121 1.078-2.062 1.184-3.379.059-.727.004-1.715-.121-2.172-.098-.371-.5-1.445-.555-1.484-.023-.016-.145-.227-.27-.465-.504-.961-.898-1.281-2.379-1.945l-.934-.414c-.82-.367-2.781-.805-4.355-.969-.375-.039-.82-.098-.984-.129-.391-.078-3.18-.078-3.57 0-.164.031-.68.09-1.141.129s-.953.094-1.09.117l-.246.047.215.039c.121.02.57.07.996.109 2.801.246 5.266.609 6.383.934 2.371.695 3.715 1.402 4.914 2.594.824.816 1.574 2.094 1.941 3.297.238.793.238 1.789 0 2.66-.168.613-.594 1.594-.855 1.98-.109.16-.07.137.188-.109a6.3 6.3 0 0 0 .68-.84zm-23.125.453l.293-.43-.629-1.305c-.348-.715-.715-1.527-.816-1.801s-.203-.52-.227-.543c-.043-.043-.711.172-1.277.41-.238.102-.254.098-1.371-.215l-1.234-.344c-.098-.027-.102-.016-.023.176.164.387.84 1.762 1.039 2.102l.527.938.332.598 1.074.289c.594.16 1.273.359 1.516.441s.449.141.469.129.164-.211.328-.445zm-5.719-4.059c-.094-.211-.184-.367-.203-.348s.047.199.145.406c.23.48.281.434.059-.059zm3.25-.363l.66-.555.172-.133-.215-.582a33.72 33.72 0 0 1-.746-2.309c-.059-.199-.117-.359-.137-.359-.156 0-1.398.285-1.527.352-.152.074-.203.066-.531-.094-.199-.098-.613-.281-.922-.41s-.77-.328-1.02-.445-.477-.199-.492-.184c-.066.066.855 2.891 1.281 3.93l.176.426.836.238c.461.129.879.254.93.277.188.082.781.254.914.262.082.004.32-.156.621-.414zm-4.332-1.961a52.96 52.96 0 0 1-.375-1.23c-.145-.496-.285-.957-.316-1.023a5.98 5.98 0 0 1-.129-.406c-.043-.152-.113-.324-.16-.383s-.258-.754-.465-1.555c-.672-2.594-.715-2.746-.812-2.82-.094-.066-.176-.406-.602-2.461a27.08 27.08 0 0 0-.215-.965c-.051-.207-.184-.805-.289-1.328-.129-.629-.23-.988-.305-1.062-.129-.129-.211-.715-.355-2.59-.211-2.691-.219-2.738-.305-2.805-.047-.035-.102-.133-.121-.219-.055-.254-.152-2.039-.227-4.133-.051-1.453-.094-2.02-.148-2.086-.137-.16-.152-.285-.285-2.211-.145-2.137-.148-3.777-.012-4.66.074-.496.078-.633.012-.754a.7.7 0 0 1-.039-.395c.066-.41.449-1.68.742-2.473l.777-2.082c.27-.738.543-1.449.605-1.578.094-.191.098-.258.035-.375-.066-.129-.023-.246.387-1.07a23.22 23.22 0 0 1 .93-1.676l.543-.902c.043-.086-.715.656-1.684 1.645-1.332 1.363-1.762 1.836-1.746 1.93.063.375.078 2.203.02 2.398a36.93 36.93 0 0 0-.168.625c-.219.84-.605 1.508-1.352 2.332l-.281.313-.047 1.711c-.047 1.813-.008 4.391.094 5.633l.125 1.836c.086 1.348.289 3.766.402 4.762.293 2.57 1.02 6.883 1.219 7.242.031.059.227.707.43 1.441.375 1.352 1.508 4.848 1.906 5.883l.645 1.742c.234.652.672 1.77.969 2.488l.598 1.449c.109.273.113.133 0-.187zm3.109-2.453l.742-.496c.016-.012-.023-.184-.086-.379-.117-.375-.465-1.777-.598-2.414-.043-.203-.098-.391-.117-.418s-.16-.016-.316.016l-.621.129c-.187.039-.406.102-.488.137-.16.078-.125.09-1.223-.406a38.25 38.25 0 0 0-1.34-.574c-.324-.133-.625-.266-.668-.301-.055-.043-.078-.027-.078.043 0 .141.547 2.34.621 2.496.031.07.129.387.215.707s.164.59.172.594c.027.016.582.258 1.09.473.25.105.766.344 1.152.527s.73.34.766.344.383-.215.777-.477zm-1.238-4.109c.219-.16.48-.371.582-.465s.238-.191.297-.211c.133-.043.133-.016.016-.867-.133-.941-.34-2.879-.344-3.145 0-.414-.012-.418-1.035-.223-.508.094-1.016.199-1.125.234-.176.051-.262.027-.746-.23-.301-.16-.578-.316-.613-.344a47.47 47.47 0 0 0-1.984-1.055c-.02 0-.035-.082.461 2.332.34 1.668.59 2.754.641 2.809a16.64 16.64 0 0 0 .703.293 34.7 34.7 0 0 1 1.086.465c.82.383 1.527.691 1.598.695.039 0 .246-.129.465-.289zm59.02-2.289l.41-1.496c.074-.273.156-.555.188-.621.113-.25.938-4.012 1.094-4.98.02-.137.063-.305.094-.375.059-.133.355-1.941.555-3.391.164-1.211.355-3.047.438-4.262l.129-1.535c.078-.695.078-3.258 0-3.75-.035-.211-.094-.758-.129-1.223-.152-1.953-.523-4.605-.785-5.633-.227-.891-.777-2.418-1.121-3.102-.168-.336-.664-1.152-1.027-1.691-.531-.797-2.047-2.395-2.945-3.109-.207-.16-.418-.34-.469-.398-.129-.141-1.898-1.457-2.613-1.937-.324-.219-.625-.43-.664-.469s-.223-.164-.406-.277l-1.078-.687c-1.223-.785-2.031-1.219-4.262-2.281a25.23 25.23 0 0 0-1.371-.59l-1.43-.578a36.79 36.79 0 0 0-1.898-.684c-.687-.223-1.301-.434-1.371-.469-.215-.105-3.262-.992-4.637-1.352a118.88 118.88 0 0 0-2.426-.59l-1.836-.43c-.566-.145-4.805-.93-5.016-.93-.035 0 .039.152.172.34l.234.34.719.164 1.305.32 2.707.703 5.883 1.535 1.434.402 2.645.754c.68.238 2.48.93 2.828 1.09l1.402.609c2.074.891 4.477 2.16 6.195 3.262 3.09 1.984 4.996 3.75 6.496 6.012.367.551 1.094 1.973 1.371 2.676.383.973.875 2.605 1.059 3.516.402 2.02.547 3.82.547 6.879-.004 3.48-.18 6.137-.609 9.18a116.8 116.8 0 0 1-1.273 6.473c-.707 2.801-1.066 4.348-.871 3.734.027-.086.18-.602.336-1.148zm-121.414.953c-.02-.051-.035-.035-.035.035s.012.102.031.082.023-.074.004-.117zm-.168-.676c-.203-.738-.59-2.355-.867-3.609l-.379-1.707c-.367-1.605-.895-5.078-1.117-7.348-.23-2.324-.332-7.215-.187-8.965.168-1.992.578-4.312 1.035-5.82a24.54 24.54 0 0 1 1.074-2.664 16.91 16.91 0 0 1 1.93-2.961c.52-.613 1.656-1.734 2.34-2.312l.563-.473c.398-.336 1.59-1.16 2.863-1.977 1.336-.855 4.395-2.437 6.035-3.121l.965-.414c1.91-.867 3.934-1.504 8.965-2.828l2.086-.551c.227-.066 4.301-1.102 5.66-1.437 1.023-.254 1.039-.266 1.371-.703l.184-.242-.309.043a138.53 138.53 0 0 0-4.512.836l-1.215.285-1.367.316c-1.66.375-5.836 1.527-6.566 1.813a162.98 162.98 0 0 1-1.543.527c-.793.27-1.566.543-1.711.609s-.727.305-1.293.527c-1.723.68-4.41 2.027-5.598 2.809-2.91 1.902-3.633 2.414-5.035 3.547-1.852 1.5-3.422 3.281-4.156 4.715-.902 1.766-1.207 2.664-1.578 4.652-.07.395-.16.816-.191.934s-.074.332-.09.469l-.121.996c-.539 4.387-.434 8.586.336 13.441.5 3.145.754 4.445 1.328 6.848.512 2.129.586 2.406.906 3.41.301.953.418 1.164.195.355zm41.758-.172c-.793-1.805-.352-4.055 1.164-5.934.363-.449.387-.496.277-.551-.203-.109-.465-.07-.871.129-.852.426-1.437 1.313-1.687 2.543-.285 1.422.043 2.813.887 3.762.172.191.32.348.336.348s-.035-.133-.105-.297zm40.02-.687c.75-1.266.734-3.062-.035-4.465-.305-.551-.703-.941-1.223-1.203-.34-.168-.469-.203-.672-.176-.137.02-.266.047-.281.066s.082.168.223.328c.422.477 1.074 1.555 1.289 2.133a5.32 5.32 0 0 1 .383 2.336c-.031.719-.074.93-.387 1.809-.078.227.441-.391.703-.828zm-36.422.477c-.187-.328-.504-1.102-.605-1.484-.113-.418-.098-1.512.027-1.992.148-.555.676-1.605.988-1.961.246-.281.25-.297.137-.379-.172-.129-.574-.223-.73-.176-.824.254-1.359 1.086-1.473 2.293-.125 1.332.215 2.52.953 3.328.254.273.797.703.836.66.012-.012-.047-.141-.133-.289zm32.293.137c.34-.223.902-.898 1.117-1.34.32-.66.418-1.121.414-1.988-.004-1.449-.402-2.285-1.293-2.715-.293-.145-.512-.129-.871.059l-.199.102.273.305c.352.391.809 1.301.969 1.934.277 1.102.094 2.34-.512 3.441-.195.355-.18.387.102.203zm-39.109-1.332c-.02-.566.004-.812.109-1.23.238-.953.945-2.242 1.758-3.203.234-.281.426-.539.426-.578 0-.113-.457-.078-.816.066-.879.352-1.422 1-1.844 2.199-.227.641-.266 1.684-.086 2.395.102.406.387 1.059.461 1.059.008 0 .004-.316-.008-.707zm46.105.113c.203-.539.273-1.043.238-1.754-.07-1.441-.801-2.723-1.824-3.203-.441-.207-1.023-.309-1.023-.18 0 .039.238.363.523.719.727.898 1.238 1.801 1.602 2.832.121.348.145.531.145 1.293.004.855.008.883.113.746.059-.074.16-.281.227-.453zm-23.48-1.883c1.051-.73 1.828-1.348 1.801-1.422-.02-.047-.051-.852-.074-1.793s-.062-1.715-.09-1.715a63.51 63.51 0 0 0-2.551.465c-.086.031-.969-.496-1.801-1.07-.187-.129-.566-.371-.84-.535s-.551-.348-.609-.398-.121-.086-.137-.074c-.031.035.191 3.355.277 4.102.09.766.117.875.242.926.18.074 1.715.902 1.75.941.016.023.297.176.621.344s.676.352.777.414.191.113.199.113a14.91 14.91 0 0 0 .434-.297zm-8.781-.527c.234-.418.496-.875.59-1.012.152-.23.434-.742.434-.797 0-.035-.437-.164-.75-.219l-.277-.051-.539 1.066-.535 1.066.316.355c.176.195.324.355.332.355s.199-.344.43-.766zm19.086.109c.027-.043-.176-.535-.465-1.137l-.512-1.062-.187.031c-.852.152-.852.156-.434.871l.633 1.109c.152.277.32.566.371.645l.094.145.227-.262a4.57 4.57 0 0 0 .273-.34zm-17.539-2.441c.508-.52.766-1.398.703-2.383-.035-.52-.195-1.289-.289-1.383-.074-.082-.762.402-1.094.773-.605.668-.902 1.434-.902 2.332v.516l.234.039c.125.02.398.102.605.184s.406.145.445.148.172-.098.297-.227zm15.691.094c.102-.035.211-.09.242-.117s.133-.055.219-.055c.418-.004.441-.031.441-.598 0-1.082-.629-2.258-1.523-2.852-.184-.121-.363-.223-.395-.223-.121 0-.289 1.016-.285 1.742.004.594.027.777.148 1.09.266.688.664 1.195.871 1.113a7.53 7.53 0 0 1 .281-.102zm-29.816-1.391c.172-.051.676-.109 1.121-.129.828-.035 1.574-.16 2.535-.43.801-.227 1.727-.605 1.727-.707 0-.172-2.926-.219-3.641-.055-.574.133-.73.195-1.227.496-.266.156-.504.363-.641.547-.191.266-.254.375-.199.375.008 0 .152-.043.324-.098zm43.602.063c0-.109-.379-.539-.652-.742-.398-.301-.883-.516-1.434-.641-.719-.164-3.711-.117-3.605.059.055.09.707.367 1.375.586.879.289 2.039.512 2.883.551.395.02.855.074 1.027.121.387.109.406.109.406.066zm-22.937-.902a59.33 59.33 0 0 1 1.184-.723c.547-.328 1.043-.637 1.105-.691.09-.078.109-.203.109-.723 0-.695.102-2.418.18-2.977.039-.309.031-.359-.051-.359-.055 0-.59.129-1.187.285l-1.246.32a1.41 1.41 0 0 0-.34.152c-.254.152-.285.148-.535-.105-.117-.121-.5-.461-.852-.75s-.637-.551-.637-.574-.176-.184-.391-.355a14.92 14.92 0 0 1-.793-.684l-.535-.484-.133-.109.023.605.188 4.391.039.605.41.258 2.34 1.516c.445.289.84.527.875.527s.145-.055.246-.125zm1.246-6.102c1.441-.887 1.648-1.023 1.676-1.113.109-.344.465-3.98.473-4.805 0-.094-.039-.094-.453.023a40.98 40.98 0 0 0-1.293.395l-1.293.406-.457.141-2.199-2.195-2.195-2.195-.035.207a22.89 22.89 0 0 0-.109 1.176c-.07.859-.062 1.199.059 3.129l.184 2.277c.043.113.734.77 1.816 1.719l1.051.945a18.2 18.2 0 0 0 .754.66l.34.277.5-.312 1.184-.734zm-10.363.188c1.066-.129 1.125-.148 1.383-.504.238-.316.277-.449.184-.598-.039-.062-.176-.02-.578.188-.527.273-1.129.551-1.859.867l-.406.172.375-.031.902-.094zm19.855.086c-.152-.055-.844-.379-1.863-.871-.895-.434-.969-.457-1.043-.355-.047.063-.082.156-.082.211 0 .156.348.672.477.707.215.063 1.355.238 1.855.285.633.063.777.066.656.023zm-57.672-1.574c.188-1.566.676-3.891 1.102-5.215a28.22 28.22 0 0 1 1.094-2.656 13.91 13.91 0 0 1 1.883-2.773c1.496-1.664 3.477-2.812 5.945-3.449 1.672-.434 2.465-.531 4.234-.535h1.309l.305-.359.305-.355-.273-.043c-.531-.078-2.52-.051-3.305.047-.891.113-2.242.398-2.914.613-1.488.473-2.781.992-3.27 1.313a7.99 7.99 0 0 1-.562.332c-.469.234-1.301.898-1.961 1.563-1.012 1.016-1.867 2.367-2.449 3.863l-.312.777c-.383.867-.91 3.266-1.156 5.262-.062.52-.148 2.531-.105 2.488.02-.016.078-.41.133-.871zm94.398-.809c-.039-.602-.113-1.41-.164-1.805-.109-.848-.445-2.547-.52-2.645-.031-.035-.074-.16-.102-.281s-.195-.68-.383-1.242c-.523-1.578-1.203-2.848-2.16-4.043-.324-.402-.711-.832-.863-.957a15.49 15.49 0 0 1-.551-.477c-.289-.27-1.598-1.113-2.172-1.406s-2.598-1.02-3.379-1.219c-1.633-.41-2.75-.555-4.273-.551-.633.004-1.258.023-1.387.047l-.234.043.32.359.316.355h1.207c1.695 0 2.625.117 4.25.531 1.793.453 3.379 1.215 4.699 2.258.512.402 1.375 1.273 1.707 1.727.754 1.016.883 1.219 1.367 2.145 1.039 1.992 1.75 4.359 2.125 7.066.223 1.609.215 1.574.242 1.367.012-.102-.008-.676-.047-1.273zm-58.016-1.246c.492-.055.547-.074.73-.273.234-.254.309-.488.227-.715-.082-.234-.223-.23-.758.008-.43.195-.898.348-2.457.809-.383.109-.453.148-.344.184.227.063 2.02.055 2.602-.012zm24.184 0c.105-.047.109-.055.016-.059-.062 0-.16-.027-.219-.059s-.516-.172-1.012-.316a18.87 18.87 0 0 1-1.492-.5c-.617-.258-.711-.254-.762.027-.039.203.082.465.32.688.156.148.273.195.551.227.547.059 2.457.055 2.598-.008zm-48.117-2.238c.191-1.699.336-2.363.887-4.07.563-1.727 1.492-3.332 2.57-4.43l.43-.441-.387.023c-.414.027-.348-.02-.949.648-.355.391-1.34 1.926-1.625 2.531-.484 1.023-.785 1.98-.926 2.934-.09.625-.207 3.805-.133 3.738.016-.02.078-.437.133-.934zm69.539-.797c-.055-1.906-.227-3.008-.637-4.105-.227-.602-1.266-2.477-1.461-2.629-.02-.02-.195-.242-.387-.5-.621-.84-.645-.855-1.113-.891l-.414-.027.406.383c.75.703 1.668 2.133 2.223 3.461l.281.652c.352.781.805 2.84 1.039 4.715.078.617.098.25.063-1.059zm-34.121.41L66.918 58l.289-.18.039-1.977.074-2.992c.039-.977.035-1.02-.078-.988-.586.18-2.359.805-2.418.855-.098.082-.512-.344-2.707-2.758-.395-.434-.73-.789-.754-.789s-.07.117-.113.262c-.094.32-.547 1.531-.633 1.684-.133.242-1.344 3.711-1.402 4.02-.039.207-.016.234 2.254 2.504l2.293 2.293.188-.113.75-.453zm12.137-1.426c.273-.035.582-.09.688-.121a19.63 19.63 0 0 1 .996-.254c.445-.105.848-.219.898-.242.051-.031-.195-.055-.551-.055-1.129-.012-2.395-.152-3.199-.355-.344-.09-.367-.086-.543.039-.336.238-.301.648.082.969.176.148.215.156.66.117.258-.02.695-.066.969-.098zm-23.75-.152c.262-.301.266-.582 0-.801-.219-.184-.344-.195-.758-.07-.473.145-1.602.281-2.652.324-.609.027-.91.059-.824.09.074.031.59.137 1.145.242l1.32.254a14.67 14.67 0 0 0 1.441.172c.082.008.211-.074.328-.211zm12.012-5.68a8.05 8.05 0 0 1 .629-.414 37.81 37.81 0 0 0 1.004-.641l.598-.395.039-.379c.023-.211.039-.57.039-.801 0-.641.117-1.621.246-2.094l.305-1.07.188-.648-1.031.043c-.562.027-1.059.059-1.098.074-.066.023-.887-1.301-1.246-2.016-.09-.18-.187-.324-.215-.324-.062 0-.957 1.258-1.258 1.762-.117.199-.262.41-.316.473-.098.105-.691 1.117-1.023 1.75-.102.188-.199.355-.223.371s-.113.199-.203.398c-.187.426-.23.324.473 1.066a23.12 23.12 0 0 1 .625.684 11.69 11.69 0 0 0 .52.566c.168.176.602.645.961 1.039s.676.723.707.727.156-.078.281-.172zM54.816 52c1.754-.281 2.703-1.016 3.676-2.828.258-.488.422-.922.371-.973-.02-.02-.254.012-.523.066-1.336.277-5.039.91-8.266 1.406-1.277.199-3.773.496-4.16.496-.133 0-.23.023-.219.055s.324.258.691.504l.668.453 1.043.203a47.03 47.03 0 0 0 2.164.367c2.422.352 3.543.41 4.555.25zm23.918-.375c1.555-.172 3.434-.523 4.152-.773.316-.109 1.668-.926 1.617-.977-.016-.012-.441-.066-.953-.121a36.78 36.78 0 0 1-1.484-.18l-2.023-.309c-2.363-.355-4.84-.773-5.379-.902l-.723-.152-1.238-.223c-.562-.102-1.031-.176-1.039-.168-.027.027.32.992.441 1.223.484.941 1.32 1.867 2.016 2.234.926.492 2.359.598 4.613.348zm9.773-.312c1.398-.152 2.793-.488 3.039-.734.102-.105.066-.312-.098-.52l-.109-.137-2.148.34c-3.957.625-3.773.586-4.027.871l-.219.246.613.035c.82.047 1.922.008 2.949-.102zm-44.383-.121c.043-.043-.445-.148-1.566-.328-6.324-1.027-5.934-.984-6.113-.637-.078.145-.062.25.055.422a12.88 12.88 0 0 0 .629.164c.645.16 1.684.316 2.707.41.824.074 4.207.051 4.289-.031zm-9.98-1.461c.203-.07.691-.305 1.086-.52.805-.445.801-.434.523-.992-.238-.484-.422-.687-.574-.641-.445.145-2.258 1.617-2.25 1.836a.39.39 0 0 0 .074.195c.211.273.574.313 1.141.121zm60.652.063a.71.71 0 0 0 .223-.266l.102-.199-.453-.426c-.812-.758-1.758-1.402-1.953-1.328-.113.043-.387.504-.543.914l-.117.313.297.188c.316.203 1.051.566 1.508.75.316.125.758.148.938.055zM34.531 47.5l.355-.242.203-1.051c.391-2.031 1.625-7.07 1.762-7.211.031-.031.059-.102.059-.152 0-.187.82-2.719 1.07-3.305l.281-.668c.016-.043-.145-.004-.387.098-.41.168-.422.176-.566.543a76.28 76.28 0 0 0-1.395 4.074l-.469 1.535c-.367 1.105-1.164 4.676-1.359 6.094-.098.68-.117.668.445.285zm59.301-.734c-.176-1.168-.473-2.68-.687-3.477l-.738-2.707c-.281-.996-1.391-4.301-1.656-4.934-.207-.5-.246-.535-.754-.746l-.223-.094.34.91.418 1.098c.16.363.707 2.258 1.34 4.637a57.14 57.14 0 0 1 .391 1.648l.387 1.711.359 1.598.199.883.371.25c.207.137.379.242.387.234s-.051-.461-.133-1.012zM67.656 44.91l.777-.242c0-.004.094-.277.207-.602.664-1.887.93-2.711.887-2.758-.074-.078-1.805.027-1.895.113-.043.043-.27.57-.508 1.176s-.457 1.094-.488 1.094-.031-.086 0-.199c.059-.219.051-2.52-.012-2.969l-.039-.273-.328.629c-.441.84-.566 1.047-.871 1.414l-.434.543-.172.234.199.352c.109.191.223.383.258.418s.242.398.465.797l.402.727.387-.113 1.164-.34zm1.547-4.523c.297-.105.563-.211.59-.238.105-.102.352-1.348.352-1.781 0-.344-.109-1.027-.176-1.094-.09-.09-.977.113-1.285.297l-.309.188-.078.68A18 18 0 0 1 68 40.012l-.219.914c0 .012.199-.059.441-.164s.68-.273.98-.375zm-19.004-2.187c-.121-.211-.254-.383-.297-.379a4.3 4.3 0 0 0-.406.133l-.332.121.172.109c.223.148.832.398.969.402.094.004.074-.059-.105-.387zm-1.094-.488c.566-.191 2.742-1.344 2.742-1.453 0-.008-.187.035-.422.102-.641.18-1.207.207-1.824.09-.293-.055-.664-.102-.82-.102-.387 0-.805.191-1.012.461-.094.125-.195.207-.223.191s-.055-.246-.055-.5c0-.875.281-1.234 1.184-1.516.125-.039.711-.109 1.309-.16 1.242-.102 1.582-.195 1.941-.539l.254-.242.246.164c.93.613 1.672.379 2.383-.762a7.65 7.65 0 0 0 .734-1.656c.039-.164.031-.164-.418.141-.437.301-.461.324-.531.648-.172.766-.629 1.078-1.48 1.008a21.76 21.76 0 0 0-1.141-.055c-.527-.012-.715-.043-.957-.156-.352-.16-.414-.168-.414-.055 0 .121-.727.801-1.059.988-.242.141-.812.355-1.461.547a4.19 4.19 0 0 1-.57.098c-.75.082-.949.336-.84 1.066.102.664.508 1.293 1.059 1.645.355.223.805.238 1.375.047zm2.645-.316c-.117-.18-.223-.352-.234-.391-.023-.07-.211.016-.473.215l-.152.113.242.109c.246.105.707.266.789.273.023 0-.055-.141-.172-.32zm17.398-.586c.293-.176.555-.363.586-.414.078-.121-.145-.672-.477-1.184-.258-.402-.641-.797-1.082-1.105l-.258-.18-.09.137c-.246.375-.426.828-.488 1.23l-.066.445.313.289c.305.285.691.961.691 1.215 0 .113.008.113.172.004.094-.062.41-.258.699-.437zm-15.934-.305c0-.02-.059-.117-.133-.215-.129-.176-.141-.18-.316-.086-.211.109-.203.129.121.242.266.094.328.105.328.059zm8.016-.965l.449-.082v-.23a15.79 15.79 0 0 0-.27-1.723c-.027-.023-.645.277-.68.332-.016.031-.051.457-.074.949-.039.742-.031.891.039.867a4.81 4.81 0 0 1 .535-.113zm5.859-.91c.074-.531.129-.977.117-.988-.082-.074-.848-.172-1.547-.199l-.848-.035-.07.363a2.96 2.96 0 0 0-.035 1.09l.035.223.508.066c.598.074 1.227.23 1.461.359.094.047.184.09.203.094s.102-.437.176-.973zm-4.219.594c.055-.027.402-.07.77-.094l.672-.043-.039-.598c-.023-.328-.059-.707-.082-.84l-.039-.242-.691.047c-1.152.078-1.285.105-1.426.297-.117.156-.121.199-.051.715.137.977.094.898.465.852a1.99 1.99 0 0 0 .422-.094zm12.223-1.961c-.336-.262-1.895-1.043-1.957-.98-.07.07.277.285.93.578l.902.41c.246.109.277.109.125-.008zm-22.883-.117c.012-.012-.09-.137-.227-.277-.324-.332-.469-.645-.422-.902.055-.297.527-.961.898-1.262.18-.148.352-.266.383-.266s.063.082.063.184c0 .406.227.625.656.625.445 0 .801-.605.617-1.043-.043-.105-.125-.215-.18-.246-.078-.043-.012-.117.328-.379 1.379-1.047 4.117-2.488 5.766-3.031 1.105-.359 1.051-.34 1.016-.375-.094-.094-2.867.516-3.363.738-.113.051-.637.25-1.168.441-1.117.406-3.172 1.414-3.945 1.934-1.102.738-1.711 1.348-2.004 1.992-.437.973-.098 1.695.906 1.902.195.043.621.02.676-.035zm-13.32-.469c.359-.187.57-.586.57-1.062 0-.34-.047-.422-.328-.562-.543-.277-1.484-.203-1.613.125-.102.266-.027 1.105.117 1.301.285.391.754.465 1.254.199zm51.133.102c.32-.133.434-.367.465-.953.039-.719.008-.793-.332-.891-.59-.164-1.477.074-1.566.422-.055.219.008.641.133.906.227.469.832.711 1.301.516zm-53.137-1.66l.215-.223-1.016-.27c-.555-.148-1.055-.293-1.105-.324s-1.031-.293-2.18-.586c-4.559-1.168-4.656-1.187-5.152-.934l-.324.164-.855-.125c-.926-.133-2.785-.332-4.125-.441a82.81 82.81 0 0 0-6.965-.062c-2.176.141-2.07.16.836.16 1.77 0 3.254.031 3.891.078 3.621.285 5.461.523 9.242 1.195 2.73.488 4.531.914 6.383 1.516.734.234.844.258.891.172.031-.055.152-.199.266-.32zm56.734-.246c1.973-.547 3.105-.793 5.563-1.215l1.617-.277c3.691-.633 7.902-.969 11.691-.926 2.246.023 2.277 0 .164-.137-1.555-.098-5.742-.047-7.344.09-1.543.133-2.785.27-3.77.418l-.816.121-.336-.156a1.7 1.7 0 0 0-.543-.156 23.73 23.73 0 0 0-2.379.5c-.117.039-1.168.316-2.332.613l-2.645.711-1.047.297-.52.133.184.195c.102.105.215.254.25.324l.066.133.551-.184c.301-.102 1.039-.32 1.645-.484zm-54.844-.422a9.76 9.76 0 0 0-.301-.637c-.621-1.223-1.812-2.633-2.969-3.512-.309-.234-.574-.445-.59-.469-.074-.098-1.434-.875-1.531-.875-.027 0 .191.188.477.414 1.117.875 2.445 2.227 3.121 3.168.289.402.859 1.359 1.004 1.688.117.258.121.258.453.262.184.004.336-.016.336-.039zm51.438-.578c.375-.754 1.039-1.687 1.824-2.562.332-.367.617-.668.637-.672s.27-.223.563-.492.73-.641.977-.824.453-.348.453-.371c.004-.09-1.211.621-1.816 1.066-.727.527-1.977 1.746-2.461 2.398-.297.398-.937 1.492-1.117 1.91l-.074.172.355-.008.355-.012zm-20.895-.336c.508-.195 2.199-.52 2.836-.539.207-.008.168-.02-.172-.062-.867-.113-2.445.047-3.199.324-.41.148-.422.172-.113.285.27.102.375.098.648-.008zm-6.953-.305c.484-.074.566-.109 1.004-.426.813-.582 1.813-1.016 3.352-1.445l.344-.098-.562.035c-1.086.063-2.375.355-3.035.691a6.83 6.83 0 0 0-1.473 1.043c-.152.156-.25.285-.219.281s.297-.039.59-.082zm-49.375-.699c-.02-.02-.074-.023-.121-.004s-.035.035.039.035c.066.004.102-.012.082-.031zm0 0" fill="#97c3e6"/><path d="M82.969 103.938c-1.98-.211-4.883-1.004-6.84-1.871-2.312-1.02-3.871-2.113-5.863-4.098-1.336-1.332-1.824-1.879-3.262-3.641l-.937-1.141c-.059-.07-.211-.012-.84.301-1.488.746-5.828 2.551-8.121 3.375-3.941 1.422-7.195 2.289-9.988 2.664-1.254.168-3.773.23-5.074.125-4.066-.324-7.086-1.422-8.406-3.055-.691-.855-1.246-2.062-1.086-2.359.063-.117.082-.121.238-.043.094.051.512.359.938.688 1.188.926 1.844 1.258 3.184 1.613 1.496.398 2.605.539 4.555.582 2.246.047 2.91-.031 5.371-.641 2.773-.691 5.391-1.578 7.531-2.562.918-.422 6.324-3.117 8.23-4.102 1.227-.637 1.262-.656 1.223-.82-.059-.246-.422-1.223-.863-2.312-.488-1.199-1.254-3.305-2.113-5.82-.852-2.477-1.387-3.961-1.461-4.035-.066-.066-.738.828-2.055 2.742-1.062 1.551-1.312 1.824-1.668 1.82-.469-.004-1.289-.449-2.289-1.246-.273-.219-1.254-1.145-2.18-2.062s-1.715-1.668-1.75-1.668c-.293 0-.531.922-.539 2.055 0 .766.012.875.164 1.27.094.234.156.441.137.461-.059.059-1.047-.586-1.371-.895-.41-.395-.832-1.238-1.047-2.102-.086-.336-.176-.633-.199-.656-.07-.07-.52.488-.711.887-.324.68-.461 1.344-.453 2.199.004.422-.008.766-.023.766-.117 0-.426-.234-.668-.5-.562-.629-.926-1.449-1.07-2.434-.141-.93-.148-.941-.496-.539-.504.59-.898 1.676-.902 2.492v.355l-.348-.379c-.652-.707-.934-1.594-.875-2.73.043-.84.195-1.352.656-2.246.207-.398.434-.941.504-1.211.273-1.016.719-1.609 1.527-2.031.695-.363 1-.43 2.176-.469 1.816-.062 3.32.109 4.172.477.395.172.965.754 2.477 2.531l1.469 1.703c.301.332.309.34.406.219.055-.07.297-.52.535-.996l.434-.871.059-.871c.082-1.219.332-1.836 1.02-2.512.473-.465.578-.543 1.148-.875.246-.145.449-.297.449-.336l-.059-.699c-.082-.887-.172-1.262-.328-1.344-.074-.039-.457-.094-.852-.125-1.203-.09-3.004-.477-4.129-.891-2.074-.762-5.379-2.418-9.445-4.727-2.012-1.141-3.664-1.988-4.57-2.336-2.406-.926-4.957-1.211-6.508-.719-1.207.379-2.418 1.16-2.637 1.699-.051.129-.152.629-.223 1.105-.145.969-.195 1.223-.223 1.195-.012-.012-.066-.488-.121-1.059s-.125-1.109-.156-1.195c-.117-.309-1.09-.844-1.887-1.035-1.062-.258-3.035-.223-3.875.063-1.703.586-3.816 2.199-5.402 4.125l-.527.641-.137.902c-.156 1.039-.301 1.727-.371 1.77-.082.051-.129-.496-.129-1.527v-.98l-.574-.383c-.805-.539-1.715-.969-2.445-1.16-.555-.141-.707-.156-1.43-.129-.656.023-.895.059-1.262.184-2.461.855-4.516 3.828-5.957 8.602-.922 3.07-1.41 5.449-1.418 6.91-.004 1.02.094 1.629.605 3.793.227.953.398 1.746.379 1.766-.125.125-1.113-2.637-1.746-4.875C1.449 74.469.332 68.27.129 64.762-.156 59.828.09 55.777.871 52.5c.332-1.395 1.395-4.641 2.07-6.316 2.395-5.957 6.934-11.578 13.023-16.125.582-.434 1.137-.875 1.234-.98l.18-.187-.363-.047c-.848-.105-5.727.047-7.266.227-.305.035-.566.051-.582.035-.078-.078 2.188-.543 3.344-.687 4.191-.52 9.781-.43 13.75.219l.824.133.492-.168c.379-.129.559-.156.758-.125.484.074 2.359.527 4.402 1.066l4.582 1.141c.133 0 .152-.02.113-.137-.074-.246-.281-.621-.633-1.168-.766-1.18-1.289-1.785-3.152-3.641-1.105-1.098-1.41-1.434-1.32-1.453.148-.031 1.57.684 2.309 1.16 1.184.762 2.355 1.832 3.066 2.785.281.379.531.785 1.227 1.988.168.293.469.723.668.953.563.645.734 1.266.52 1.867-.133.379-.332.621-.75.898-.203.137-.426.309-.488.383-.383.441-1.352 3.121-2.062 5.699-.676 2.453-1.648 6.57-1.648 6.977 0 .074.125.273.281.449.438.488.703 1.098.875 2.016.012.07.07.148.129.172.082.031 2.902.504 7.789 1.305l.52.082.152-.242a5.54 5.54 0 0 0 .254-.453c.055-.113.16-.227.23-.246s.914-.133 1.875-.254c2.754-.352 6.422-.918 9.664-1.496 1.688-.301 2.031-.336 2.25-.219.133.074.254-.035 2.148-1.937 2.57-2.578 3.672-3.781 4.203-4.578 1.445-2.176 1.711-4.707.598-5.73-.355-.328-.73-.406-1.902-.398-1.047.008-2.043.129-3.234.391-1.523.336-1.645.391-1.949.863-.332.512-.789.918-1.488 1.328-.871.512-1.328.883-2.457 1.977-1.676 1.633-2.055 1.902-2.582 1.852-.375-.039-1.133-.41-1.164-.57-.027-.148.313-.504.953-.992 1.148-.871 2.246-2.168 2.621-3.094.23-.566.32-1.262.207-1.617-.078-.246-.113-.281-.285-.297-.105-.012-.465.07-.797.18-.594.195-.902.371-.902.512 0 .039.188.262.418.496l.422.43-.219-.039a26.54 26.54 0 0 1-.891-.203c-.586-.141-.707-.152-.922-.086-.441.133-.395.328.246 1.012.293.313.367.422.281.422-.336-.008-1.105-.191-1.516-.367-.691-.293-.934-.172-.684.344.125.254.363.508.777.82l.188.141-.312-.035c-.539-.066-1.031-.211-1.68-.492-.395-.172-.762-.289-.996-.312-.441-.043-.781-.227-1.137-.613-.559-.609-.844-1.602-.633-2.211.094-.273.375-.418 1.023-.539 1.164-.219 1.922-.559 2.477-1.113.355-.355.438-.559.344-.836-.043-.125-.078-.48-.078-.789 0-.516.016-.59.207-.949.867-1.617 4.598-3.645 8.668-4.711 1.34-.352 2.398-.484 2.969-.371.184.039.168.055-.508.473-.383.238-.895.59-1.133.785-.566.453-1.734 1.621-2.098 2.098-.281.371-.547.84-.547.961 0 .152.824.008 2.223-.398.941-.27 1.07-.336 1.539-.781 1.344-1.273 2.516-1.766 4.734-1.992.664-.066 1.523 0 1.523.117 0 .027-.496.289-1.105.582-1.637.797-2.258 1.188-2.113 1.332.051.051 1.441.199 2.285.242l.777.043.746-.262c.688-.238.809-.262 1.496-.281.715-.023.723-.02.246.035-.926.105-1.836.352-1.836.5 0 .031.156.098.348.156.266.074.387.082.516.031.297-.117 1.961-.492 2.375-.531.223-.023.516-.059.652-.082l.25-.043-.312-.043c-.219-.031-.137-.039.27-.023 1.02.031.922.137-.625.676-1.082.379-1.543.57-1.539.645 0 .102.105.195.805.719.77.57 1.219.82 2 1.109.66.242 1.434.617 1.895.914.402.262.852.688.809.762-.016.023-.402-.004-.855-.066-1.465-.191-1.723-.207-1.762-.105-.02.051.008.297.066.551a37.92 37.92 0 0 1 .215 1.102c.059.352.152.699.203.777s.484.531.957 1.012c.91.918 1.348 1.477 1.184 1.512-.051.012-.355-.07-.676-.176s-.754-.25-.973-.312l-.391-.113-.051.137c-.07.184-.238 1.254-.289 1.855l-.043.477.316.523c.535.883.809 1.543.645 1.543-.027 0-.293-.156-.594-.344s-.566-.34-.59-.34c-.047 0-1.023 3.457-1.316 4.664-.285 1.18-.344 1.07.668 1.297 2.055.461 7.813 1.426 10.496 1.758 1.723.211 1.98.285 2.059.602.086.344.262.453.715.453.676 0 4.949-.687 5.461-.879.355-.133.352-.129.699-1.184.164-.496.43-.914.723-1.141.117-.09.215-.215.215-.277 0-.172-.758-3.488-1.187-5.184-.961-3.812-1.941-6.734-2.496-7.457-.066-.082-.309-.293-.535-.465-.273-.207-.488-.43-.621-.652-.176-.297-.199-.391-.199-.777 0-.559.164-.93.66-1.48.223-.246.52-.691.762-1.145.914-1.703 1.754-2.711 3.207-3.832.887-.684 2.41-1.543 3.578-2.012l.227-.09-.219.191-.953.852c-.953.852-2.258 2.203-2.91 3.012-.895 1.113-1.715 2.469-1.574 2.605.078.078.973-.117 2.762-.598 2.16-.578 5.281-1.359 5.961-1.492l.582-.109.352.148c.434.176.727.18 1.879.004 2.32-.352 4.207-.473 7.41-.473 3.867 0 6.203.219 8.613.809.691.172.848.25.379.195-1.383-.172-3.844-.293-5.977-.289-1.645 0-1.918.02-1.879.145.027.09.734.66 2.02 1.641 5.609 4.281 10.109 9.973 12.336 15.602a61.87 61.87 0 0 1 2.051 6.355c.848 3.383 1.098 8.426.66 13.195-.352 3.82-1.145 8.16-2.246 12.285-.34 1.277-.895 3.082-1.184 3.848-.266.715-.699 1.664-.742 1.625-.016-.016.238-1.191.57-2.605l.598-2.578-.137-.84c-.512-3.152-1.484-6.77-2.469-9.184-1.18-2.891-3.035-5.109-4.82-5.762-.902-.328-1.996-.359-2.918-.082-.844.254-2.371 1.043-2.723 1.406-.164.176-.168.184-.215 1.398-.07 1.719-.117 1.715-.406-.012-.109-.668-.254-1.332-.32-1.477-.305-.68-2.523-2.832-3.777-3.66-.543-.359-1.609-.895-2.18-1.094-.484-.168-.527-.172-1.926-.176-1.637-.004-1.789.027-2.898.578-.531.262-.602.32-.727.57-.094.195-.156.496-.203.965-.078.773-.117.965-.191.965-.027 0-.07-.09-.09-.203s-.086-.516-.148-.898-.172-.805-.238-.934c-.164-.32-.855-.891-1.402-1.16-.973-.473-1.863-.664-3.133-.664-1.008 0-1.555.074-2.609.344-1.805.461-3.453 1.195-6.16 2.742-3.371 1.922-5.504 3.035-8.035 4.184-2.305 1.051-4.02 1.504-6.156 1.633-.48.027-.922.07-.98.094-.164.063-.238.414-.277 1.289l-.039.797h.152c.375.004 1.363.133 1.668.223 1.199.348 2.07 1.191 2.395 2.328.055.188.137.711.188 1.16.063.578.137.941.258 1.242.172.441.516 1.102.73 1.414l.121.176.188-.176c.102-.098.723-.805 1.375-1.578 1.68-1.973 2.285-2.59 2.75-2.801.516-.234 1.133-.363 2.133-.445 1.059-.082 2.902-.027 3.371.102.859.234 1.641.82 2.004 1.5.094.168.242.574.332.898.098.352.277.785.438 1.059.34.586.492.977.617 1.578.273 1.316-.035 2.699-.793 3.555l-.27.309-.039-.496c-.066-.82-.496-1.875-.957-2.352-.266-.273-.395-.227-.395.148 0 .898-.516 2.148-1.187 2.879-.227.25-.426.414-.484.402-.086-.016-.113-.18-.172-.949-.09-1.133-.262-1.727-.699-2.367-.395-.578-.504-.578-.609-.012-.156.855-.625 1.84-1.102 2.313-.305.305-1.129.844-1.289.844-.043 0-.055-.043-.031-.109.16-.398.281-1.094.281-1.586 0-.895-.293-2.039-.52-2.039-.055 0-.906.793-1.898 1.766-1.961 1.926-2.66 2.512-3.48 2.918-.875.434-1.254.371-1.703-.293-1.266-1.871-2.777-3.953-2.867-3.957s-.047.359.203 1.781c.504 2.863 1.301 6.727 1.422 6.926.043.066.25.004 1.012-.324 1.637-.699 3.168-1.156 5.219-1.559 2.141-.422 3.328-.535 5.602-.535 2.98 0 4.816.27 7.047 1.031 3.91 1.34 6.309 3.75 7.195 7.23.156.629.168.746.164 2.039 0 1.184-.02 1.457-.145 2.023-.836 3.875-4.918 7.391-9.754 8.41-.535.113-.879.133-2.578.148-1.078.012-2.199-.008-2.488-.039zm1.141-.039a8 8 0 0 0 .07-.777l.043-.73-.176-.035c-.098-.02-.555-.129-1.016-.246l-.93-.23c-.07-.016-.113.152-.203.789-.062.445-.102.84-.09.875s.371.113.793.176l1.051.16c.387.07.426.07.457.02zm2.535-.055l.219-.035-.047-.504c-.031-.289-.086-.527-.129-.555s-.191-.051-.332-.051c-.238 0-1.187-.129-1.621-.219l-.191-.043-.043.695c-.023.383-.023.723 0 .762.043.07 1.68.031 2.145-.051zm1.918-.312c.355-.078.66-.16.676-.176.063-.059-.082-.625-.156-.621-.043.004-.5.012-1.016.016l-.941.012.043.414c.023.23.059.457.078.504.039.105.355.07 1.316-.148zm-7.062-.754l.137-.895c.035-.152-.008-.18-.762-.551-.437-.215-.949-.488-1.133-.605s-.34-.203-.348-.195-.102.246-.215.52c-.441 1.078-.645 1.648-.613 1.73.027.07 1.23.398 2.504.684.32.074.316.082.43-.687zm9.172.063c.598-.266.762-.406.555-.488-.047-.016-.379.039-.742.125s-.715.16-.789.16c-.141 0-.16.09-.09.422.035.18.043.188.23.125.109-.035.484-.191.836-.344zm-12.383-.437c.082-.156.758-2.047.754-2.109 0-.027-.238-.234-.531-.453s-.605-.473-.699-.562-.34-.324-.547-.523l-.383-.355-.156.328c-.086.184-.301.699-.473 1.152s-.367.922-.426 1.043l-.105.223-.562-.035-.562-.039.559.32c1.488.852 2.969 1.328 3.133 1.012zm10.996-.078c2.063-.23 3.934-.965 5.332-2.098.863-.695 2.074-2.293 2.52-3.316.625-1.441.816-3.434.496-5.109-.145-.75-.441-1.75-.621-2.094-.074-.145-.09-.016-.121 1.141-.043 1.625-.172 2.25-.715 3.48-.328.746-.547 1.082-1.094 1.68s-1.504 1.266-2.191 1.531c-.832.32-2.609.738-3.668.859-.785.094-2.746.082-3.453-.02a13.22 13.22 0 0 1-3.336-.945c-.348-.148-.711-.285-.812-.309-.402-.098-2.48-1.109-2.73-1.332-.066-.059-.348-.258-.621-.441-1.328-.883-3.004-2.562-4.125-4.125a14.27 14.27 0 0 1-1.387-2.223c-.656-1.23-.895-1.832-1.25-3.109-.215-.781-.227-.844-.586-2.738l-.781-3.953c-.121-.539-.363-1.914-.484-2.77-.199-1.375-.254-2.262-.297-4.605-.051-2.824.066-5.082.355-6.973.109-.734.188-1.055.492-2.086.113-.383.574-2.285.84-3.453l.219-.941c.145-.625.43-2.281.625-3.664l.137-.965-.207-.312c-.324-.477-.727-1.32-.863-1.797a6.44 6.44 0 0 1-.191-2.289c.059-.492.293-1.344.5-1.824.063-.148 1.055-3.672 1.313-4.672.223-.84.512-2.301.66-3.297.156-1.07.16-3.617.004-4.449-.434-2.309-1.453-3.73-3.359-4.684-2.152-1.07-5.527-1.277-8.949-.543-.812.176-1.523.355-1.93.492-.648.215-1.418.438-1.516.438-.156 0-.125-.109.234-.84.496-1.008 1.02-1.656 2.09-2.598l.406-.355-.527.344c-1.035.672-2.445 2.043-3.223 3.141-.426.598-1.164 1.336-1.555 1.555-.16.086-.266.191-.266.258 0 .234-.258.785-.453.965-.352.336-.906.426-1.504.246-.219-.066-.34-.074-.48-.023-.27.094-.887.012-1.277-.168-.289-.129-.332-.137-.332-.047 0 .117-.664.77-.996.977-.496.313-1.676.656-2.246.656a.9.9 0 0 0-.684.453c-.105.254-.023.945.16 1.324.086.172.313.473.504.668.559.563.961.625 1.926.316.449-.145 2.527-1.273 2.801-1.523.281-.254 2.043-1.055 2.57-1.168.563-.121.617-.105.703.195.043.145.078.543.078.887 0 .508-.023.684-.141.965a12.36 12.36 0 0 1-.961 1.711c-.266.355-.969 1.031-1.492 1.43-.77.586-1.258.98-1.277 1.027-.027.078.895.438 1.125.438.359 0 .77-.332 2.195-1.777.77-.777 1.492-1.453 1.648-1.543s.539-.293.852-.457c.645-.336 1.281-.867 1.457-1.219.34-.66.398-1.84.113-2.344a1.17 1.17 0 0 0-.359-.383c-.242-.145-.215-.199.148-.297A45.62 45.62 0 0 0 60 33.238l1.34-.406c.496-.148 1.844-.398 2.699-.5 1.383-.16 3.297-.094 3.555.125.051.043.207.113.344.156.887.281 2.004 1.109 2.52 1.867.281.414.574 1.074.598 1.34.008.102.074.398.145.652.168.625.273 1.617.223 2.117-.062.609-.219 1.328-.312 1.43-.047.059-.207.953-.398 2.238-.199 1.363-.348 2.176-.402 2.238-.051.055-.137.352-.191.66l-.16.82c-.035.145-.059.676-.059 1.18 0 .828.035 1.191.184 1.902.031.137.023.254-.016.293s-.039.207.008.516c.176 1.156.227 2.113.227 4.262s-.004 2.254-.121 2.387c-.086.094-.137.266-.168.582-.148 1.57-.375 2.988-.648 4.023-.086.324-.199.816-.25 1.09-.227 1.223-.27 1.363-.453 1.469-.16.094-.184.16-.34.996-.25 1.32-.281 1.621-.359 3.262-.059 1.172-.094 1.516-.168 1.602-.086.098-.082.254.043 1.496.262 2.699.285 3.078.199 3.238-.098.184-.016.91.402 3.629.281 1.832.32 2.23.223 2.289-.051.031.016.578.207 1.688.051.309.238 1.074.41 1.703.211.766.297 1.164.254 1.215-.062.074.246 1.125.441 1.5.199.383.859 2.223.859 2.395 0 .102.086.371.191.605.34.746 1.23 2.527 1.453 2.918.121.207.227.434.238.5.051.324 1.324 2.199 1.988 2.93.641.703 3.285 3.324 3.75 3.715a17.32 17.32 0 0 0 2.191 1.492c.406.207 1.465.633 2.074.832.797.266 2.414.547 3.766.66.582.051 2.285.039 2.801-.02zM47.66 36.52c.098-.473.254-.84.414-.973.148-.121.543-.102.641.031.109.148.105.207-.039.441-.078.129-.195.219-.328.254s-.285.156-.453.371c-.137.176-.266.32-.285.324s0-.199.051-.449zm44.691 65.461c-.012-.016-.195.031-.406.102-.27.09-.371.152-.348.215s.113.047.406-.102c.207-.105.363-.203.348-.215zm-16.75-1.445c.051-.129.145-.344.207-.48.195-.434.75-1.848.75-1.91s-1.687-1.855-1.742-1.855c-.039.004-.406.492-1.113 1.492l-.621.859-.113.148-.918-.137c-.508-.07-.93-.121-.941-.109-.059.055 1.316 1.16 2.117 1.703l.465.32c.152.113.609.176 1.391.191l.422.012zm-28.863-1.137c3.512-.504 6.48-1.277 8.406-2.191.359-.172.863-.402 1.121-.516.621-.273 2.898-1.379 5.203-2.531l1.871-.937.238-1.797c.129-.992.223-1.816.207-1.832-.051-.047-.852.332-4.645 2.203l-1.34.684c-.18.141-3.133 1.59-3.773 1.852l-.996.414c-1.328.598-4.84 1.676-7.047 2.164-2.047.449-2.777.527-4.437.477-1.832-.055-2.902-.199-4.488-.598-1.703-.434-2.602-.879-3.598-1.797-.523-.484-.785-.684-.785-.605 0 .141.367.941.637 1.387.602.996 1.355 1.645 2.633 2.27 1.363.668 2.445.965 4.543 1.258 1.773.246 2.035.27 3.527.277a13.69 13.69 0 0 0 2.723-.18zm27.004-2.281c.418-.59.762-1.094.762-1.117s-.219-.312-.484-.641a14.14 14.14 0 0 1-1.316-1.895c-.113-.199-.227-.34-.254-.309s-.43.57-.902 1.203l-.895 1.195c-.035.039-2.309-.543-2.699-.691-.094-.035-.172-.047-.172-.02.004.141 1.848 2.293 2.5 2.914.328.313.34.316.984.426l1.09.184c.414.074.441.07.531-.051.051-.066.434-.609.855-1.199zm14.785.383a10.65 10.65 0 0 0 3.195-1.164c1.543-.926 2.645-2.301 3.098-3.867.242-.84.305-1.297.301-2.281-.004-.773-.027-.992-.168-1.52-.227-.852-.461-1.387-.953-2.172s-.562-.84-1.914-1.516c-1.391-.695-2.539-1.051-4.48-1.395-1.336-.234-1.473-.246-3.285-.285-1.746-.039-3.328.027-4.059.168l-.934.156c-1.043.16-2.266.398-3.051.594-.852.211-1.68.512-1.492.543.719.109 2.445.605 3.199.914.57.234 1.609.762 1.914.973l.262.18h1.48c1.523 0 2.781.07 3.383.188a19.25 19.25 0 0 1 1.992.504c2.379.762 4.051 2.676 4.211 4.816.059.793-.152 1.375-.855 2.363a7.71 7.71 0 0 1-1.559 1.551c-.383.258-1.449.781-1.848.906-.457.145-1.316.313-1.586.313-.391 0-.086.086.469.133.727.059 1.926.012 2.68-.102zm-2.52-.434c1.402-.273 2.617-.945 3.594-1.977.805-.848 1.328-1.812 1.332-2.449.004-.594-.449-1.406-1.156-2.086-1.129-1.082-2.73-1.75-5.129-2.141-1.309-.215-4.699-.18-6.629.07-1.391.18-3.617.938-4.262 1.445l-.148.121.535.715c1.125 1.496 2.305 2.766 3.441 3.688 1.699 1.383 3.773 2.422 5.352 2.676a12.26 12.26 0 0 0 3.07-.062zm-15.16-2.355a10.02 10.02 0 0 1 .473-.641c.063-.066.164-.203.23-.305s.266-.383.445-.625l.324-.437-.594-1.148c-.582-1.121-1.145-2.344-1.145-2.48 0-.094-.168-.086-.23.012-.027.047-.219.219-.43.391l-1.172.977c-.434.367-.844.668-.906.668s-.625-.141-1.246-.316-1.141-.305-1.156-.289c-.047.043.883 1.941 1.422 2.91.289.516.523.867.613.914.082.043.555.191 1.051.332l1.402.402a9.48 9.48 0 0 0 .527.148c.02.004.191-.227.391-.512zm-5.594-3.668c-.316-.645-.461-.887-.461-.773 0 .105.727 1.477.781 1.477.016 0-.129-.316-.32-.703zm3-.621l.918-.793.867-.727c.211-.18.395-.352.406-.387s-.09-.324-.227-.645c-.328-.77-.68-1.707-.875-2.34l-.164-.52-.746.492c-1.391.922-1.984 1.27-2.113 1.238-.137-.031-1.531-.637-2.273-.988-.328-.152-.473-.195-.484-.141-.043.172.512 1.934 1.098 3.5.301.809.348.875.648.953s.844.238.98.293a38.6 38.6 0 0 0 1.496.445c.02.004.23-.168.469-.383zm-4.105-1.648c-.051-.098-.277-.82-.504-1.609s-.437-1.477-.473-1.535-.059-.141-.059-.18-.039-.102-.09-.141-.277-.809-.523-1.781c-.555-2.187-.656-2.539-.773-2.633-.098-.074-.223-.598-.641-2.656l-.273-1.309c-.031-.117-.125-.547-.211-.953-.102-.473-.195-.766-.262-.812-.141-.105-.242-.93-.406-3.363-.125-1.84-.145-2-.266-2.094-.113-.094-.133-.223-.203-1.246-.043-.633-.109-2.039-.148-3.129-.062-1.777-.082-2-.187-2.148-.102-.141-.133-.441-.254-2.25-.145-2.223-.137-3.746.023-4.672.07-.41.07-.496-.008-.613s-.082-.191.02-.617c.121-.527.652-2.184.801-2.508.055-.117.309-.789.566-1.5s.535-1.441.617-1.625c.188-.402.199-.461.141-.672-.039-.125.031-.328.324-.941.203-.43.469-.93.586-1.113s.434-.695.699-1.141l.48-.805-.996 1.023-1.875 1.926-.883.902.047.375c.023.207.043.75.035 1.215-.008 1.02-.184 1.797-.57 2.535-.242.465-.32.574-.945 1.313l-.289.34-.047 1.531c-.051 1.723-.004 4.883.094 6.012l.156 2.211c.148 2.293.469 5.609.59 6.129a16.78 16.78 0 0 1 .16.934c.301 2.023.82 4.684.969 4.945.031.055.227.688.434 1.406.371 1.285 1.773 5.543 1.867 5.664.023.031.074.16.109.277.117.402.953 2.645 1.07 2.875.066.129.32.73.563 1.34.48 1.199.551 1.352.598 1.305.02-.016-.012-.109-.062-.211zM67.027 86l1.391-.91c.742-.484.789-.531.754-.684-.02-.09-.059-.191-.082-.223s-.062-.121-.082-.187c-.316-1.109-.43-1.633-.551-2.492-.078-.562-.16-1.027-.176-1.027-.039 0-.426.285-2.281 1.676l-.531.402-.324-.164c-.176-.09-.797-.359-1.379-.605s-1.164-.496-1.293-.562l-.234-.121.035.141.883 3.406.125.402.492.211.957.43c.871.414 1.527.695 1.613.695.047.004.355-.172.684-.387zm-.516-4.664l1.441-1.105c.43-.328.445-.348.418-.559l-.527-3.301a27.29 27.29 0 0 1-.156-1.023c-.051-.371-.102-.687-.109-.695s-.398.262-.859.605l-1.273.926c-.238.164-.488.352-.559.414a4.35 4.35 0 0 1-.699.512c-.125.066-.215.031-.777-.293a32.52 32.52 0 0 0-1.168-.637c-.293-.145-.707-.375-.922-.512s-.406-.23-.426-.211.051.422.152.895a66.19 66.19 0 0 1 .383 1.887c.215 1.16.352 1.781.469 2.148.078.242.102.254.914.613 1.898.832 2.516 1.094 2.609 1.094.055 0 .547-.34 1.09-.758zM3.449 80.223c-.348-1.488-.406-1.773-.738-3.609l-.469-2.484c-.25-1.191-.699-4.594-.969-7.344-.297-3.07-.383-5.223-.281-7.105.082-1.562.117-1.914.316-3.258s.645-3.25.871-3.734a52.98 52.98 0 0 0 .406-.996c.477-1.195 1.332-2.758 1.895-3.453.125-.156.383-.492.578-.746.613-.812 1.918-2.078 3.023-2.941.711-.551 2.645-1.852 3.293-2.215.285-.156.566-.332.629-.391s.262-.187.449-.289c1.781-.984 3.574-1.785 6.406-2.859.324-.121.676-.262.777-.309.871-.395 6.719-2.305 8.746-2.855l1.773-.5c.543-.168 1.914-.488 3.328-.781l1.402-.305c.273-.062.855-.164 1.293-.227s.793-.129.793-.152-.07-.117-.152-.215c-.16-.184-.316-.512-.371-.785-.035-.152-.051-.156-.27-.109-.129.031-.43.078-.672.113-.988.129-5.52 1.031-6.156 1.223-.168.051-.742.191-1.277.313-1.23.277-2.797.68-3.238.832a23.27 23.27 0 0 1-1.152.344c-1.207.336-2.211.648-2.5.781-.141.063-.336.137-.434.16a45.23 45.23 0 0 0-2.566.906c-.148.066-.484.207-.746.313-1.453.578-3.707 1.637-4.434 2.086-.219.137-.562.336-.758.445-.48.266-1.566.973-3.191 2.074a44.95 44.95 0 0 0-2.957 2.277c-1.84 1.59-2.992 3.012-3.875 4.777a10.73 10.73 0 0 0-.945 2.551c-.512 1.992-.656 2.855-.945 5.695-.156 1.5-.152 5.645 0 7.406.117 1.355.312 3.105.441 3.953.234 1.547.965 5.484 1.18 6.348.699 2.816 1.02 4.02 1.293 4.836.18.539.336.969.348.957s-.055-.34-.145-.727zm121.512-.926c.563-1.934.719-2.547 1.086-4.234l.379-1.711c.387-1.625.965-5.621 1.297-8.961.172-1.727.133-5.77-.066-7.066a64.24 64.24 0 0 1-.187-1.461c-.172-1.445-.398-2.82-.621-3.742-.18-.758-.777-2.395-1.125-3.086-.277-.547-1.176-1.898-1.586-2.387-.484-.57-1.93-2.008-2.32-2.312-.172-.129-.477-.379-.68-.551-.602-.504-2.027-1.539-3.187-2.309l-1.184-.789c-1.359-.918-3.734-2.145-5.711-2.941l-1.305-.547c-.098-.047-.805-.301-1.574-.566l-1.586-.562c-.516-.234-4.195-1.258-7.285-2.023-.715-.18-3.684-.793-5.691-1.184l-.965-.184c-.137-.027-.437-.066-.664-.09l-.414-.039-.043.25c-.023.141-.141.398-.258.57s-.207.324-.195.332.27.047.578.09c.617.082 1.039.164 3.047.594 1.379.293 2.637.605 4.359 1.086l3.078.91 1.59.5a124.41 124.41 0 0 1 4.98 1.742c.102.047.648.254 1.211.469s1.211.469 1.434.57a31.73 31.73 0 0 0 1.09.465c.992.406 2.82 1.336 3.578 1.82l.965.598c.617.355 2.191 1.434 2.895 1.98 2.293 1.793 3.793 3.586 5.023 6.02.535 1.055.836 1.859 1.348 3.602.91 3.09 1.012 8.438.285 14.754a28.22 28.22 0 0 1-.312 2.176l-.219 1.402c-.258 1.73-.578 3.613-.773 4.543l-.496 2.367c-.445 2.121-.41 2.109.227-.094zm-79.594-1.191c.156-.609.469-1.223.871-1.727.176-.219.32-.414.32-.437.008-.25.152-.992.25-1.293a1.81 1.81 0 0 1 .434-.754c.094-.105-.039-.281-.437-.582-.359-.27-.609-.305-1.043-.152-.277.098-.383.172-.891.645-.648.605-1.008 1.652-1.016 2.969-.004 1.234.289 2.098.953 2.793l.348.367.043-.676c.023-.375.098-.891.168-1.152zm39.828.68c.32-.656.41-1.105.41-2.012-.004-.922-.133-1.523-.473-2.207-.395-.785-1.293-1.496-1.887-1.496-.27 0-.57.16-.914.484-.195.184-.211.242-.086.383.289.336.535 1.02.629 1.762.027.207.105.375.277.59.73.922 1.117 2 1.125 3.164l.004.496.359-.387a3.99 3.99 0 0 0 .555-.777zm-36.465.598c-.039-.168-.07-.625-.074-1.02-.004-.91.133-1.426.656-2.477.371-.746.383-.785.406-1.344l.027-.578-.211-.176c-.262-.219-.691-.406-.934-.406s-.504.141-.844.453c-.437.402-.656.906-.797 1.828-.164 1.09.141 2.277.828 3.219.203.277.93.887.988.828.012-.016-.008-.16-.047-.328zm32.695-.25c.406-.406.719-.961.93-1.641.117-.387.141-.605.141-1.246-.004-.844-.098-1.336-.359-1.852s-.91-1.012-1.305-1.012c-.25.004-.707.207-.93.422-.156.148-.172.203-.172.617 0 .547.145 1.02.531 1.73a4.13 4.13 0 0 1 .531 1.676c.055.465 0 1.395-.105 1.777l-.043.152.211-.137c.117-.074.375-.293.57-.488zm-39.191-.988c.168-.566.527-1.207.961-1.715.219-.25.309-.418.34-.621.094-.559.582-1.691.883-2.055l.172-.203-.246-.215c-.234-.207-.262-.215-.629-.184-.734.055-1.535.699-1.945 1.563s-.531 1.344-.527 2.121c0 .809.137 1.332.504 1.953l.234.398.055-.273c.027-.148.117-.496.199-.77zm45.629.391c.359-.715.457-1.93.219-2.797-.234-.848-.523-1.391-.988-1.848-.5-.488-.91-.711-1.395-.746-.332-.023-.367-.012-.586.191l-.238.215.195.242c.273.34.539.957.734 1.703.152.574.195.664.461.941.387.402.898 1.418 1.074 2.133l.133.555.121-.148c.063-.082.184-.281.27-.441zm-30.758-.887c.398-.914.527-1.359.609-2.07l.07-.637-.18-.187c-.426-.445-1.039-.762-1.75-.902l-.195-.039-.535 1.066c-.328.648-.52 1.102-.492 1.148s.16.203.297.352c.664.711 1.23 1.609 1.461 2.316l.078.242.184-.336c.102-.184.305-.613.453-.953zm16.238.391c.203-.457.723-1.18 1.281-1.793.137-.152.246-.293.238-.309s-.254-.504-.543-1.074l-.523-1.043-.254.039c-.414.059-.949.301-1.379.621l-.406.305.027.652c.039.848.234 1.504.77 2.598.414.84.418.844.496.664.043-.102.176-.398.293-.66zm1.547-1.27l1.914-1.656.883-.762a15.46 15.46 0 0 1 .539-.473c.184-.156.184-.164.125-.5a2.77 2.77 0 0 1-.031-.672l.035-.336-.16.148a8.62 8.62 0 0 0-.535.613 33.38 33.38 0 0 1-.766.934l-2.187 2.645a5.22 5.22 0 0 1-.32.367c-.359.383-.125.238.504-.309zm-20.062.051c-.176-.23-.508-.645-.742-.918l-2.25-2.707c-.125-.152-.336-.402-.469-.551l-.234-.27-.004.52c0 .285-.02.59-.043.676-.031.133.008.191.277.402a15.18 15.18 0 0 1 .695.594c.211.191.574.512.816.715l.465.406c.02.02.215.188.438.375s.613.527.867.758.473.422.484.422-.125-.187-.301-.422zm9.969-.59c1.742-1.258 2.922-2.18 2.922-2.277s-.27-3.121-.352-3.918l-.031-.332-.199.133a16.78 16.78 0 0 1-.633.383l-1.367.828-1.281.762-.348.195-.457-.277c-.254-.152-.699-.441-.992-.641s-.758-.492-1.031-.656-.578-.367-.676-.449-.191-.141-.203-.125c-.031.031.199 3.414.293 4.289.043.367.105.695.145.727a6.22 6.22 0 0 0 .504.285l.965.531c.832.477 1.891 1.043 1.965 1.051.039.004.387-.227.777-.508zm7.656-2.34c.289-.137.719-.285.953-.328l.426-.078-.023-.727c-.027-.695-.039-.746-.277-1.246-.492-1.02-1.285-1.711-2.297-1.992-.332-.094-1.418-.121-1.5-.039-.031.027-.055.434-.051.902 0 .801.012.891.207 1.477.148.449.938 3.266.941 3.371 0 .004.246-.238.547-.539.445-.441.645-.594 1.074-.801zm-14.187-.293c.121-.926.129-1.125.063-1.613l-.34-2.379c-.074-.074-.926.453-1.262.781-.77.75-1.023 1.316-1.066 2.379l-.031.719.43.078c.555.098 1.34.48 1.73.84.168.156.316.277.324.266s.078-.492.152-1.07zm-7.484-.711c.004-1.254-.27-1.504-2.078-1.906-.289-.066-.91-.109-1.836-.133-1.746-.039-2.207.031-2.992.453-.586.316-.883.57-1.059.914-.137.266-.133.27.16.145.848-.355 4.449-.254 6.41.18.637.141.949.473 1.098 1.164l.07.328.113-.277c.082-.199.113-.441.113-.867zm28.473.117c.273-.305.406-.371.949-.484 2.063-.437 5.402-.516 6.266-.148.129.055.25.086.266.066s-.051-.152-.148-.309c-.336-.527-1.238-1.055-2.074-1.215-.77-.145-3.078-.086-3.941.105-1.195.262-1.695.559-1.828 1.082-.105.418-.086 1.27.035 1.625l.105.305.117-.437c.063-.238.18-.504.254-.59zm-15.051-1.801c.137-.094.598-.371 1.023-.617s.855-.512.945-.586.438-.285.766-.473l.594-.34v-.437c0-1.328.164-2.777.488-4.297.066-.309.113-.566.109-.574s-.359.203-.785.469l-2.457 1.508-1.172.723c-.48.313-.402.332-1.008-.227l-1.082-.965-1.305-1.168c-.328-.301-.621-.547-.648-.547-.066 0-.012 1.797.113 3.879l.09 1.605c0 .082.148.211.453.406l1.078.699 1.02.664.742.473.352.227.215-.129a12.29 12.29 0 0 0 .469-.293zm-.219-5.195a17.54 17.54 0 0 0 .715-.441 49.54 49.54 0 0 1 1.121-.695l2.637-1.641c.066-.059.137-.23.16-.406a11.87 11.87 0 0 1 .41-1.812c.219-.699.398-1.684.523-2.848.148-1.434.152-1.465.074-1.414l-.574.355-.848.539a7.03 7.03 0 0 1-.555.336c-.117.063-.477.281-.801.492a37.54 37.54 0 0 1-.996.625c-.223.133-.676.414-1.008.629-.617.402-1.102.672-1.141.637-.016-.012-1.004-1-2.203-2.195s-2.195-2.156-2.215-2.137c-.07.066-.215 1.512-.215 2.16-.004.789.238 4.609.297 4.707.051.086 1.641 1.516 2.309 2.074.258.219.625.547.816.73a8 8 0 0 0 .59.516c.215.16.266.176.375.105.066-.043.305-.184.527-.316zm-9.773-.371c.258-.02.594-.062.75-.098l.871-.187a17.74 17.74 0 0 0 1.133-.289l.547-.164v-.316a10.83 10.83 0 0 0-.039-.734l-.039-.414-.164.102c-.09.055-.277.195-.422.309-.437.352-1.207.785-2.078 1.172l-1.18.539-.344.168.25-.027.715-.059zm20.105-.137a20.62 20.62 0 0 0-.746-.332c-.445-.176-1.574-.742-1.684-.844-.055-.051-.121-.09-.145-.09-.059 0-.641-.414-.992-.703-.164-.137-.312-.234-.328-.219-.043.047-.273 1.121-.273 1.301-.004.145.043.176.48.32a14.06 14.06 0 0 0 1.789.477 18.91 18.91 0 0 1 .621.133c.207.051 1.105.148 1.621.172.066.004-.086-.09-.344-.215zm-57.383-1.766c.215-1.203.93-3.969 1.27-4.914 1.055-2.941 2.137-4.59 3.77-5.742 2.098-1.48 4.461-2.473 6.781-2.844.867-.141 2.145-.254 2.84-.254h.457l.211-.328a5.66 5.66 0 0 1 .5-.648l.293-.324-.297-.051c-.543-.09-2.668-.051-3.441.066-1.277.195-3.18.648-3.703.883-.105.047-.543.223-.977.395-1.051.414-1.504.648-2.355 1.223-1.594 1.07-2.852 2.586-3.742 4.523-.641 1.387-.965 2.414-1.367 4.301-.32 1.527-.5 3.191-.484 4.516.008.516.008.523.063.227l.184-1.027zm94.332.141a25 25 0 0 0-.355-3.5c-.137-.77-.457-2.129-.551-2.336-.031-.066-.16-.43-.289-.809a13.05 13.05 0 0 0-1.297-2.801c-.605-.992-.793-1.227-1.684-2.105-.754-.742-1.863-1.527-2.746-1.937-1.73-.805-3.293-1.285-5.285-1.625-.855-.148-1.18-.172-2.305-.176-.719-.004-1.414.016-1.543.043l-.238.051.371.398c.203.215.418.504.48.641l.105.242.945.043c1.914.082 4.074.469 5.043.906.141.063.449.191.688.285.813.32 1.43.633 2.324 1.176 1.211.73 2.066 1.406 2.531 2.004.203.266.41.516.453.559.27.277 1.203 2.09 1.566 3.043.32.848.848 2.484 1.027 3.207a53.74 53.74 0 0 1 .664 3.285c.039.281.09.512.105.512s.012-.496-.012-1.105zm-57.699-1.918c1.359-.172 2.434-.531 3.223-1.07l.43-.293-.012-.777c-.008-.426-.023-.789-.039-.801-.023-.023-.805.383-2.145 1.109-1.977 1.07-2.32 1.211-3.98 1.641l-.836.215.336.039c.934.102 1.879.082 3.023-.062zm23.895.055l.309-.039-.215-.055c-1.148-.289-1.988-.535-2.441-.715-.707-.277-1.812-.898-2.82-1.586-.445-.305-.812-.531-.82-.504s-.086.379-.176.785-.176.773-.195.816c-.051.137.75.594 1.422.813 1.504.488 3.422.676 4.938.484zm-47.836-3.973c.141-.687.199-.906.605-2.191.266-.836.398-1.074 1.145-2.078.598-.801.973-1.242 2.004-2.344.332-.359.605-.68.605-.711s-.023-.043-.051-.027-.383.051-.789.074l-.734.039-.32.344c-.363.395-1.254 1.66-1.594 2.258a16.28 16.28 0 0 0-.93 2.113c-.258.762-.383 1.891-.41 3.645l-.023 1.617.176-1.023.316-1.715zm69.219.5c-.102-1.781-.305-2.863-.73-3.891-.453-1.094-1.5-2.738-2.273-3.566l-.27-.289-.621-.035c-.344-.016-.703-.051-.801-.07-.227-.047-.199.008.305.539.629.664.984 1.063 1.363 1.523l.586.711c.629.758 1.184 1.727 1.43 2.504.234.734.566 2.246.895 4.094l.082.465.051-.402c.031-.234.023-.887-.016-1.582zm-32.961.125a18.96 18.96 0 0 1 .859-.527c.195-.109.434-.266.527-.34s.422-.289.73-.473a39.03 39.03 0 0 0 1.219-.758c.66-.426.664-.43.703-.715.059-.437.043-2.867-.023-3.848-.066-.965-.195-2.293-.227-2.328-.02-.023-.609.332-1.629.992l-1.32.836c-.324.203-.676.43-.777.504-.258.188-1.148.711-1.215.711-.027 0-.301-.273-.602-.602l-.984-1.078-1.082-1.168c-.355-.387-.664-.699-.687-.699s-.125.23-.223.512-.254.684-.348.887-.383.992-.648 1.742l-.562 1.559c-.117.254-.402 1.262-.402 1.414 0 .082.801.922 2.289 2.406l2.285 2.281.809-.492 1.309-.816zm10.676-.684c.898-.105 1.313-.191 2.301-.48l.715-.207-.465-.016c-1.855-.055-3.008-.223-4.203-.605l-.684-.219-1.121.227c-1.125.23-1.273.293-1.273.535 0 .125.195.223.996.5.98.344 2.285.434 3.734.266zM55 57.945c.27-.035.602-.094.738-.133 1.449-.41 1.805-.66 1.137-.793l-1.656-.34-1.328-.27-.844.27c-.465.152-1.082.316-1.375.367-.504.086-1.82.199-2.687.23l-.406.016.934.211a17.73 17.73 0 0 0 2.305.41c.844.098 2.531.117 3.184.031zm2.227-2.211c.027-.535.035-.984.023-.996s-.164.027-.336.094c-.852.324-2.289.324-3.754.008-2.363-.512-3.055-.727-4.645-1.422l-1.918-.836c-.84-.363-.875-.391-1.371-.898-.406-.418-.543-.523-.691-.523-.105-.004-.648-.086-1.215-.187a560.21 560.21 0 0 0-6.816-1.078c-.133-.02-.242.016-.375.125-.102.082-.465.238-.809.344-.602.188-.957.406-.875.539.035.059.816.32 3.273 1.102a50.15 50.15 0 0 0 1.93.559l1.867.508c.426.121 1.676.441 2.77.707l2.59.648c.332.086 1.313.313 2.18.504l2.465.563c.484.121 1.703.387 2.707.59l2.754.59c.191.063.203.008.246-.937zm15.66.598l1.68-.348c1.164-.242 2.277-.492 2.582-.582.289-.086.621-.164 2.395-.562l2.023-.477 2.336-.59 2.137-.551c.098-.039.867-.254 1.711-.477s1.914-.531 2.375-.68l2.133-.684c.707-.23 1.309-.445 1.328-.48.078-.121-.258-.348-.719-.484-.492-.145-1.023-.391-1.094-.5-.02-.035-.129-.051-.238-.031l-2.871.438-2.957.477c-.219.055-.402.188-.844.605-.547.52-.586.543-1.461.902-.496.203-1.195.508-1.555.676-1.082.508-1.344.602-2.23.82l-1.598.398c-.996.25-1.926.43-2.637.508-1.004.109-2.406-.105-2.824-.437-.09-.066-.172-.125-.187-.125s-.047.105-.066.234-.109.652-.195 1.16-.141.949-.125.977.098.027.172-.004.402-.113.73-.184zM65.34 51.98c.301-.199.781-.508 1.07-.684s.703-.441.918-.586c.355-.242 1.109-.73 2.023-1.309.352-.223.371-.246.336-.437-.18-.953-.156-2.777.047-3.844.051-.254.082-.469.074-.477s-.18.035-.383.102-.523.164-.711.219-.398.129-.469.16c-.277.129-2.301.684-2.316.637a14.35 14.35 0 0 0-.414-.711 32.26 32.26 0 0 1-.664-1.164c-.148-.277-.285-.504-.305-.5-.113.008-1.629 2.289-2.273 3.422-.586 1.027-.902 1.641-.902 1.742 0 .039.215.301.48.586l1.293 1.402 1.199 1.301c.211.227.387.434.387.457 0 .086.059.055.609-.316zM55.238 52a5.07 5.07 0 0 0 .902-.312c.961-.457 1.582-1.07 2.262-2.234.258-.437.59-1.258.531-1.305-.02-.016-.645.098-2.637.484-.445.086-1.789.309-2.988.5l-2.582.406c-2.016.332-4.344.629-4.902.629-.145 0-.262.02-.262.047s.117.113.262.195.457.289.688.457c.473.348.551.367 2.332.695 1.945.355 3.051.504 4.094.559.695.031 1.828-.027 2.301-.121zm23.371-.316a42.6 42.6 0 0 0 3.727-.621c.633-.145.789-.203 1.223-.488l.801-.512.301-.191-.207-.035c-.117-.02-.504-.066-.863-.102s-.82-.094-1.027-.133-1.086-.16-1.961-.281c-1.488-.203-3.711-.559-4.699-.746-1.867-.355-4.223-.77-4.246-.746-.039.039.195.758.383 1.184.387.875 1.316 1.906 2.059 2.285 1.008.52 2.34.633 4.512.387zm-44.43-1.391c1.148-.195 1.777-.465 1.926-.824.152-.367-.539-1.918-.855-1.918-.176 0-1.234.719-1.723 1.168-.516.477-1.023 1.141-1.148 1.5l-.07.195h.582c.32 0 .902-.055 1.289-.121zm61.547.059c0-.105-.187-.441-.434-.777-.504-.68-1.832-1.781-2.375-1.973-.187-.066-.211-.055-.375.141-.207.246-.551 1.098-.598 1.477-.027.223-.008.297.117.426.27.289.93.523 1.895.672.68.105 1.77.129 1.77.035zm-62.027-2.23a10.79 10.79 0 0 1 .598-.422 4.3 4.3 0 0 0 .398-.285c.066-.062.141-.109.164-.109s.09-.281.148-.625a45.3 45.3 0 0 1 .313-1.508l1.086-4.527c.75-2.82 1.262-4.352 2.098-6.262.02-.043-.078-.086-.242-.105a4.23 4.23 0 0 1-.574-.141c-.16-.055-.316-.078-.344-.051s-.234.559-.465 1.176l-.484 1.234c-.031.066-.332.934-.668 1.93-.414 1.238-.719 2.285-.957 3.309l-.508 2.113c-.086.344-.32 1.438-.527 2.43l-.406 1.945c-.043.191.004.176.371-.102zm61.031.164c0-.082-.266-1.34-.371-1.758-.035-.137-.176-.77-.309-1.402l-.805-3.582a49.21 49.21 0 0 0-1.625-5.07l-.535-1.461a12.43 12.43 0 0 0-.258-.613c-.145-.305-.187-.355-.293-.32-.398.129-.766.219-.891.219-.141 0-.141.004-.051.176.152.293.461 1.094.855 2.219.469 1.328.594 1.746 1.012 3.328l.438 1.621c.129.461.48 1.992.813 3.547.48 2.273.41 2.078.91 2.414a15.22 15.22 0 0 1 .723.523c.32.25.387.277.387.16zm-27.852-3.117c.137-.055.695-.227 1.246-.387 1.734-.508 1.828-.535 1.891-.613.074-.09.125-.359.41-2.281l.242-1.57c.031-.133-.047-.117-.582.125-.352.152-1.91.75-2.309.879-.172.055-.238.121-.301.305-.176.504-.781 2-.801 1.98s.277-1.227.48-1.93l.125-.43-.316-.586c-.176-.32-.328-.574-.336-.562s-.164.305-.336.652a9.19 9.19 0 0 1-1.187 1.852c-.172.203-.312.406-.312.445s.145.324.324.625l.656 1.152.336.598.258-.078c.145-.043.375-.125.512-.176zm1.414-4.391c.215-.102.852-.355 1.414-.566s1.059-.41 1.098-.441c.113-.094.223-.926.219-1.664 0-.602-.141-1.613-.246-1.797-.039-.07-.984.469-1.504.855-.117.09-.375.254-.566.367l-.352.207-.039.473c-.051.547-.297 1.789-.469 2.355-.141.461-.125.469.445.211zm-.703-.703l.145-.535-.141-.613c-.078-.34-.215-.785-.309-.988l-.168-.375-.039.531c-.023.289-.098.75-.164 1.023l-.121.5.293.543c.16.297.309.52.328.496s.102-.285.176-.582zm-17.191-1.605a2.49 2.49 0 0 1-.273-.434c-.062-.137-.129-.25-.145-.254s-.223.066-.457.148l-.422.148.352.203c.359.203.773.359.984.367.102.004.098-.016-.039-.18zm1.535-.891c-.07-.078-.199-.262-.297-.406l-.168-.27-.23.137c-.391.234-.422.281-.254.375.195.105.801.297.953.301.117.004.117 0-.004-.137zm17.895-1.203c.457-.305.844-.586.859-.625s-.082-.293-.219-.562c-.473-.937-1.402-1.762-2.387-2.125l-.355-.133-.062.324c-.074.402-.098.543-.266 1.633l-.133.875.371.371c.27.27.41.473.508.734a2.34 2.34 0 0 1 .133.445c0 .059.121.008.359-.148l1.191-.789zm-16.711-.09l-.168-.25-.172.109c-.25.16-.246.223.035.309.453.137.488.117.305-.168zm7.816-.676l.777-.168-.02-.277c-.02-.281-.145-1.07-.262-1.656l-.066-.32-.57.148c-.316.082-.82.238-1.121.348l-.551.195.246.254c.426.445.68 1.043.656 1.547-.008.137-.027.137.91-.07zm6.039-.055c0-.023.086-.648.188-1.383s.176-1.371.16-1.41-.219-.09-.453-.113c-.453-.051-2.32-.055-2.367-.008-.016.016.016.418.07.895a15.08 15.08 0 0 1 .098 1.219c0 .23.027.363.078.375.043.012.344.055.668.09s.789.137 1.027.219c.52.176.531.18.531.117zm-4.078-.293c.379-.051.848-.094 1.047-.094.234 0 .367-.027.391-.082.016-.043-.016-.484-.074-.98l-.133-1.164c-.016-.145-.043-.266-.059-.266-.082 0-1.539.266-1.937.352-.379.082-.449.117-.41.203.023.055.113.52.199 1.035.176 1.066.18 1.086.242 1.086a19.46 19.46 0 0 0 .734-.09zm-23.754-1.5c.441-.184.762-.609.848-1.129.105-.641-.402-1.461-1.035-1.672-.453-.148-1.07-.098-1.449.125-.719.422-.914 1.445-.406 2.145.441.609 1.313.836 2.043.531zm51.215-.059c1.223-.602 1.18-2.219-.07-2.742-.34-.141-.945-.113-1.312.059-.531.246-.957.859-.961 1.383-.008 1.156 1.23 1.844 2.344 1.301zm-15.078-.336c-.07-.07-1.035-.66-1.293-.793-.102-.055-.363-.164-.586-.246L73 32.176l.137.191c.141.199.48.379 1.508.809.598.246.734.289.633.191zm-22.77-.457c.313-.316 1.098-.746 1.598-.875.613-.16 1.316-.75 2.051-1.73.555-.738.988-1.23 1.645-1.859 1.141-1.105 1.977-1.687 3.008-2.102l.531-.211-.375.035c-1.699.145-4.578 1.078-6.691 2.172-.711.367-1.652.895-1.773.992l-.652.508c-.895.699-1.371 1.395-1.414 2.078-.039.66.176.957.863 1.203a1.44 1.44 0 0 0 .641.09c.258-.027.348-.074.57-.301zm-16.035-1.016c0-.164.191-.484.449-.75l.246-.258-.332-.078a10.84 10.84 0 0 1-.582-.156c-.75-.238-1.355-.406-3.074-.852l-2.496-.648a34.59 34.59 0 0 0-1.602-.367l-1.012-.207-.398.18c-.359.164-.434.176-.707.125-.566-.105-2.965-.375-4.59-.512-2.371-.195-6.051-.195-8.121.004l-1.477.156c-.008.008 1.141.016 2.551.016 2.516.004 3.68.051 5.805.246 1.891.168 4.332.516 5.414.766.137.031.992.211 1.898.402 2.953.613 6.254 1.43 7.344 1.813.707.246.684.242.684.121zm55.801-.152c.332-.121 2.539-.699 4.512-1.184a117.43 117.43 0 0 1 2.551-.551l2.367-.5c2.656-.605 8.207-1.055 12.074-.977 1.652.035 1.578 0-.402-.191-1.719-.164-5.57-.148-7.719.031-1.699.145-3.414.328-4.359.473l-.559.086-.41-.18-.41-.176-.742.148c-1.008.207-2.332.52-2.484.59-.066.031-1.02.281-2.113.559l-2.117.563c-.07.031-.449.148-.84.254l-.719.199.238.254c.129.141.281.387.336.547.117.324.063.32.797.055zm-53.496-1.277c0-.16-.781-1.641-.934-1.77-.023-.02-.195-.242-.387-.5s-.625-.734-.961-1.062c-.555-.543-.867-.801-1.586-1.32-.273-.199-1.734-1.047-1.734-1.008 0 .016.227.246.508.512 1.887 1.797 3.027 3.18 3.695 4.496.164.328.32.621.344.656.063.082 1.055.082 1.055-.004zm51.719-.453c.25-.52 1.168-1.898 1.477-2.223a11.76 11.76 0 0 0 .508-.59c.184-.223.813-.879 1.402-1.457s.984-1.004.879-.937a18.74 18.74 0 0 1-.684.391c-1.379.766-2.937 2.125-3.754 3.273-.48.68-1.133 1.84-1.133 2.023 0 .02.242.031.535.027l.531-.012zm-27.754-.816c.09-.004.355-.141.594-.305.547-.387 1.496-.887 2.02-1.07a19.39 19.39 0 0 1 1.148-.348l.75-.203-.43-.004c-.234 0-.723.039-1.082.094-1.355.199-2.383.559-3.043 1.059-.422.324-1.02.848-1.02.898 0 .02.203.004.453-.039s.523-.082.609-.082zm-50.215-.605c-.043-.016-.125-.02-.187 0s-.023.027.078.027.152-.012.109-.027zm103.133 0c-.043-.016-.113-.016-.156 0s-.008.031.078.031.121-.016.078-.031zm0 0" fill="#0b99d2"/><path d="M83 103.938c-1.047-.102-2.961-.516-4.418-.957-2.383-.727-4.82-1.992-6.621-3.437-1.301-1.043-2.937-2.762-4.797-5.027l-1.094-1.328c-.062-.07-.199-.023-.754.254-1.336.668-5.176 2.289-7.402 3.125-2.551.957-5.859 1.977-7.84 2.418-2.164.48-3.594.664-5.543.711-4.691.105-8.73-.937-10.547-2.73-.828-.812-1.652-2.426-1.41-2.758.07-.098.105-.086.418.125.184.129.535.395.773.59 1.031.848 1.875 1.254 3.422 1.645 2.148.547 5.293.707 7.316.383 2.672-.434 7.07-1.789 9.703-2.996 1.605-.734 9.457-4.672 9.605-4.816.074-.07-.105-.602-.824-2.434-.773-1.973-1.27-3.336-2.227-6.133-.809-2.352-1.301-3.707-1.371-3.777-.09-.09-.664.68-2.215 2.953-.895 1.309-1.172 1.605-1.5 1.605-.484 0-1.336-.473-2.391-1.324-.262-.211-1.223-1.125-2.141-2.035-1.812-1.797-1.762-1.762-1.984-1.305-.176.359-.266.988-.262 1.801 0 .688.02.824.168 1.207.094.234.16.438.145.449-.039.043-.637-.305-1.062-.617-.652-.484-1.125-1.328-1.379-2.465-.145-.629-.172-.648-.465-.324-.305.336-.582.902-.754 1.543-.102.375-.129.668-.129 1.277 0 .438-.023.797-.051.797-.109 0-.602-.43-.82-.719-.488-.648-.773-1.398-.922-2.441-.043-.285-.105-.535-.141-.555s-.168.074-.297.215a4.31 4.31 0 0 0-.93 2.465v.375l-.312-.309c-.762-.746-1.098-2.09-.844-3.398.113-.598.305-1.098.633-1.672a5.66 5.66 0 0 0 .438-1.059c.219-.777.453-1.23.82-1.602.332-.332.977-.695 1.508-.852.734-.219 3.52-.184 4.668.055a4.71 4.71 0 0 1 .867.289c.477.219.93.672 2.191 2.16 1.191 1.41 2.051 2.379 2.109 2.379.074 0 .656-1.039.871-1.562.156-.383.195-.59.246-1.27.063-.906.117-1.145.391-1.684.238-.477 1.055-1.293 1.59-1.586.703-.387.656-.301.578-1.152-.094-1.043-.172-1.285-.445-1.379-.117-.043-.363-.078-.551-.078-.742-.008-2.473-.336-3.703-.699-.348-.102-1.246-.453-2-.781-2.25-.977-4.551-2.164-8.129-4.191-1.984-1.125-3.828-2.055-4.75-2.395-2.211-.812-4.703-1.086-6.121-.676-.84.246-1.883.789-2.367 1.23-.387.355-.488.629-.66 1.781-.078.551-.164 1.02-.184 1.039s-.039 0-.039-.047c0-.34-.191-2.051-.242-2.184-.121-.324-1.195-.914-1.969-1.082-.605-.133-2.168-.187-2.879-.098-.664.082-1.371.324-2.059.699-1.516.828-2.977 2.059-4.223 3.563l-.562.68-.168 1.063c-.187 1.184-.277 1.594-.344 1.594-.082 0-.129-.547-.129-1.527v-.953l-.57-.383c-.949-.637-1.84-1.039-2.676-1.215-.559-.117-1.59-.113-2.105.008-.965.223-2.148 1-2.961 1.941-.969 1.129-2 2.965-2.727 4.871-.867 2.281-1.867 6.285-2.031 8.156-.105 1.195-.004 1.953.598 4.508.434 1.828.406 1.711.367 1.711-.164 0-1.27-3.176-1.855-5.324-1.172-4.289-2.18-9.988-2.41-13.598-.094-1.445-.098-6.184-.004-7.219.172-1.988.563-4.297.977-5.785.266-.957.973-3.109 1.465-4.465 2.34-6.434 7-12.375 13.5-17.211.531-.395 1.043-.805 1.141-.906l.18-.191-.363-.047c-.855-.105-5.859.047-7.332.227-.27.031-.504.047-.52.031-.062-.062 1.793-.461 2.91-.621 3.926-.57 9.965-.512 14.004.133l1.023.164.5-.176.5-.172.652.133c.359.074 1.691.398 2.957.723l3.875.996c.863.223 1.641.402 1.73.402.199 0 .18-.09-.141-.684-.66-1.227-1.449-2.184-3.375-4.105-1.168-1.164-1.562-1.594-1.477-1.609.176-.035 1.605.711 2.496 1.305 1.758 1.172 2.875 2.383 3.832 4.168.211.391.496.848.629 1.02l.492.625c.699.875.543 1.879-.395 2.523-.211.145-.434.324-.496.395-.73.855-2.309 6.016-3.437 11.238l-.328 1.523.293.32c.426.465.875 1.434.879 1.895 0 .363-.203.316 4.758 1.121l3.566.59c.113.027.172-.02.32-.25.098-.156.211-.363.25-.461s.137-.195.211-.215.934-.133 1.91-.258a224.13 224.13 0 0 0 10.332-1.617c.895-.164 1.367-.195 1.555-.094.098.051.223-.039.789-.578.887-.836 3.258-3.262 4.305-4.402 1.492-1.625 2.191-2.895 2.461-4.477.125-.715.125-1.09.012-1.633s-.273-.84-.613-1.152c-.371-.344-.734-.422-1.941-.41-1.125.012-1.98.121-3.336.422-1.379.309-1.559.391-1.832.84-.285.473-.828.969-1.402 1.277-.742.398-1.391.918-2.516 2.012-1.668 1.617-2.078 1.91-2.586 1.863-.398-.039-1.094-.371-1.152-.547s.105-.344 1.027-1.09c1.719-1.379 2.672-2.77 2.781-4.035.051-.613-.07-.91-.367-.91-.223 0-1.125.281-1.465.457-.109.059-.199.152-.199.211s.188.297.418.531l.422.43-.219-.039c-.121-.023-.527-.113-.906-.203-.687-.164-.887-.164-1.109 0-.187.137-.09.352.406.879l.445.477-.336-.039c-.492-.059-.777-.137-1.316-.352-.406-.168-.504-.184-.637-.125-.086.039-.156.121-.156.184 0 .203.359.691.711.961.188.145.332.273.32.285-.043.043-.887-.121-1.242-.242-.203-.07-.57-.219-.812-.332a2.93 2.93 0 0 0-.812-.246 1.72 1.72 0 0 1-1.172-.59c-.488-.523-.773-1.293-.711-1.914.063-.578.23-.703 1.211-.891 1.078-.207 1.707-.488 2.242-.996.461-.441.547-.629.441-.992-.047-.156-.082-.5-.082-.758 0-.406.031-.531.203-.883.551-1.117 2.73-2.582 5.496-3.691 2.113-.848 4.5-1.465 5.621-1.453.746.004.75.043.039.484-1.129.703-2.184 1.625-3.047 2.66-.492.59-.785 1.09-.73 1.23.035.094.305.063 1.055-.113 1.707-.398 2.137-.578 2.777-1.172 1.309-1.219 2.445-1.684 4.672-1.91.605-.062 1.266-.016 1.457.105.031.02-.543.332-1.277.695-1.395.691-1.922 1.008-1.922 1.16 0 .051.008.09.016.09s.344.043.738.09a25.3 25.3 0 0 0 1.527.133l.801.043.777-.266c.699-.238.848-.266 1.465-.281.375-.012.57-.008.434.008-.898.09-2.086.387-2.086.52 0 .027.176.098.391.152a9.32 9.32 0 0 1 .789.262l.402.16.41-.156c.734-.281 1.582-.586 1.945-.699.395-.125.398-.191.016-.234-.137-.016-.012-.016.277-.004.926.035.828.133-.637.641-1.035.359-1.602.598-1.598.672 0 .105.109.207.777.703a7.49 7.49 0 0 0 2.105 1.16c.949.363 1.895.887 2.309 1.281.453.43.516.414-1.07.227-.867-.105-1.156-.121-1.211-.066s-.035.254.094.836c.09.422.184.918.203 1.105.055.434.152.563 1.18 1.59.863.859 1.293 1.414 1.125 1.449-.051.012-.426-.094-.828-.23-1.098-.371-1.191-.391-1.258-.262-.055.098-.172.824-.305 1.891l-.055.422.313.512c.168.281.402.707.516.945.352.723.289.77-.434.313-.293-.187-.559-.34-.582-.34-.105 0-1.512 5.191-1.512 5.578 0 .199 0 .199.543.324 1.906.438 7.281 1.352 10.102 1.719 2.461.32 2.695.383 2.766.707.023.102.109.246.195.328.219.207.578.184 2.973-.195 2.859-.453 3.273-.551 3.371-.801.023-.055.141-.379.258-.723.254-.723.484-1.113.82-1.387l.242-.191-.18-.797c-1.242-5.453-2.02-8.312-2.824-10.387-.48-1.246-.66-1.531-1.234-1.977-.684-.527-.871-.871-.836-1.539.031-.516.242-1.008.547-1.262.25-.215.457-.516.875-1.289 1.047-1.918 2.109-3.094 3.855-4.262.719-.48 1.855-1.094 2.77-1.492.207-.09.332-.125.281-.078s-.473.426-.934.836c-1.617 1.438-3.047 3.012-3.93 4.32-.461.688-.781 1.344-.695 1.43.078.078.832-.086 2.895-.633 2.316-.617 5.215-1.34 5.844-1.457.492-.09.563-.09.813 0 .633.23.633.23 1.816.066 2.797-.395 4.391-.5 7.574-.5 3.191 0 5.184.145 7.242.523.844.156 2.047.441 2.012.477-.016.016-.277 0-.582-.035-1.59-.187-7.141-.34-7.48-.207-.145.055-.137.066.246.398.219.188.816.66 1.332 1.051 5.262 3.988 9.242 8.652 11.766 13.785.723 1.473 1.121 2.457 1.77 4.359.879 2.578 1.406 4.586 1.676 6.379.879 5.871.441 12.594-1.34 20.414-.523 2.316-1.312 5.129-1.832 6.539-.27.73-.734 1.77-.777 1.727-.012-.012.246-1.187.574-2.605l.598-2.578-.105-.684c-.453-2.91-1.434-6.633-2.387-9.059-1.125-2.867-2.949-5.191-4.664-5.953-1.012-.445-2.277-.488-3.379-.113-.949.32-2.43 1.125-2.617 1.422-.055.09-.102.512-.141 1.348-.027.664-.066 1.227-.082 1.242-.074.07-.145-.191-.242-.867-.164-1.109-.328-1.801-.492-2.082-.324-.559-2.23-2.406-3.32-3.223-.621-.469-2.281-1.309-2.816-1.426-.281-.062-.812-.09-1.742-.086-1.496 0-1.637.023-2.59.465-.988.457-1.043.531-1.152 1.66-.039.402-.094.793-.117.859-.047.117-.055.117-.113-.039a4.4 4.4 0 0 1-.137-.684c-.141-1.008-.227-1.227-.656-1.641-.68-.664-1.633-1.105-2.836-1.312-.832-.145-2.457-.098-3.355.094-2.027.438-3.82 1.215-7.031 3.055-1.965 1.121-3.762 2.098-5.086 2.762-1.711.855-3.945 1.859-4.625 2.082-1.375.445-2.848.734-4.18.816-.492.031-.941.074-1 .098-.164.066-.242.422-.277 1.285l-.035.785.488.039c1.355.109 2.246.445 2.887 1.098.621.633.898 1.371 1.055 2.836.035.363.129.742.25 1.059.199.516.738 1.469.844 1.496.063.016.512-.488 2.258-2.531 1.566-1.832 1.836-2.039 2.941-2.277 1.082-.23 3.77-.266 4.574-.059 1.234.32 2.051 1.152 2.359 2.398a4.24 4.24 0 0 0 .293.777c.531 1.02.613 1.199.711 1.582.156.613.188 1.617.066 2.184-.125.582-.449 1.246-.812 1.652l-.277.309-.004-.34c-.008-.516-.16-1.086-.465-1.711-.32-.66-.672-1.082-.805-.973-.051.039-.105.262-.129.52a3.87 3.87 0 0 1-.473 1.633c-.215.445-.414.746-.699 1.055-.484.527-.605.531-.605.012 0-.617-.129-1.609-.262-1.992-.238-.707-.746-1.465-.934-1.391-.059.02-.113.152-.141.336-.117.832-.629 1.926-1.152 2.445-.293.293-1.211.871-1.277.801-.012-.012.035-.187.105-.395.195-.59.223-1.719.055-2.414-.145-.621-.277-.914-.41-.914-.051 0-.883.777-1.852 1.727-2.238 2.195-2.887 2.719-3.852 3.109-.48.195-.797.168-1.062-.086-.102-.098-.508-.652-.906-1.234-.883-1.301-2.199-3.082-2.277-3.082s-.008.621.242 2.004c.598 3.285 1.293 6.625 1.398 6.734.031.031.445-.109.957-.328 2.262-.961 4.836-1.625 7.633-1.973 1.699-.211 5.348-.176 7.012.07 1.316.195 3.055.688 4.32 1.223 3.258 1.375 5.289 3.609 6.105 6.711.336 1.273.316 3.168-.047 4.602-.187.75-.809 2.008-1.355 2.77-1.992 2.75-5.523 4.844-8.93 5.297-.738.098-3.445.113-4.398.023zm1.176-.746c.027-.414.031-.766.016-.781s-.316-.094-.672-.176-.828-.207-1.047-.277-.422-.105-.445-.082c-.07.082-.258 1.641-.203 1.691s.594.152 1.516.285c.883.129.785.203.836-.66zm2.602.664c.074-.031.094-.102.078-.242-.016-.109-.043-.355-.062-.551l-.035-.352-.387-.031c-.211-.016-.703-.078-1.094-.133s-.715-.094-.723-.082-.035.348-.062.754l-.043.738 1.109-.027c.613-.016 1.16-.047 1.219-.074zm1.801-.324l.691-.16-.043-.32c-.023-.176-.047-.32-.055-.328s-.477.008-1.043.023l-1.035.031.023.289a18.09 18.09 0 0 0 .059.496l.035.211.34-.039c.188-.023.648-.113 1.027-.203zm-7.074-.66l.152-.898.047-.258-.68-.328a22.64 22.64 0 0 1-1.148-.598c-.262-.148-.48-.262-.484-.254s-.059.129-.121.266c-.172.371-.777 1.984-.754 2.008.07.07 2.523.684 2.781.695.078.004.121-.125.207-.633zm8.656.203c.527-.187 1.148-.508 1.148-.598 0-.152-.113-.156-.82-.02-.883.168-.98.203-.953.348.09.449.098.465.203.422.059-.023.25-.094.422-.152zm-11.727-1.027c.309-.789.613-1.652.613-1.73 0-.043-.203-.23-.453-.418s-.73-.602-1.07-.918-.637-.578-.656-.578-.203.414-.402.918l-.547 1.371-.184.453-.57-.035-.566-.039.188.117a11.56 11.56 0 0 0 1.398.734c1.059.453 1.465.586 1.77.57l.309-.012zm10.852.277c1.773-.199 3.406-.77 4.762-1.664 1.066-.707 2.418-2.309 3.023-3.59.59-1.242.895-3.145.719-4.484-.137-1.059-.484-2.297-.848-3.02a9.97 9.97 0 0 0-1.672-2.332c-.906-.895-.895-.895-.535-.125.438.934.668 1.668.75 2.379.156 1.371-.113 2.809-.785 4.215-.742 1.563-2.418 2.992-4.301 3.668-1.965.711-4.301.777-6.867.195-.363-.082-.777-.168-.926-.191a8.29 8.29 0 0 1-1.41-.43l-.762-.336c-.895-.387-2.312-1.293-3.113-1.984-.656-.57-1.812-1.699-2.086-2.039-.137-.172-.383-.465-.543-.656-1.012-1.164-2.332-3.469-2.855-4.988-.355-1.035-.512-1.727-1.414-6.371l-.523-2.707c-.352-1.785-.484-3.25-.539-5.91-.074-3.613.211-7.168.699-8.652.152-.465.578-2.172.957-3.828l.254-1.059c.078-.262.461-2.539.625-3.711l.137-.973-.156-.211c-.211-.281-.633-1.113-.816-1.609-.41-1.102-.414-2.906-.008-3.762.113-.242.133-.301.625-2.09l.617-2.148c.688-2.203 1.223-5.609 1.125-7.187-.09-1.449-.359-2.746-.75-3.594-.125-.262-.246-.492-.27-.512a2.34 2.34 0 0 1-.246-.32c-.727-1.07-2.113-1.996-3.57-2.391-1.32-.355-3.172-.566-4.316-.496-1.641.102-4.402.605-5.434.992-.395.145-1.262.406-1.355.406-.051 0-.09-.023-.09-.051s.18-.41.398-.855c.336-.672.492-.906.961-1.434.309-.348.707-.754.883-.902l.547-.48c.387-.344-.824.453-1.363.898-.672.559-1.871 1.82-2.27 2.387-.473.676-1.078 1.313-1.52 1.598-.215.141-.375.285-.375.348 0 .215-.254.777-.422.938-.359.348-.937.449-1.531.27-.219-.066-.348-.074-.523-.023-.305.082-.734.027-1.195-.16-.258-.105-.375-.125-.375-.074 0 .133-.855.926-1.168 1.086-.5.254-1.082.43-1.664.504-1.004.129-1.195.313-1.125 1.086.051.547.25.977.652 1.391.516.531.824.629 1.508.473.254-.055.609-.168.789-.25.457-.199 2.098-1.113 2.453-1.367s1.105-.633 1.699-.859l.559-.227c.16-.074.645-.18.824-.184.191 0 .289.371.293 1.152.004.609 0 .633-.262 1.176-.648 1.344-1.289 2.148-2.359 2.938-.27.199-.559.426-.641.508a3.5 3.5 0 0 1-.391.309c-.129.086-.234.18-.234.203 0 .086.777.395 1.059.422.48.047.797-.211 2.555-2.062.738-.773 1.32-1.223 2.133-1.637.559-.285.785-.441 1.137-.793.422-.422.449-.465.586-.969a2.82 2.82 0 0 0 .109-1.062c-.035-.641-.16-.941-.5-1.187-.156-.113-.215-.187-.16-.203l.801-.258c.395-.125 1.023-.312 1.402-.414l.902-.246c1.98-.57 5.52-.742 6.086-.297.086.066.199.117.25.117s.395.148.754.328c.563.277.738.41 1.223.887.477.473.613.652.859 1.152.164.328.305.691.313.813a5.8 5.8 0 0 0 .148.684c.188.684.281 1.695.207 2.285-.07.555-.23 1.203-.309 1.254-.051.035-.121.461-.508 3.078-.102.711-.219 1.297-.273 1.379-.098.145-.215.629-.344 1.406-.09.539-.086 2.109.004 2.582.039.203.074.605.082.898a18.92 18.92 0 0 0 .129 1.43c.09.723.113 1.371.117 3.258l.004 2.355-.141.113c-.098.082-.137.18-.137.344 0 .371-.301 2.625-.441 3.297-.07.344-.195.875-.277 1.184s-.18.742-.215.965c-.152.91-.262 1.242-.437 1.371-.156.109-.187.203-.336.977-.223 1.172-.332 2.195-.383 3.594-.035.922-.062 1.207-.137 1.293-.109.125-.105.234.133 2.598.184 1.781.195 2.063.09 2.164-.094.094-.008.855.387 3.414.316 2.039.352 2.383.254 2.488-.051.055-.008.441.176 1.582.074.449.27 1.215.605 2.383.105.379.137.57.094.625-.07.082.254 1.172.484 1.629.242.488.813 2.086.813 2.281 0 .227 1.102 2.598 1.57 3.379.164.27.297.551.301.621.008.289 1.285 2.168 2.012 2.961 1.059 1.152 2.883 2.965 3.699 3.672.48.418 1.699 1.25 2.215 1.508a21.12 21.12 0 0 0 2.34.93c.543.16 1.871.398 2.676.484l.84.09c.438.051 2.285.035 2.801-.023zM47.641 36.676c.09-.496.238-.906.379-1.051.211-.207.551-.203.691.008.094.145.094.168-.012.328-.145.227-.277.32-.441.324-.09 0-.211.109-.363.313-.125.172-.242.313-.262.313s-.012-.105.008-.234zm44.84 65.246a3.25 3.25 0 0 0-.844.297c-.055.156.031.145.426-.051.242-.125.43-.234.418-.246zm-16.676-1.836c.258-.586.754-1.867.75-1.953-.004-.039-.395-.477-.875-.973l-.867-.902-.16.188c-.086.102-.301.395-.477.652s-.516.73-.754 1.059l-.449.609c-.008.012-.426-.039-.93-.113s-.922-.121-.934-.109c-.031.027.977.898 1.363 1.18l.84.59.523.363.816.063.816.066.078-.16.258-.559zm-29.187-.617l1.277-.223c3.25-.562 5.609-1.238 7.594-2.18.309-.148.668-.312.801-.367a321.3 321.3 0 0 0 8.328-4.133c.129-.074.344-.187.484-.262l.25-.129-.68-1.363-.684-1.363-.484.219c-1.242.563-7.937 3.906-8.137 4.063-.25.199-3.219 1.375-4.641 1.84l-1.5.504c-.637.234-3.34.879-5.004 1.191-.777.145-3.465.145-4.574-.004-3.203-.426-4.789-1.008-6.184-2.27-.535-.488-.789-.684-.789-.605 0 .086.281.828.363.969.672 1.098 1.207 1.695 1.949 2.18.473.305 1.656.883 2.156 1.051.652.219 1.539.434 2.465.594 1.496.262 1.672.289 2.277.344.746.066 4.273.027 4.73-.055zm26.477-1.422l.848-1.199c.305-.426.559-.809.559-.848s-.145-.25-.324-.469c-.586-.715-1.016-1.32-1.316-1.84-.164-.285-.316-.527-.344-.543-.051-.035-.203.129-.437.465-.082.117-.445.609-.801 1.086l-.66.887a98.56 98.56 0 0 1-2.633-.719c-.25-.078-.258-.016-.027.25a15.02 15.02 0 0 1 .41.516 13.49 13.49 0 0 0 .656.781l.602.68c.41.496.969.969 1.199 1.012l1.5.27.473.082c.004.008.137-.18.297-.41zm14.73-.453c.676-.062 1.875-.305 2.145-.434a21.79 21.79 0 0 1 .531-.223c1.359-.555 2.531-1.406 3.227-2.348.234-.316.496-.699.586-.855.598-1.039.941-2.805.805-4.121-.125-1.168-.488-2.16-1.199-3.254-.457-.711-.516-.75-2.328-1.637-.793-.387-2.199-.809-3.488-1.051l-.965-.176c-.926-.172-2.219-.234-4.102-.195-1.012.02-2.051.066-2.309.102-1.453.199-2.855.441-3.828.656-.809.184-1.84.465-1.93.527-.031.027-.258.109-.496.188-1.309.426-2.582 1.008-2.582 1.18 0 .23.938 2.574 1.059 2.648.031.02.328-.109.664-.281 1.145-.59 3.188-1.121 5.277-1.371 1.375-.168 5.07-.125 5.984.063l.926.191c2.359.484 4.016 1.617 4.91 3.359.309.598.535 1.469.535 2.031-.004.523-.238 1.133-.727 1.859-.422.629-1.137 1.383-1.625 1.723a10.57 10.57 0 0 1-2.035 1 7.44 7.44 0 0 1-1.105.238c-.332.047-.59.094-.578.105.031.031 1.051.117 1.527.129.223.004.727-.02 1.121-.055zm-2.18-.469c1.914-.266 3.625-1.336 4.684-2.934.469-.699.57-.973.574-1.543 0-.445-.02-.547-.203-.902-.82-1.637-2.848-2.781-5.832-3.305-1.855-.324-6.277-.187-7.973.246-1.145.293-2.746.926-3.133 1.242l-.152.121.457.613c1.625 2.172 3.082 3.605 4.855 4.77 1.273.836 3.074 1.586 4.109 1.715.742.09 1.867.082 2.613-.023zm-27.539-.801c.203-.082.375-.18.391-.215.02-.07-.656.203-.766.309-.086.086-.027.07.375-.094zm1.281-.516c.309-.121.426-.199.426-.277 0-.062-.02-.113-.047-.113-.105 0-.848.41-.871.48-.035.109.004.102.492-.09zm1.215-.465c.41-.152.488-.211.547-.41.059-.187-.023-.18-.535.055-.352.164-.426.223-.426.344 0 .082.012.148.027.148s.188-.062.387-.137zm10.137-.473a10.03 10.03 0 0 1 .438-.605c.113-.137.355-.461.539-.715l.477-.66.141-.187-.684-1.336c-.594-1.168-.871-1.758-1.074-2.285-.055-.152-.105-.113-1.637 1.199-.809.695-1 .84-1.102.84-.062 0-.609-.141-1.219-.309s-1.145-.312-1.191-.312c-.062 0-.051.051.039.203.066.109.313.605.547 1.102s.531 1.074.652 1.277.328.563.457.793.25.43.281.449c.047.027 1.074.336 1.777.535a21.63 21.63 0 0 1 .688.207c.273.082.527.152.566.156s.176-.152.305-.352zm-8.637-.176c.305-.133.328-.156.328-.363 0-.121-.016-.223-.035-.223a5.15 5.15 0 0 0-.465.219c-.395.199-.434.234-.434.41 0 .215.023.215.605-.043zm1.395-.59c.086-.047.113-.133.113-.34v-.281l-.434.219c-.43.219-.437.227-.437.469v.246l.32-.125c.176-.07.375-.156.438-.187zm.953-.445l.406-.199v-.586l-.434.195-.437.191v.301c0 .164.012.297.031.297s.211-.09.434-.199zm1.402-.676c0-.027-.059-.148-.129-.27-.09-.152-.16-.207-.234-.18-.207.074-.258.164-.258.457v.301l.309-.129c.172-.07.313-.152.313-.18zm-.566-1.879c-.344-.699-.492-.949-.492-.836 0 .059.609 1.27.73 1.457.195.297.125.117-.238-.621zm2.945-.652c.215-.199.531-.473.703-.605a24.8 24.8 0 0 0 .703-.59l.617-.535.223-.191-.113-.34c-.059-.184-.187-.508-.285-.719a32.27 32.27 0 0 1-.812-2.187l-.086-.266-.496.313-1.156.738-.906.566-.25.137-.867-.371-1.418-.629c-.301-.145-.559-.254-.566-.242-.043.047.668 2.344.977 3.168.473 1.25.496 1.285.813 1.367.145.035.43.125.637.195.363.125 1.762.539 1.848.547.027 0 .223-.16.438-.355zm21.926-.191c-.09-.168-.316-.453-.496-.633l-.332-.324-.043.207c-.043.184-.027.223.133.332.098.07.332.273.52.449s.352.313.363.301-.051-.16-.145-.332zm-16.09-1.27c0-.266-.012-.484-.031-.484s-.23.098-.469.219c-.484.246-.473.18-.137.762l.152.266.484-.273zm14.902.414c.094-.242.035-.449-.164-.598-.258-.195-1.012-.613-1.102-.613-.039 0-.066.137-.066.336v.332l.605.313c.715.367.676.355.727.23zm-13.594-.508c.152-.055.297-.113.32-.133s.063-.23.086-.469l.047-.426-.148.047c-.082.023-.391.125-.687.23l-.543.184-.02.438c-.012.238-.008.445.004.457s.164-.035.34-.105a11.88 11.88 0 0 1 .602-.223zm-11.23-.156c-.078-.18-.316-.93-.527-1.668-.426-1.477-.496-1.68-.605-1.773-.039-.031-.246-.746-.457-1.586-.609-2.422-.691-2.699-.812-2.801s-.242-.613-.652-2.707c-.117-.617-.246-1.234-.285-1.371a38.17 38.17 0 0 1-.215-.922c-.09-.422-.187-.707-.25-.75-.141-.105-.238-.926-.402-3.367-.125-1.848-.145-2-.262-2.094s-.137-.215-.207-1.305a120.08 120.08 0 0 1-.152-3.137c-.051-1.437-.09-1.953-.152-2.023-.137-.156-.148-.266-.289-2.367-.141-2.156-.133-3.68.023-4.609.07-.41.07-.496-.008-.613s-.082-.191.02-.617c.121-.508.492-1.723.621-2.031.188-.449.254-.617.625-1.652l.68-1.785c.238-.578.273-.723.215-.816s-.02-.227.219-.758c.266-.598 1.668-3.012 1.789-3.086.031-.02.055-.062.055-.098s-.594.547-1.32 1.293l-1.867 1.898-.551.535.051.676c.145 1.934-.187 3.207-1.148 4.391l-.516.637-.152.188-.004 3.211c0 2.496.023 3.531.109 4.672l.203 2.77c.156 2.176.379 4.324.586 5.602l.215 1.34c.34 2.113.777 4.094 1.035 4.695.031.07.25.801.492 1.625s.488 1.629.555 1.781.281.773.473 1.379.477 1.426.633 1.828l.656 1.762c.203.563.438 1.164.52 1.336s.328.762.551 1.309c.488 1.215.547 1.336.605 1.336.027 0-.016-.145-.094-.324zm23.117-.383v-.402l-.621-.199c-.344-.113-.637-.191-.648-.176s-.043.199-.066.41c-.035.305-.027.395.039.414l1.25.344c.027.004.047-.172.047-.391zm-10.176.066l.621-.145.051-.395c.027-.215.035-.402.023-.418-.027-.027-1.531.266-1.562.305-.012.008-.035.211-.059.445l-.043.43.172-.039.797-.184zm8.496-.539c0-.309-.023-.437-.078-.437-.043-.004-.355-.043-.699-.086s-.629-.078-.637-.07-.035.18-.055.387l-.043.379.461.074c.254.043.543.102.645.129.402.105.406.102.406-.375zm-6.441.18c.309-.039.578-.102.605-.141s.047-.203.047-.367c0-.289-.004-.297-.187-.297s-1.387.125-1.41.148c-.004.004-.031.188-.055.406l-.047.398.242-.043a19.97 19.97 0 0 1 .805-.105zm4.594-.414c.02-.195.027-.363.012-.379s-.371-.039-.797-.059l-.77-.031-.043.148a2.02 2.02 0 0 0-.039.379v.23l.387.023c.215.012.574.027.801.031l.414.012zm-1.961-.094l.02-.359h-.805c-.906 0-.879-.012-.883.516v.234l1.648-.031zm-14.922-1.434l.586-.395c.105-.074.52-.336.922-.59s.762-.488.797-.531c.047-.055-.016-.336-.219-.969-.254-.797-.406-1.559-.578-2.867l-.051-.359-.496.352c-.273.191-.582.426-.684.523s-.426.344-.719.551l-.719.535-.187.156-.621-.273-1.336-.566c-.391-.164-.84-.363-.992-.441s-.285-.137-.293-.125c-.035.031.719 3.008.879 3.488l.148.438.613.266.984.449c.633.313 1.363.621 1.465.625.055 0 .277-.117.5-.266zm-.742-4.477l1.113-.844.855-.652.328-.258-.039-.281c-.02-.156-.164-1.086-.316-2.059l-.363-2.324c-.043-.301-.094-.562-.105-.574-.031-.031-1.664 1.141-2.758 1.98-.789.609-.719.59-1.133.336-.437-.262-.602-.355-1.48-.816-.375-.199-.82-.445-.984-.551s-.312-.168-.332-.148.051.438.156.926l.477 2.348c.395 2.012.336 1.875.867 2.09a24.54 24.54 0 0 1 .957.422c.762.355 1.957.848 2.074.852.055 0 .363-.199.684-.445zM3.488 80.348l-.336-1.488c-.133-.566-.359-1.781-.844-4.512l-.215-1.184c-.137-.707-.449-2.98-.594-4.293-.738-6.629-.652-11.105.281-14.801.168-.668.297-1.094.426-1.383a11.5 11.5 0 0 0 .238-.621c.215-.605.707-1.707.813-1.828.051-.059.164-.254.258-.441.559-1.113 1.598-2.457 2.887-3.73 1.297-1.281 2.789-2.406 4.895-3.68l1.094-.668c.594-.367 2.27-1.199 3.168-1.574.43-.176.891-.371 1.027-.437.426-.191 1.273-.531 2.117-.84.445-.16.949-.355 1.121-.434s.719-.27 1.215-.434 1.227-.418 1.621-.566 1.375-.465 2.176-.707l1.586-.504c.23-.102.676-.23 2.117-.617l1.934-.535c.633-.191 2.063-.523 3.668-.848 1.676-.34 1.727-.348 2.332-.434.293-.039.539-.078.551-.09s-.07-.141-.187-.289c-.234-.293-.363-.551-.363-.734 0-.062-.02-.129-.047-.145-.051-.031-.969.117-2.223.363l-5.121 1.051c-.301.086-1.215.313-2.039.508a42.5 42.5 0 0 0-2.145.563c-.359.113-1.16.352-1.777.527s-1.344.398-1.617.492l-3.422 1.23c-.242.105-.801.34-1.246.52-.867.348-3.262 1.488-3.672 1.75-.137.086-.445.262-.684.395-.477.258-2.395 1.5-3.492 2.262-1.629 1.125-3.078 2.316-4.137 3.402-1.258 1.289-1.832 2.063-2.563 3.453-.453.863-.73 1.621-1.062 2.898-.5 1.938-.594 2.504-.887 5.441-.137 1.34-.187 4.527-.098 6.102.117 2.031.438 4.965.684 6.285.031.152.129.715.223 1.246.273 1.598.707 3.789.867 4.418l.527 2.086c.332 1.309 1.063 3.727 1.113 3.676.012-.012-.066-.395-.168-.852zm52.496.68c.379-.297.441-.883.184-1.77-.187-.656-.586-1.578-.785-1.824l-1.414-1.684a45.28 45.28 0 0 1-.566-.687l-.641-.777a23.35 23.35 0 0 1-.594-.715c-.262-.324-.531-.652-.605-.723a6.49 6.49 0 0 1-.301-.344l-.168-.211.039.313c.098.797-.227 1.727-.77 2.215-.328.289-.457.52-.672 1.18-.02.063.277.422.859 1.027 2.473 2.594 3.656 3.613 4.613 3.98.484.188.605.191.82.02zm18.613-.172c.902-.453 2.219-1.621 4.285-3.812l.922-.98-.16-.398a3.78 3.78 0 0 1-.16-.449c0-.027-.164-.195-.363-.375-.578-.52-.895-1.426-.785-2.234l.043-.309-.336.367c-.184.199-.437.504-.566.676s-.324.41-.437.535-.434.504-.715.848l-1.086 1.316-.937 1.125c-.234.285-.445.633-.609 1.016l-.309.711c-.281.621-.344 1.574-.133 1.918.262.426.57.434 1.348.047zm50.289-1.312l.426-1.43c.129-.375.414-1.555.762-3.145l.34-1.523c.242-1.039.719-3.945.93-5.695l.188-1.523c.371-2.895.426-6.699.129-8.965l-.129-.996c-.348-2.785-.762-4.855-1.137-5.668-.055-.117-.184-.43-.289-.695-.285-.73-.562-1.242-1.133-2.105-.891-1.348-1.699-2.215-3.504-3.754-.969-.828-2.461-1.887-4.867-3.457-2.113-1.375-5.152-2.809-8.27-3.898l-1.93-.676c-.344-.117-1.125-.359-1.742-.535l-1.465-.437c-.34-.117-2.258-.609-3.484-.895-.359-.082-.863-.207-1.121-.277-.547-.145-5.711-1.168-6.504-1.285a7.52 7.52 0 0 1-.387-.066c-.113-.023-.141 0-.145.113 0 .184-.207.641-.406.902l-.156.203h.148c.539 0 5.328 1.004 6.516 1.367a63.69 63.69 0 0 0 1.465.41c1.121.293 1.82.496 2.363.68.242.082 1.066.348 1.836.586 2.375.742 6.723 2.309 7.844 2.828.188.086.637.273.996.418.867.348 3.055 1.422 3.52 1.73.203.133.734.461 1.18.727s.824.508.84.531.27.195.559.375a20.49 20.49 0 0 1 4.797 4.258c.988 1.223 2.184 3.379 2.668 4.805.66 1.953.992 3.566 1.195 5.809.113 1.246.098 4.957-.031 6.723-.102 1.402-.352 3.914-.469 4.699l-1.125 6.973a207.09 207.09 0 0 0-.867 4.305c.008.109.145-.309.461-1.414zM58.109 78.02l.707-1.027.234-.34-.113-.227c-.223-.445-1.039-1.465-1.426-1.789-.547-.453-1.016-.68-1.633-.785l-.23-.039-.324.641c-.785 1.566-.754 1.484-.664 1.605.043.059.316.387.602.723.727.863 1.223 1.93 1.406 3.035l.043.238.465-.676.934-1.359zm14.711 1.672c.008-.238.301-1.168.52-1.645s.621-1.031 1.215-1.691c.18-.199.32-.387.309-.414s-.25-.516-.535-1.086l-.52-1.039-.23.035c-.59.102-1.074.328-1.586.746a7.44 7.44 0 0 0-1.395 1.699l-.199.367.32.461.836 1.207c1.109 1.613 1.172 1.695 1.219 1.594a.83.83 0 0 0 .047-.234zm-27.453-1.586c.176-.68.586-1.434 1.047-1.93.121-.133.168-.27.207-.629.063-.566.25-1.113.496-1.457.102-.145.188-.285.188-.316 0-.086-.449-.484-.672-.602-.258-.133-.668-.125-.98.016-.137.063-.445.301-.687.523-.336.313-.484.512-.637.84-.805 1.746-.613 3.871.449 4.984l.379.395.043-.676c.023-.371.102-.887.168-1.148zm39.844.66c.316-.664.402-1.094.395-1.992-.008-1.383-.34-2.293-1.109-3.047-.816-.801-1.484-.867-2.121-.215-.227.23-.238.258-.156.387a3.74 3.74 0 0 1 .625 1.711c.039.301.094.438.238.602.102.117.344.477.535.793.418.703.586 1.301.641 2.258l.039.684.363-.398c.203-.219.449-.57.551-.781zm-36.477.594c-.094-.508-.074-1.648.039-2.113.051-.227.258-.746.457-1.156.438-.898.5-1.113.5-1.684 0-.398-.016-.453-.172-.602-.234-.219-.68-.422-.93-.422-.277 0-.547.141-.879.457-.441.422-.711 1.063-.82 1.938-.168 1.344.449 2.922 1.426 3.66.219.164.406.289.418.277s-.004-.172-.039-.355zm32.645-.164c.801-.785 1.223-2.008 1.129-3.27-.074-1-.324-1.648-.805-2.09-.531-.488-.883-.566-1.398-.305-.496.254-.59.398-.586.906 0 .563.281 1.414.582 1.785.031.039.145.305.254.594.188.496.199.559.199 1.453.004.512-.02 1.055-.047 1.203l-.055.273.199-.117c.109-.062.348-.258.527-.434zm-39.207-.801a5.34 5.34 0 0 1 1.137-2.074c.105-.109.207-.293.23-.402.129-.613.5-1.578.75-1.953l.277-.426c0-.047-.242-.273-.355-.332-.207-.109-.703-.086-1 .051-.684.309-1.187.848-1.535 1.633-.379.855-.441 1.148-.434 2.008.004.754.016.84.191 1.258.105.246.27.582.367.746l.176.301.043-.187c.027-.102.094-.383.152-.621zm45.711.082c.281-.598.406-1.441.316-2.121-.133-1.043-.559-1.969-1.164-2.539-.512-.477-.82-.633-1.328-.668-.387-.027-.414-.02-.648.184l-.242.211.172.184c.355.383.695 1.172.848 1.965.063.34.121.445.367.711.328.352.758 1.059.875 1.43.043.137.098.277.125.313s.094.281.152.543l.105.484.121-.152a3.39 3.39 0 0 0 .301-.543zm-22.996-2.309l2.488-1.844.359-.273-.035-.504c-.02-.277-.09-1.082-.16-1.781l-.148-1.633c-.012-.199-.039-.375-.062-.395s-.227.086-.461.238-.703.438-1.047.637-.805.48-1.027.625-.605.371-.852.512l-.445.25-.613-.395-1.344-.848c-.406-.25-.879-.559-1.055-.687s-.328-.223-.34-.211c-.031.031.219 3.656.301 4.344.039.332.09.629.109.66s.277.176.566.32a16.05 16.05 0 0 1 .742.398c.375.234 2.086 1.148 2.164 1.156.043.004.43-.254.859-.57zm5.582-.312c.492-.797 1.344-1.652 1.941-1.941.422-.207 1.059-.406 1.297-.406.121 0 .148-.195.117-.84-.059-1.336-1.066-2.641-2.379-3.086-.477-.16-1.457-.234-1.66-.125-.129.066-.137.109-.137.926 0 1.309.117 4.219.188 4.742.129.918.199 1.309.238 1.309.023 0 .199-.262.395-.578zm-11.34-.109c-.398-1.527-1.055-5.156-1.059-5.832 0-.332-.074-.359-.441-.176-1.371.691-2.078 1.938-1.996 3.531.012.215.023.23.277.27.82.125 1.477.48 2.148 1.168.578.59.664.695.949 1.18.117.195.219.348.227.34s-.039-.227-.105-.48zm-8.992-1.879c-.047-.297-.242-.75-.371-.855-.227-.187-2.187-.418-4.078-.48-1.906-.059-2.883.098-3.148.504-.059.09-.094.176-.082.191s.191-.059.398-.16c.43-.215.945-.312 1.262-.238.121.027.34.133.488.234l.262.188.477-.246c.391-.199.535-.242.828-.242a1.63 1.63 0 0 1 1.184.523l.207.234.133-.121c.438-.398 1.035-.465 1.613-.184.395.195.66.469.715.738.055.285.16.207.113-.086zm29.547-.348c.195-.207.652-.43.984-.477.324-.047.719.07 1.035.313l.203.156.219-.242c.516-.57 1.289-.656 2.063-.234.414.227.414.23.547.109.469-.426 1.23-.441 1.961-.035.32.176.379.156.227-.074-.266-.402-1.242-.562-3.113-.504-1.953.063-3.879.289-4.113.48-.133.113-.324.566-.363.871l-.039.277.125-.246c.07-.133.191-.312.266-.395zm-.527-.363c.121-.219.5-.512.746-.578.703-.187 3.238-.434 4.496-.437.82-.004 1.719.102 2.074.242.277.105.301.082.168-.18-.266-.52-1.207-1.105-2.066-1.281-.437-.09-.742-.102-2.012-.07-1.535.039-2.078.105-2.727.34-.566.207-.883.418-1.008.672-.219.469-.234 1.379-.023 2.004l.078.23.102-.402c.055-.223.133-.465.172-.539zm-28.492.664c.18-.43.184-1.426.008-1.879-.195-.504-1.133-.895-2.512-1.047-.93-.098-2.766-.082-3.32.031-.832.172-1.754.758-2.02 1.281-.133.258-.109.281.168.176.742-.289 3.098-.316 4.906-.059 1.449.207 1.797.277 1.98.402.27.18.551.656.586.996.039.328.098.355.203.098zM64.547 71l2.277-1.379.695-.41.039-.98c.023-.539.082-1.312.133-1.727.082-.687.355-2.266.441-2.547.031-.105.004-.098-.266.055-.168.098-.484.293-.707.438s-.699.441-1.059.656l-2.52 1.543-.305.199-.16-.141-.504-.449c-.187-.172-.664-.594-1.051-.934l-1.273-1.137c-.309-.281-.586-.516-.617-.516-.098 0 .035 3.496.195 5.199l.039.434.535.344 1.016.652 1.266.824.777.512.227-.137.82-.5zm6.781-4.25a6.49 6.49 0 0 0 .684-.086l.656-.125c.59-.117 1.969-.449 1.945-.473-.012-.012-.344-.051-.734-.09-1.371-.133-2.551-.379-3.355-.707-.219-.09-.418-.164-.441-.164s-.07.23-.109.512c-.137.977-.145 1.156-.062 1.188s.613.012 1.418-.055zm-13.664-.695c-.023-.402-.055-.75-.066-.773s-.258.039-.539.133c-.648.227-1.609.422-2.461.508-.367.035-.805.082-.977.109l-.312.043.84.199c1.605.383 2.211.484 3.074.5l.488.012zm6.602-.215l1.398-.859 1.773-1.086c.516-.309.98-.605 1.035-.656.063-.055.141-.328.203-.684.055-.328.145-.746.199-.934.414-1.453.578-2.328.738-3.953.051-.512.086-.937.078-.941-.016-.016-.957.547-1.535.914l-3.051 1.895c-.223.133-.437.273-.48.316s-.262.18-.492.305l-.414.234-2.23-2.234-2.234-2.168c0 .031-.043.395-.094.809a15.19 15.19 0 0 0-.09 1.414c-.004 1.07.242 4.609.328 4.727.074.105 3.418 3.086 3.738 3.336.18.137.18.137.375.02l.754-.453zm-9.961-.211a7.82 7.82 0 0 0 .871-.121l.84-.184c.258-.055.719-.172 1.02-.266l.555-.164-.039-.371c-.023-.203-.039-.535-.039-.734 0-.242-.027-.363-.074-.363-.039 0-.23.121-.422.27-.652.512-1.262.871-2.141 1.266l-1.254.578-.375.188.281-.027.777-.07zm20.125-.082c-.195-.098-.516-.238-.715-.32-.992-.395-2.125-1.039-2.891-1.652l-.41-.324-.07.289c-.043.16-.113.492-.16.742-.098.527-.148.473.617.719 1.191.391 2.602.648 3.859.715.066.004-.035-.07-.23-.168zm-57.387-2.309c.316-1.402.48-2.086.699-2.891a29.13 29.13 0 0 1 1.301-3.551 15.95 15.95 0 0 1 1.23-2.086c1.434-1.898 4.926-3.797 8.109-4.414.66-.129 1.781-.246 2.727-.289l.953-.047.258-.379a7.53 7.53 0 0 1 .496-.641l.242-.262-.238-.051c-.129-.027-.84-.047-1.574-.047-1.187 0-1.449.02-2.336.176-2.125.375-3.883.934-5.535 1.766-1.176.59-2.656 1.816-3.43 2.844-.766 1.012-.922 1.289-1.695 2.969-.355.777-.684 1.914-1.16 4.023-.223.996-.414 2.93-.402 4.121l.004.594.145-.777.207-1.059zm94.227.637c-.023-.641-.086-1.543-.141-2.008-.191-1.617-.594-3.512-.898-4.199-.027-.07-.156-.418-.281-.777-.285-.832-.535-1.391-.922-2.055-.988-1.703-1.93-2.723-3.414-3.711-.812-.539-1.297-.789-2.387-1.215l-.746-.301c-.48-.215-2.117-.625-3.395-.855-.855-.148-1.117-.168-2.363-.172-.77 0-1.48.02-1.574.047l-.176.047.332.359c.18.199.398.488.48.645l.152.281.938.047c1.941.09 3.996.461 4.977.902a4.97 4.97 0 0 0 .41.16c.957.289 3.141 1.516 4.191 2.359a8.09 8.09 0 0 1 1.801 2.059 21 21 0 0 1 1.262 2.613 46.47 46.47 0 0 1 1.051 3.363c.18.727.566 2.703.629 3.219.082.676.113.359.074-.809zm-54.742-.219c.156-.109.418-.324.578-.473l.297-.273-.043-.66c-.023-.363-.055-.777-.074-.918l-.035-.254-.695.352c-1.547.781-2.812 1.027-5.07.988l-1.234-.008a64.68 64.68 0 0 0 .508.223c.887.375 2.535.902 3.238 1.035.898.164.953.172 1.652.184.578.008.605.004.879-.195zm16.91.109c1.09-.16 2.676-.637 3.902-1.176l.441-.191-1.625-.004c-1.281-.004-1.746-.027-2.211-.113-.777-.145-1.645-.43-2.359-.766-.316-.152-.598-.262-.621-.238s-.129.383-.234.797c-.23.883-.273.766.504 1.387l.496.398h.523c.285 0 .82-.043 1.184-.098zM53.109 62.02c1.613-.172 2.777-.523 3.664-1.105l.477-.312-.035-.781c-.016-.434-.039-.793-.047-.805-.02-.023-1.418.691-2.238 1.148-1.758.973-2.316 1.199-4.016 1.633-.777.203-.781.184.031.281.465.055 1.355.031 2.164-.059zm24.465-.012c.156-.043.102-.066-.434-.195-1.02-.242-2.234-.668-2.902-1.016-.348-.18-1.109-.645-1.691-1.027l-1.062-.699-.187.793c-.105.434-.211.824-.234.863-.062.102.551.477 1.156.711 1.137.438 2.758.688 4.215.648.531-.016 1.043-.047 1.141-.078zm-48.102-3.066c.348-1.898.863-3.648 1.234-4.203l.332-.492a21.25 21.25 0 0 1 1.273-1.621 15.94 15.94 0 0 0 .402-.465c.098-.121.438-.496.758-.832s.57-.625.559-.637-.383.012-.828.051l-.805.07-.312.34c-.406.441-1.258 1.652-1.613 2.293-.402.73-.887 1.91-1.031 2.516-.191.824-.312 2.23-.305 3.586l.008 1.23.086-.496.242-1.34zm69.422.688c.012-1.918-.273-3.75-.77-4.969-.402-.988-1.168-2.234-1.988-3.25l-.48-.59-.816-.074c-.449-.043-.832-.062-.848-.047s.137.207.34.418c1.027 1.066 2.254 2.539 2.73 3.281.68 1.059 1.023 2.141 1.465 4.637l.191 1.008c.031.145.063.34.066.438l.004.176.051-.156c.027-.086.051-.48.055-.871zm-44.852.645c.18-.09.41-.223.512-.297s.734-.414 1.402-.762l1.211-.629.02-.414c.012-.227 0-.41-.027-.41a5.26 5.26 0 0 0-.484.148c-.836.285-1.312.355-2.668.391-1.641.043-2.836-.074-4.617-.457l-.906-.191-.152-.035.125.141c.27.305 2.02 1.406 2.953 1.859.875.422 1.906.809 2.199.816.059 0 .254-.07.434-.16zm21.105-.047c.875-.293 2.18-.934 2.996-1.473.879-.578 1.473-1.004 1.445-1.031-.012-.012-.398.074-.859.191-1.18.297-1.98.41-3.168.449-1.254.043-2.16-.07-3.082-.387l-.68-.23c-.031-.008-.102.191-.152.449l-.09.461.223.184c.453.367 2.445 1.598 2.59 1.598.082 0 .434-.098.777-.211zm-9.703-1.266c.105-.055.203-.117.219-.137s.215-.152.438-.293l.934-.582 1.398-.863c1.137-.691 1.457-.926 1.492-1.09.066-.336.082-1.988.031-3.09-.059-1.219-.234-3.18-.285-3.234-.016-.016-.078.004-.137.043s-.258.164-.445.277-.652.398-1.027.645l-1.316.844-1.297.816c-.367.234-.687.422-.719.422s-.273-.242-.539-.543l-.98-1.07-1.516-1.637c-.312-.34-.332-.352-.387-.219-.105.25-.496 1.289-1.133 3-.613 1.648-.984 2.793-.984 3.043 0 .082.793.914 2.289 2.402l2.285 2.277.746-.457.934-.555zm11.805-1.07c.398-.07.93-.184 1.184-.258l.832-.234.371-.105-1.027-.043c-1.723-.07-2.473-.187-3.699-.586-.375-.121-.75-.211-.84-.195-.785.137-2.16.457-2.195.508-.105.156-.047.348.137.441.398.207 1.176.457 1.621.523l.621.094c.395.07 2.309-.023 2.996-.145zm-21.605-.031c.258-.059.516-.133.574-.164s.137-.059.172-.059.23-.074.438-.168c.25-.109.371-.199.371-.273 0-.133.023-.125-1.211-.367l-1.531-.32-.535-.121-.848.281a11.26 11.26 0 0 1-1.395.375c-.617.105-1.914.215-2.625.227l-.477.008.469.117c.824.211 1.883.418 2.488.484l.777.094c.105.016.789.02 1.527.008.977-.012 1.465-.043 1.805-.121zm1.613-2.637c0-.281-.004-.512-.012-.512s-.172.055-.359.117c-.875.309-2.242.324-3.547.043-1.07-.23-2.82-.664-2.895-.719-.031-.023-.195-.078-.363-.121-.289-.07-2.305-.902-2.437-1.004-.035-.027-.469-.219-.965-.43-.898-.375-.902-.379-1.367-.879l-.469-.5-.965-.16-1.211-.207c-.137-.027-1.629-.262-3.316-.523-2.797-.437-3.074-.469-3.164-.379-.156.156-.52.32-1.035.469-.32.094-.531.195-.645.313l-.164.176.238.102c.133.055.824.281 1.539.5l1.648.527c.195.07 1.039.309 1.879.535l2.023.559c.449.137 3.207.828 5.414 1.359l2.332.531c.805.176 1.52.344 1.59.375s1.258.297 2.645.59l3.02.645.496.109.043-.5a21.58 21.58 0 0 0 .047-1.016zm15.781 1.102l2.492-.551 2.332-.527 2.367-.562 2.707-.656 2.18-.559c1.832-.469 4.918-1.328 5.164-1.437.07-.031.363-.129.652-.219 1.66-.512 2.68-.867 2.68-.937 0-.102-.359-.398-.488-.398-.184-.004-.953-.297-1.199-.461l-.238-.16-2.852.434-3.145.504c-.242.059-.367.145-.684.477-.508.531-.684.637-1.945 1.172l-1.824.805c-.598.273-1.012.406-1.898.621l-1.484.367c-.195.055-.727.176-1.184.27-.723.152-.969.176-1.918.176-1.234 0-1.715-.09-2.152-.402l-.254-.18-.074.352c-.07.344-.32 1.961-.32 2.082 0 .035.059.043.137.02s.504-.125.949-.227zm-17.051-1.617c.777-.109.988-.211 1.512-.734.598-.59 1.199-1.531 1.422-2.219.137-.422.168-.645.191-1.445.027-.867-.043-1.77-.16-2.066-.043-.113-.066-.109-2.406.332-1.48.281-2.012.371-4.086.695-3.672.578-4.785.73-6.109.852l-.855.078-.195.422-.199.422.402.406c.449.457.613.547 2.02 1.148.547.23 1.289.563 1.652.734.477.223.996.395 1.867.621l1.551.402a15.37 15.37 0 0 0 .996.223l.777.152c.242.055 1.191.039 1.621-.023zm19.605-.367c.473-.078 1.715-.344 3.234-.695 1.02-.234 1.523-.383 1.961-.582a51.38 51.38 0 0 1 1.215-.531c.344-.141.875-.379 1.184-.527s.668-.305.797-.352c.332-.113 1.125-.945 1.051-1.098-.375-.777-.273-.715-1.352-.824-1.57-.16-5.859-.801-7.656-1.145l-4.371-.785c-.082 0-.297.508-.434 1.008-.168.625-.137 1.844.07 2.633.207.805.633 1.648 1.082 2.145.379.426.629.563 1.227.676l.496.098a8.07 8.07 0 0 0 1.496-.02zm-9.937-2.555l1.5-.965 2.48-1.582c.07-.043.07-.156.008-.652-.094-.742-.039-2.789.09-3.461.047-.258.082-.473.078-.48-.016-.016-1.527.434-1.906.57a32.14 32.14 0 0 1-1.109.34l-.828.238-.152-.238c-.082-.133-.152-.258-.152-.277s-.078-.152-.176-.297-.34-.551-.531-.91l-.383-.676c-.047-.047-.246.219-1.008 1.352-1.234 1.828-2.254 3.66-2.156 3.871.051.102.395.484 1.609 1.797l1.672 1.848c.063.113-.008.148.965-.477zM34.047 50.32c.676-.094 1.535-.332 1.789-.5.078-.051.191-.191.246-.309.098-.199.094-.23-.07-.727-.215-.645-.516-1.164-.703-1.211-.207-.051-1.125.539-1.781 1.148-.641.598-.773.75-.988 1.133-.25.434-.266.547-.098.578.195.039.82-.008 1.605-.113zm61.547.117c.117-.031.137-.066.102-.203-.074-.305-.406-.734-1.043-1.352-.621-.605-1.477-1.234-1.785-1.312-.223-.055-.453.258-.754 1.031-.223.578-.242.672-.18.859.156.477 1.145.793 3.047.977.406.039.469.039.613 0zm-62.012-2.203c.066-.074.387-.309.715-.527l.594-.398.207-1.078.324-1.512c.066-.238.219-.883.344-1.43.848-3.742 1.723-6.59 2.738-8.91.02-.043-.078-.086-.258-.113a3.39 3.39 0 0 1-.547-.133c-.141-.051-.289-.078-.328-.062s-.199.375-.355.797l-.453 1.172c-.215.508-1.086 3.039-1.363 3.957a33.66 33.66 0 0 0-.434 1.68l-.387 1.613c-.211.844-1.078 4.902-1.078 5.055 0 .07.188-.008.281-.109zm61.148.043c0-.105-.344-1.734-.684-3.246l-.5-2.27c-.141-.637-.309-1.336-.375-1.559l-.34-1.184c-.258-.898-.773-2.488-.836-2.582-.023-.035-.09-.187-.145-.34-1.156-3.23-1.129-3.176-1.48-2.988-.066.035-.289.098-.496.141l-.375.078.129.246c.379.738 1.328 3.543 1.805 5.355l.531 2.008c.18.684.406 1.68.836 3.715.172.816.348 1.539.391 1.605.074.113 1.398 1.086 1.492 1.098.027.004.047-.031.047-.078zm-27.633-3.172c.465-.156.891-.281 2.23-.672.539-.156.719-.234.762-.332.027-.07.168-.926.305-1.902l.289-1.883c.016-.059.02-.109.004-.109-.055 0-.887.32-1.148.441-.145.066-.656.266-1.133.438l-.867.32-.105.32a22.42 22.42 0 0 1-.711 1.77c-.086.133-.09.156.277-1.203l.289-1.059-.285-.527c-.156-.289-.312-.555-.344-.586s-.16.168-.328.527c-.258.551-.754 1.418-.852 1.488-.121.086-.687.848-.687.926 0 .047.234.484.523.965l.652 1.141.129.258.266-.082c.148-.043.477-.152.734-.238zm6.926-3.262c-.07-.187-.273-.535-.437-.758-.176-.23-.223-.199-.281.184-.031.211-.02.227.27.391.164.094.32.199.34.23.063.105.152.066.109-.047zm-5.609-1.113c.285-.121.953-.387 1.48-.582l.965-.363.086-.426c.125-.602.113-1.797-.02-2.496-.059-.316-.133-.594-.16-.609-.055-.035-.223.066-1.59.961l-.812.535-.074.656c-.062.574-.406 2.133-.539 2.441-.066.164.074.137.664-.117zm-.77-.832c.281-1.098.418-1.969.332-2.102-.023-.035-.164-.137-.32-.23a3.75 3.75 0 0 1-.41-.293l-.129-.117-.039.672c-.02.371-.094.926-.164 1.234l-.133.559.297.539c.164.297.316.52.34.496s.125-.367.227-.758zM50.328 38.34c-.121-.172-.234-.367-.254-.437-.051-.152-.043-.152-.562.023l-.43.145.309.18c.328.191.859.398 1.043.398.09.004.07-.051-.105-.309zm24.93-.426c-.152-.215-1.227-1.234-1.391-1.324-.102-.055-.109-.02-.109.465 0 .445.016.531.109.559.059.016.414.145.793.281s.695.238.707.227-.039-.105-.109-.207zm-23.328-.336c-.066-.078-.191-.25-.27-.391l-.16-.27c-.023-.027-.578.301-.621.367-.062.102.703.41 1.059.426.113.008.113.004-.008-.133zm17.918-1.199c.469-.309.852-.594.855-.641s-.09-.281-.203-.527c-.191-.414-.613-.949-1.129-1.434-.25-.23-.859-.574-1.305-.727l-.352-.121-.051.141c-.027.074-.129.711-.227 1.41l-.172 1.277.375.383c.352.359.637.887.637 1.176 0 .063.109.02.359-.145l1.211-.793zm-1.941.797c-.004-.105-.375-.684-.539-.836-.207-.191-.641-.488-.711-.488-.031 0 .027.176.129.387.152.332.242.434.598.688.414.289.523.34.523.25zm-14.687-.75c-.031-.059-.109-.172-.172-.246l-.109-.141-.211.129c-.234.141-.211.203.117.297.355.098.449.09.375-.039zm7.031-.66l1.09-.242.371-.082-.02-.277c-.02-.293-.164-1.203-.266-1.687l-.062-.289-.48.121c-.719.184-1.687.492-1.723.555-.02.031.027.176.105.32.125.234.141.359.145 1.031 0 .48.023.754.063.738s.383-.098.777-.187zm6.742-.367c.133-.793.367-2.668.336-2.695-.133-.137-2.691-.203-2.816-.074-.027.023 0 .449.055.941a15.07 15.07 0 0 1 .102 1.219c0 .203.031.328.078.34s.344.055.672.09.789.137 1.027.219a9.72 9.72 0 0 0 .473.16c.023 0 .055-.086.074-.199zm-4.035-.141c.344-.047.797-.09 1.012-.09.383 0 .391-.004.387-.172 0-.254-.105-1.219-.195-1.805-.074-.504-.082-.516-.25-.516a2.39 2.39 0 0 0-.437.066 47.33 47.33 0 0 1-1.004.211c-.402.082-.746.16-.758.172s.043.359.125.77a18.87 18.87 0 0 1 .191 1.102c.051.414-.008.398.93.262zm-23.91-1.461c.879-.328 1.207-1.273.742-2.129-.285-.527-.785-.797-1.473-.793-1.078.004-1.738.871-1.441 1.902.258.887 1.266 1.359 2.172 1.02zm51.27-.047c.328-.16.648-.453.785-.723a1.93 1.93 0 0 0 .055-1.25c-.293-.766-1.367-1.137-2.184-.758-.375.172-.789.621-.891.961-.238.801.145 1.535.969 1.848.328.129.918.09 1.266-.078zm-15.039-.383c-.078-.078-1.059-.672-1.336-.816-.191-.098-1.457-.559-1.531-.559-.016 0 .059.125.164.281s.25.418.32.582l.125.301.977.125c1.148.148 1.363.164 1.281.086zm-21.523-.195c.191-.094.301-.203.395-.395.223-.461.176-.492-.371-.27-.336.141-.996.586-.996.676 0 .023.063.059.141.07.324.063.582.039.832-.082zm-1.23-.266c.379-.352.98-.684 1.566-.855.254-.078.551-.207.652-.285l.539-.418c.125-.094.352-.332.496-.527 1.883-2.492 3.504-3.906 5.184-4.531.32-.121.348-.141.188-.145-.48-.008-2.57.453-3.48.766-3.023 1.039-5.809 2.598-6.664 3.73-.469.617-.605.969-.578 1.48.023.367.051.453.203.613.32.332.816.512 1.305.473.25-.023.348-.07.59-.301zm-16.02-1.082c.047-.195.223-.465.48-.742l.184-.195-.176-.039c-.098-.023-.555-.156-1.02-.293s-1.527-.426-2.363-.645l-1.867-.492a65.61 65.61 0 0 0-3.023-.719l-.625-.129-.43.18-.434.176-.777-.113c-4.027-.598-8.973-.773-12.324-.434l-1.371.156c-.051.016.543.02 1.316.004 3.941-.066 8.543.277 11.602.867 1.18.227 4.492.918 5.039 1.055l2.055.5c1.555.375 2.559.656 3.164.883.492.184.52.18.57-.02zm55.863-.086c.426-.137 1.395-.402 2.148-.59l1.742-.437c1.367-.352 4.676-1.051 6.598-1.395 2.863-.512 6.047-.758 9.926-.766 2.719-.004 2.699.004.531-.227-1.023-.109-4.484-.152-5.883-.074-2.059.113-4.543.355-6.133.594l-.559.086-.398-.18-.398-.184-.918.191a42.93 42.93 0 0 0-1.965.461l-4.91 1.301-.785.23-.477.137.238.246c.133.137.281.379.34.551s.105.309.113.309a55.59 55.59 0 0 0 .789-.254zm-53.59-1.289c0-.195-.715-1.473-1.145-2.043-.469-.617-1.699-1.883-2.062-2.113-.102-.066-.328-.234-.504-.375s-.582-.406-.902-.586a17.07 17.07 0 0 1-.742-.434c-.461-.316-.301-.102.504.672 1.668 1.605 2.84 3.055 3.449 4.262l.355.703.523-.004c.379 0 .523-.023.523-.082zm51.688-.359c.176-.375 1.215-1.992 1.336-2.082.023-.016.258-.281.52-.59a31.37 31.37 0 0 1 2.164-2.246c.414-.367.547-.551.277-.371a12.88 12.88 0 0 1-.652.379c-.508.281-1.195.707-1.246.773-.016.023-.297.254-.621.516-.695.555-1.516 1.43-2.004 2.133-.41.586-.961 1.551-1.016 1.773l-.035.156.539-.008.539-.004zm-27.941-.863c.305-.051.789-.109 1.074-.133.5-.043.527-.051.965-.383.246-.184.953-.586 1.57-.891l1.121-.555h-.437c-.582 0-1.605.148-2.332.34-.988.262-1.559.578-2.414 1.324-.406.355-.484.488-.25.426.078-.023.395-.082.703-.129zm-49.965-.637c-.062-.016-.16-.016-.219 0s-.012.027.109.027.168-.012.109-.027zm103.133 0c-.059-.016-.156-.016-.219 0s-.008.027.109.027.168-.012.109-.027zm0 0" fill="#09637d"/><path d="M83.941 103.961c.195-.023.195-.027.234-.473l.051-.789.012-.34-.277-.051a23.16 23.16 0 0 1-1.105-.273l-.852-.199c-.062.063-.27 1.707-.223 1.754.07.07.465.148 1.281.258.414.055.527.082.34.086-.645.016-2.527-.336-4.027-.746-2.914-.797-5.402-2.031-7.484-3.715-1.254-1.012-3.062-2.93-5.035-5.348l-.844-1.012c-.008 0-.406.184-.883.41-1.836.875-5.937 2.57-8.086 3.344-4.594 1.656-8.258 2.547-11.418 2.781-2.77.207-6.105-.152-8.367-.895-2.305-.762-3.555-1.723-4.375-3.379-.309-.621-.43-1.203-.254-1.203.113 0 .445.219.98.652.563.449 1.199.879 1.633 1.098.871.438 2.375.828 4.063 1.059 1.215.16 3.738.195 4.699.063 1.711-.238 4.414-.934 6.91-1.785 2.379-.809 3.211-1.187 8.914-4.059 2.879-1.449 4.031-2.059 4.031-2.133 0-.156-.242-.832-.793-2.207-.656-1.641-1.078-2.797-2.238-6.133l-1.434-3.969c-.121-.039-.703.738-2.227 2.973-.41.602-.867 1.211-1.016 1.363-.258.258-.285.27-.551.238-.57-.062-1.613-.707-2.727-1.68-.273-.238-1.129-1.074-1.902-1.859-1-1.012-1.445-1.422-1.547-1.422-.48 0-.75 2.355-.383 3.316.078.199.125.379.105.398-.094.094-1.336-.777-1.547-1.086-.332-.484-.73-1.434-.832-1.977a2.4 2.4 0 0 0-.117-.422l-.074-.176-.258.277c-.316.348-.594.887-.773 1.516-.109.387-.137.648-.141 1.281 0 .496-.023.793-.062.793-.156 0-.547-.344-.809-.711-.484-.68-.773-1.437-.871-2.305-.086-.742-.219-.828-.562-.363-.523.711-.836 1.582-.84 2.348l-.004.438-.332-.352c-.602-.637-.867-1.398-.867-2.48-.004-.906.145-1.473.625-2.406.219-.43.457-1.012.551-1.34.09-.316.246-.723.348-.902.371-.648 1.152-1.195 2.047-1.426.551-.141 3.773-.094 4.473.066.949.215 1.102.301 1.816 1.02a42.28 42.28 0 0 1 1.68 1.875c.559.668 1.18 1.379 1.379 1.586l.359.379.125-.156c.199-.246.789-1.43.898-1.801.055-.187.098-.551.098-.809.004-1.473.703-2.621 2.074-3.398l.547-.312-.035-.652c-.043-.754-.164-1.34-.305-1.477-.062-.066-.305-.113-.766-.152-1.293-.105-2.812-.418-4.004-.828-2.023-.691-5.457-2.395-9.773-4.848-3.633-2.062-5.164-2.707-7.375-3.094a8.86 8.86 0 0 0-1.867-.141c-1.289 0-1.699.094-2.801.641-.895.441-1.398.82-1.578 1.195-.117.246-.168.492-.387 1.906-.039.25-.078.449-.09.438s-.051-.426-.09-.922c-.098-1.164-.137-1.32-.422-1.551-.891-.734-2.484-1.094-4.184-.945-.449.039-1.004.129-1.234.203-1.215.371-2.937 1.496-4.27 2.785-.734.711-1.762 1.941-1.805 2.16-.02.098-.094.543-.16.988-.156 1.043-.25 1.461-.32 1.461-.031 0-.062-.09-.066-.199l-.047-1.23-.035-1.027-.668-.445c-.742-.492-1.594-.891-2.332-1.09-.645-.176-1.797-.187-2.387-.027-2.191.594-4.066 2.82-5.559 6.605-.887 2.254-1.934 6.363-2.109 8.289-.117 1.285-.023 1.988.609 4.652.188.781.332 1.453.32 1.484-.074.227-1.223-3.074-1.785-5.141C1.5 74.426.492 68.957.184 65.414.078 64.219.035 59.035.117 57.797c.141-2.031.516-4.387.969-6.043.266-.957.957-3.074 1.48-4.512 2.34-6.445 6.945-12.336 13.402-17.156.664-.496 1.25-.961 1.301-1.035.09-.129.086-.141-.043-.195-.336-.137-5.234-.031-7.137.152-.355.035-.656.055-.672.039-.066-.066 2.098-.48 3.281-.625.426-.055 1.027-.113 1.336-.129l.559-.027-.465.055-1.742.234c-.156.027 1.176.047 2.957.047 3.629.004 5.473.109 8.09.473a113.28 113.28 0 0 1 6.473 1.238l2.242.527c1.547.363 2.184.535 3.527.949l.793.246.133-.328a2.41 2.41 0 0 1 .344-.574l.211-.25-.613-.191c-.559-.176-1.824-.516-5.109-1.371a66.49 66.49 0 0 0-2.254-.539l-1.098-.234-.781.352-1.312-.168-1.715-.227c-.625-.09-2.648-.25-3.891-.309l-1.117-.055 1.43.027c1.98.039 3.734.188 5.391.465.918.156 1.113.152 1.492-.012.262-.109.387-.125.762-.094.484.039 2.52.531 5.68 1.371 1.707.453 3.293.848 3.414.848.164 0-.176-.727-.727-1.551-.605-.906-1.266-1.664-2.84-3.258-.871-.879-1.566-1.621-1.539-1.645.066-.066 1.742.801 2.434 1.266a11.48 11.48 0 0 1 2.148 1.805c.723.75 1.211 1.434 1.801 2.512.215.395.547.906.742 1.141.469.57.617.836.699 1.242.141.691-.176 1.262-1.012 1.828a1.84 1.84 0 0 0-.457.48c-.75 1.285-2.219 6.219-3.234 10.891l-.348 1.605.285.316c.355.398.758 1.223.852 1.738.039.219.098.43.137.473.074.082 1.211.285 5.477.984 2.371.387 2.93.457 2.973.391a15.74 15.74 0 0 0 .273-.465l.219-.387.418-.066a149.89 149.89 0 0 1 2.16-.285c2.457-.309 5.383-.77 10.094-1.594.672-.117.871-.133 1.078-.078l.254.07.512-.449c.715-.629 4.082-4.109 5.113-5.289.883-1.008 1.574-2.414 1.813-3.687.125-.656.094-1.57-.066-2.051-.137-.41-.496-.883-.781-1.031-.734-.375-2.91-.277-5.164.238-1.316.301-1.426.355-1.742.855s-.676.828-1.367 1.234c-.91.535-1.301.848-2.668 2.148-1.383 1.309-1.859 1.684-2.215 1.738-.281.047-.988-.191-1.242-.418l-.195-.172.133-.176c.07-.098.461-.445.871-.777 1.898-1.531 2.887-3 2.887-4.289 0-.195-.031-.441-.066-.547-.062-.18-.094-.195-.34-.195-.27 0-1.246.305-1.508.473-.23.148-.172.332.238.723.453.43.422.441-.562.199-.703-.172-1.055-.172-1.25.008-.113.102-.113.109.008.348.07.137.289.406.492.605.363.355.363.359.168.336-.527-.059-.984-.168-1.391-.34-.477-.195-.684-.207-.785-.043-.117.184.023.453.453.879l.43.422h-.199a3.27 3.27 0 0 1-.637-.121c-.383-.105-.641-.207-1.367-.535-.102-.047-.379-.113-.609-.148-.512-.074-.887-.281-1.191-.648-.562-.687-.801-1.777-.496-2.266.133-.207.379-.309 1-.418 1.289-.223 2.273-.75 2.703-1.445.125-.199.129-.25.066-.531-.102-.434-.09-1.16.023-1.492.355-1.039 2.367-2.504 4.98-3.621 2.418-1.039 5.379-1.809 6.488-1.691l.406.043-.211.129c-1.277.773-2.215 1.527-3.094 2.492-.867.949-1.254 1.57-1.086 1.738.141.141 2.82-.566 3.273-.863.133-.086.461-.363.73-.613.914-.855 1.992-1.367 3.418-1.617.797-.141 1.949-.191 2.285-.098l.266.07-1.418.711c-1.316.66-1.977 1.078-1.867 1.188.023.023.305.078.621.117l.578.074-1.336.039c-.773.02-1.551.078-1.836.133l-1.184.223c-.84.164-2.605.613-3.363.859-.309.102-.566.176-.578.168-.043-.035.316-.879.543-1.266.328-.559 1.199-1.582 1.66-1.949.535-.422.875-.734.809-.734-.094-.004-.82.48-1.477.977-.738.559-1.797 1.648-2.414 2.477-.566.758-1.234 1.461-1.625 1.711-.266.172-.301.223-.402.602-.211.781-.77 1.117-1.582.957-.227-.043-.492-.078-.598-.074l-.594.016c-.328.008-.484-.02-.77-.152-.363-.164-.473-.168-.473-.012 0 .047-.195.266-.434.484-.66.613-1.383.922-2.453 1.039-.887.102-1.07.289-1.016 1.066.043.66.242 1.063.734 1.512.602.543 1.027.586 2.086.199.395-.145 2.383-1.234 2.754-1.512.27-.203 1.492-.777 2-.941.223-.07.551-.148.73-.172.316-.039.324-.035.371.141.027.098.051.516.051.926l-.004.746-.332.652c-.668 1.32-1.195 1.922-2.723 3.082-.582.445-.801.645-.758.695.086.105.949.457 1.125.457.316 0 .945-.512 2.133-1.73 1.195-1.227 1.652-1.598 2.457-2.004 1.473-.742 1.93-1.457 1.855-2.91-.031-.613-.176-.98-.461-1.168-.129-.086-.148-.121-.082-.156.344-.191 2.309-.75 4.137-1.176 1.395-.328 4.551-.391 4.844-.098.047.043.293.164.551.266 1.004.387 1.816 1.004 2.348 1.781.207.305.566 1.102.566 1.262 0 .063.074.434.164.828.207.91.234 2.066.063 2.828-.059.273-.137.531-.172.57s-.203 1.035-.375 2.211c-.242 1.668-.336 2.168-.426 2.266-.266.293-.48 2.262-.379 3.492.035.406.086.871.117 1.027s.031.328.004.379-.012.383.035.738c.23 1.785.25 2.137.25 4.145 0 2.039 0 2.066-.137 2.211-.113.125-.148.281-.234 1.109-.18 1.727-.32 2.566-.625 3.672-.082.289-.219.895-.309 1.34-.207 1.027-.203 1.023-.371 1.086-.121.047-.16.156-.305.883-.238 1.203-.34 2.09-.414 3.594-.051 1.039-.086 1.348-.156 1.402-.078.055-.082.18-.027.813.031.41.086.973.121 1.246.195 1.609.258 2.484.184 2.621-.109.203-.047.816.309 3.168.406 2.645.406 2.633.316 2.734-.055.07-.055.207-.004.574.184 1.289.277 1.777.441 2.324.398 1.32.48 1.648.43 1.746-.074.145.121.762.734 2.289.293.734.539 1.449.551 1.59a1.79 1.79 0 0 0 .121.504 50.74 50.74 0 0 0 1.465 2.965c.168.297.309.574.309.621 0 .262 1.332 2.211 1.992 2.918 3.426 3.688 4.688 4.691 7.313 5.809 1.457.621 4.637 1.051 6.738.91 1.559-.105 2.84-.418 4.027-.992 1.512-.73 2.367-1.426 3.328-2.703 1.336-1.777 1.805-3.301 1.727-5.574-.035-.918-.07-1.195-.277-1.992a15.73 15.73 0 0 0-.641-1.699c-.176-.379-.254-.508-.797-1.297-.281-.402-1.582-1.734-1.699-1.734-.035 0-.051.023-.039.047 1.152 2.25 1.359 3.715.824 5.824a6.49 6.49 0 0 1-1.164 2.445c-.715.949-1.551 1.617-2.828 2.258-1.289.648-2.273.875-4.008.922-1.348.039-2.094-.023-3.23-.266l-1.199-.25c-.617-.117-1.758-.512-2.512-.863-.512-.238-2.211-1.301-2.457-1.539l-.645-.543c-1.781-1.488-3.52-3.715-4.59-5.883-.848-1.723-1.039-2.437-2.047-7.691l-.496-2.551c-.48-2.242-.625-3.973-.629-7.406 0-2.34.055-3.598.234-5.168.113-1.004.215-1.527.492-2.52.371-1.328 1.086-4.328 1.305-5.477.117-.617.297-1.723.402-2.461l.223-1.422c.016-.047-.066-.215-.18-.371-.437-.602-.887-1.695-1.031-2.527a7.4 7.4 0 0 1 .02-2.059 4.08 4.08 0 0 1 .215-.68 6.55 6.55 0 0 0 .266-.809c.215-.848.742-2.73.832-2.965.117-.309.496-1.746.688-2.609.414-1.891.59-3.238.59-4.621 0-1.211-.051-1.633-.324-2.758-.246-1.008-.523-1.621-1.047-2.309s-.664-.816-1.469-1.348c-.902-.594-1.699-.875-3.633-1.281-.262-.051-.242-.059.262-.062.457-.008.664-.047 1.277-.254.574-.191.855-.254 1.293-.273.309-.016.461-.016.34 0-.719.078-1.926.402-1.926.516 0 .023.203.105.449.18s.621.203.828.285l.375.145.652-.238 1.422-.516c.422-.148.73-.289.684-.309s.016-.035.145-.035.246.02.266.055c.051.082-.238.219-1.078.508-1.297.445-1.691.621-1.691.742 0 .152 1.398 1.195 1.961 1.465a16.79 16.79 0 0 0 1.09.461c.359.141.91.395 1.227.57.578.32 1.27.871 1.191.953-.023.023-.504-.016-1.062-.086s-1.141-.133-1.281-.133c-.32 0-.324.023-.105 1.035.086.395.156.801.156.902 0 .367.195.637 1.176 1.641.887.906 1.254 1.359 1.168 1.441-.016.02-.402-.094-.859-.242-1.07-.355-1.145-.371-1.219-.277s-.168.617-.297 1.625l-.094.738.305.508c.457.762.699 1.277.68 1.441-.02.141-.035.137-.547-.195-.574-.367-.66-.395-.711-.227a226.2 226.2 0 0 0-1.246 4.563c-.25 1.066-.246 1.035-.102 1.113.453.242 8.047 1.547 11.582 1.992.566.07 1.113.156 1.215.188.172.055.164.059-.066.035a26.18 26.18 0 0 0-.684-.066 116.32 116.32 0 0 1-5.82-.801l-2.801-.5-3.437-.637c-.086-.023-.141.039-.234.27-.066.164-.152.344-.184.402-.098.168-.25 1.098-.25 1.516 0 1.625.941 3.672 1.918 4.164.816.414 2.488.43 3.867.035.121-.031.836-.199 1.59-.371 1.867-.422 2.016-.465 2.5-.707a25.05 25.05 0 0 1 1.203-.539c.426-.18 1.027-.445 1.336-.59s.617-.281.688-.305.336-.25.59-.508l.465-.469-.234-.434c-.129-.238-.207-.434-.18-.434.082 0 .262.215.262.316 0 .047.066.172.148.273.145.184.168.188.637.18.27-.004 1.383-.152 2.477-.328 2.344-.375 2.773-.461 3.078-.609.211-.105.242-.16.477-.828.324-.914.387-1.031.77-1.387l.316-.297-.375-1.664c-.961-4.227-1.789-7.266-2.512-9.207-.547-1.473-.797-1.914-1.27-2.23-.613-.414-.887-.812-.934-1.375-.043-.52.125-.945.582-1.461a6.79 6.79 0 0 0 .715-1.043c1.195-2.195 2.266-3.387 4.141-4.613.563-.367 2.711-1.496 2.852-1.496.02 0-.289.293-.691.648a36.88 36.88 0 0 0-3.168 3.211c-.848 1.02-1.703 2.367-1.703 2.695 0 .066.047.105.133.105.195 0 1.074-.211 3.008-.727 3.023-.805 5.465-1.395 5.918-1.43.34-.027.484-.012.637.066.344.18.684.18 1.82.016 2.148-.316 4.117-.461 6.781-.5 1.965-.027 1.82-.012-.824.086a56.03 56.03 0 0 0-5.492.465c-2.012.266-1.801.262-2.215.063l-.367-.176-.98.203c-.539.113-1.555.352-2.258.531l-5.586 1.508-.266.102.242.258a1.68 1.68 0 0 1 .32.508c.043.133.098.266.125.297.047.051.684-.121 1.848-.492.246-.078 1-.273 1.68-.434l2.352-.562c2.543-.613 6.223-1.297 8.188-1.52 2.539-.289 3.621-.34 7.031-.344 1.797 0 3.145-.023 2.988-.047-.742-.121-2.637-.305-3.641-.352-.82-.039-.961-.055-.527-.059 1.703-.027 4.949.355 6.547.77.57.148.574.148.203.121-1.527-.121-2.293-.16-4.012-.219-1.898-.059-3.359-.035-3.547.066-.117.063.145.309 1.148 1.07 1.07.813 2.148 1.672 2.148 1.711 0 .02-.051-.004-.109-.055-.824-.691-3.199-2.445-3.566-2.637-.133-.07-.363-.074-1.184-.027-1.246.074-3.055.23-3.605.313l-1.027.152a72.25 72.25 0 0 0-4.172.738c-1.57.328-5.055 1.117-5.664 1.285l-2.687.785c.008.008.672.125 1.477.266l1.711.316c.137.035.992.219 1.898.406 2.371.488 6.141 1.434 7.328 1.832l1.809.563c1.152.348 1.488.461 1.848.621a52.86 52.86 0 0 0 1.371.5c.684.246 1.344.5 1.461.563s.695.316 1.277.563 1.086.469 1.121.496.508.273 1.059.547 1.094.563 1.211.645.43.27.688.422c.559.328 2.754 1.773 2.871 1.887.047.047.266.203.488.352.621.418 2.262 1.711 2.949 2.324 1.117 1 1.938 1.934 2.828 3.215.176.258.32.449.32.43s-.039-.156-.094-.301-.082-.281-.066-.293c.039-.043.926 2.676 1.273 3.902.375 1.328.516 1.969.719 3.301.879 5.719.473 12.406-1.215 19.98-.75 3.344-1.621 6.355-2.281 7.875-.266.609-.477 1.012-.422.801.238-.895 1.164-5.027 1.16-5.191 0-.582-.691-3.902-1.195-5.758-1.266-4.652-2.75-7.488-4.754-9.098-1.055-.848-1.785-1.109-3.082-1.105-.777.004-.883.02-1.422.203-1.176.402-2.535 1.156-2.715 1.504-.078.145-.109.469-.141 1.344-.02.633-.059 1.164-.082 1.18-.051.031-.137-.391-.277-1.34-.145-.961-.262-1.363-.504-1.719-.254-.375-2.09-2.195-2.773-2.75-.828-.668-2.062-1.371-2.937-1.672-.617-.207-1.797-.297-2.844-.215-.707.059-.898.098-1.391.293-1.004.402-1.266.566-1.445.906-.047.09-.125.535-.176.988-.129 1.168-.164 1.172-.344.035-.152-.945-.238-1.164-.582-1.523-.746-.777-1.957-1.285-3.453-1.445-1.27-.137-2.68.055-4.375.602-1.629.523-2.684 1.031-5.547 2.656-3.227 1.836-4.93 2.727-7.27 3.809-1.633.754-2.461 1.066-3.535 1.336-1.223.305-1.996.426-3.199.5-.555.031-.836.074-.895.137-.133.129-.215.676-.215 1.398v.641l.203.031a13.57 13.57 0 0 0 .676.066c1.434.109 2.508.77 3.043 1.863.25.512.434 1.246.438 1.73 0 .457.172 1.117.438 1.668.324.684.625 1.184.715 1.184.074 0 .742-.75 2.055-2.305a30.08 30.08 0 0 1 1.426-1.59c.527-.523.594-.57 1.059-.727.922-.309 1.125-.328 3.145-.328 1.699 0 1.934.012 2.242.117.566.195.902.391 1.289.75.434.402.703.875.887 1.555a4.72 4.72 0 0 0 .246.715l.398.777c.367.711.504 1.305.504 2.184-.004 1.066-.23 1.762-.82 2.48l-.297.367-.039-.52c-.043-.539-.207-1.113-.457-1.613-.199-.387-.637-.93-.75-.93-.129 0-.215.254-.215.633 0 .719-.391 1.68-.996 2.441-.395.496-.605.664-.684.535-.023-.039-.062-.426-.082-.859s-.082-.961-.137-1.164c-.137-.52-.469-1.195-.746-1.512-.223-.258-.238-.262-.352-.164-.074.066-.117.191-.117.332 0 .305-.371 1.355-.629 1.785-.266.441-.855.992-1.352 1.258-.426.23-.449.211-.289-.238.051-.145.125-.504.164-.801.059-.453.055-.637-.031-1.141-.121-.715-.285-1.227-.426-1.316s-.422.164-2.137 1.863c-1.988 1.973-2.852 2.648-3.758 2.949-.703.238-.836.148-1.684-1.109l-1.227-1.75-.895-1.227c-.246-.34-.434-.488-.434-.344 0 .285.695 4.063 1.188 6.457.402 1.949.512 2.367.621 2.344.051-.012.469-.184.93-.383 1.961-.844 4.383-1.484 7.168-1.898 1.625-.238 4.785-.27 6.711-.066 2.52.27 5.285 1.195 7.066 2.371 2.195 1.445 3.578 3.43 4.141 5.934.246 1.098.219 2.742-.066 4.063-.477 2.203-2.32 4.563-4.855 6.211-1.668 1.09-4.145 1.996-5.703 2.086-.27.016-.437.016-.375-.004.098-.031.105-.09.066-.574-.023-.297-.07-.574-.102-.617s-.164-.078-.285-.078-.598-.047-1.051-.102-.844-.09-.859-.074-.051.344-.078.73c-.043.598-.035.707.047.754.063.035-.059.055-.34.047-.242-.004-.348-.016-.242-.031zm3.793-.195c.102-.031.469-.121.816-.199s.668-.16.703-.184c.043-.027.039-.152-.008-.398-.07-.355-.074-.359-.258-.324-.102.023-.574.039-1.055.039h-.867l.043.512c.023.285.059.535.07.563.039.063.336.059.555-.008zm-6.141-1.164c.078-.5.129-.918.117-.93a11.85 11.85 0 0 0-.707-.332c-.711-.32-1.426-.715-1.492-.82-.109-.18-.191-.012-.711 1.395-.176.477-.297.887-.27.91.105.094 2.445.668 2.762.68.156.004.16-.004.301-.902zm8.707.445a10.66 10.66 0 0 0 .813-.344c.223-.113.246-.148.199-.27-.062-.16-.031-.16-.855.008a6.72 6.72 0 0 1-.789.133c-.086 0-.172.023-.187.047-.055.09.105.637.184.637.043 0 .328-.094.637-.211zm-11.617-1.594c.23-.598.418-1.117.418-1.152s-.227-.234-.508-.445-.785-.641-1.121-.957-.621-.562-.633-.547-.219.512-.461 1.109c-.695 1.73-.668 1.684-.867 1.652-.094-.016-.379-.043-.637-.059-.453-.031-.461-.031-.277.066.102.051.375.207.602.344s.891.441 1.465.676c.824.336 1.105.426 1.324.414l.277-.012zm13.359.75c.367-.187.801-.473.637-.418-.055.02-.328.109-.609.207-.387.129-.512.199-.512.285 0 .152.07.141.484-.074zm-16.363-1.762a38.48 38.48 0 0 0 .566-1.371l.387-.98-.891-.918c-.488-.504-.91-.926-.937-.934s-.195.195-.375.461a59.25 59.25 0 0 1-1.402 1.934l-.109.137-.934-.145c-.516-.078-.945-.133-.961-.121s.055.086.156.164.336.281.523.457.742.602 1.23.945l.887.625.605.039a15.13 15.13 0 0 1 .73.063c.324.051.344.039.523-.355zm-28.937-.973c1.738-.227 5.012-.965 6.426-1.445 1.598-.543 2.977-1.176 7.891-3.617 3.168-1.574 4.355-2.191 4.355-2.27a.98.98 0 0 0-.125-.207c-.07-.098-.379-.699-.684-1.332s-.586-1.156-.613-1.156c-.09 0-3.348 1.555-4.488 2.145-4.797 2.465-6.207 3.105-8.59 3.898-2.687.891-3.43 1.094-5.402 1.48l-1.664.324c-.687.137-2.871.102-4.168-.062-1.371-.176-2.742-.457-3.57-.727-1.02-.336-2.27-1.055-2.621-1.516-.168-.219-.738-.66-.789-.609s.105.555.305.945c.625 1.242 1.379 1.961 2.852 2.73 1.535.797 3.953 1.359 6.609 1.535 1.18.078 3.215.023 4.277-.117zm26.273-1.297c.109-.164.262-.367.34-.453a8.22 8.22 0 0 0 .477-.652l.543-.789.207-.289-.336-.402c-.477-.57-1.164-1.531-1.445-2.023-.129-.23-.258-.441-.289-.469s-.16.094-.293.277c-1.254 1.707-1.617 2.168-1.699 2.168-.051 0-.582-.141-1.184-.316-1.469-.426-1.555-.449-1.594-.41-.074.074 1.5 1.941 2.383 2.824.492.492.52.508.871.555a31.48 31.48 0 0 1 1.016.152l.73.117c.039.004.164-.129.273-.289zm14.965-.578c.75-.082 1.453-.23 2-.414a12.41 12.41 0 0 0 2.191-1.074c.492-.336 1.336-1.141 1.602-1.52l.402-.57c.227-.312.633-1.348.77-1.961a17.5 17.5 0 0 1 .168-.707c.035-.133.063-.625.063-1.094 0-.773-.016-.918-.199-1.562-.113-.395-.309-.965-.441-1.27-.262-.609-.832-1.504-1.105-1.734-.195-.164-1.852-1.02-2.371-1.227-.828-.328-2.73-.812-3.891-.988-2.223-.34-5.793-.285-7.996.121-2.516.465-3.793.789-5.254 1.336-1.055.391-1.93.777-2.051.898-.059.059.254 1.098.457 1.527l.363.793.25.543.766-.363c1.305-.617 2.602-.965 4.723-1.262.918-.129 1.266-.141 3.297-.141 2.395 0 3.02.051 4.297.344 1.387.32 2.523.871 3.418 1.66 1.199 1.059 2.008 2.992 1.719 4.133-.121.477-.297.832-.766 1.531-.316.469-.656.82-1.445 1.5-.715.613-2.762 1.359-3.797 1.383-.309.004-.172.047.371.109.664.078 1.805.082 2.461.008zm-2.211-.469c.324-.051.773-.152.996-.223.602-.195 1.867-.84 2.055-1.047.016-.02.25-.23.516-.465.551-.488 1.246-1.465 1.473-2.066.285-.762.074-1.543-.664-2.418-1.027-1.223-2.816-2.055-5.371-2.5-.555-.098-.996-.117-2.957-.117-3.145.004-4.211.125-5.883.668-1.258.406-2.301.91-2.301 1.105 0 .082 1.234 1.684 1.773 2.305 1.617 1.852 3.527 3.297 5.445 4.125.867.371 1.414.547 2.02.641a12.62 12.62 0 0 0 2.898-.008zm-28.508-.461c.152-.098.043-.125-.125-.031-.09.051-.113.086-.062.086s.137-.023.188-.055zm.918-.324c.285-.102.391-.168.391-.25 0-.062-.016-.113-.031-.113-.078 0-.84.406-.84.449 0 .074.055.063.48-.086zm1.199-.473c.152-.066.328-.137.387-.16.066-.023.109-.102.109-.199 0-.191-.07-.18-.637.105-.293.145-.359.207-.359.336s.02.145.109.098c.063-.031.234-.109.391-.18zm1.301-.52l.504-.191v-.242c0-.137-.012-.246-.031-.246-.043 0-.859.441-.949.516-.07.055-.113.355-.051.355.012 0 .25-.086.527-.191zm10.43-.941l.867-1.168a7.86 7.86 0 0 1 .328-.426c.109-.109.109-.113-.102-.441-.227-.352-1.453-2.875-1.559-3.199-.051-.16-.102-.227-.16-.203-.047.016-.637.504-1.309 1.078s-1.254 1.051-1.293 1.063-.621-.137-1.281-.328l-1.254-.344c-.062 0 1.207 2.531 1.668 3.316.328.559.371.609.605.672a13.33 13.33 0 0 1 .727.223c.492.16 1.969.57 2.082.578.039 0 .344-.367.68-.82zm-9.09.387l.41-.191v-.543l-.445.195c-.441.195-.551.32-.551.605 0 .164.109.152.586-.066zm1.25-.555l.402-.184v-.316c0-.172-.016-.312-.039-.312s-.227.102-.461.227c-.41.223-.422.234-.441.5-.027.324 0 .332.539.086zm1.289-.574l.359-.18v-.34c0-.187-.012-.344-.031-.344a6.59 6.59 0 0 0-.496.234l-.469.23v.332c0 .387.004.387.637.066zm1.063-.496l.277-.145c.023-.02-.301-.594-.336-.594-.016 0-.109.043-.211.094-.172.09-.184.121-.184.438 0 .395.043.414.453.207zm.168-1.129c-.555-1.121-.887-1.758-1-1.898-.062-.082-.23-.449-.371-.812l-.34-.852c-.113-.258-.289-.816-.629-2.023-.156-.562-.324-1.07-.367-1.121-.125-.145-.238-.527-.727-2.488-.25-1.012-.469-1.844-.488-1.852-.164-.09-.223-.215-.352-.801-.082-.375-.262-1.266-.402-1.984s-.332-1.629-.434-2.02-.18-.762-.18-.82-.055-.148-.125-.195c-.152-.109-.266-.973-.402-3.098-.133-2.016-.16-2.289-.258-2.367-.047-.039-.098-.16-.117-.266-.066-.387-.16-1.977-.246-4.043-.07-1.754-.098-2.121-.184-2.215-.121-.133-.187-.738-.324-2.996-.109-1.816-.09-2.832.07-3.844.082-.52.082-.594-.004-.727s-.086-.195.063-.777c.09-.348.254-.891.371-1.203l.707-1.945c.273-.754.559-1.508.637-1.68.332-.762.441-1.141.363-1.266-.062-.102-.016-.234.309-.891.477-.965 1.012-1.887 1.574-2.723.367-.539.406-.621.25-.512-.105.074-.25.211-.32.305s-.879.934-1.793 1.867l-1.656 1.699-.004 1.383c-.008 1.242-.02 1.426-.148 1.84-.246.813-.676 1.555-1.332 2.281l-.281.313-.074 1.277c-.082 1.375-.055 4.395.055 5.723l.16 2.238c.34 5.117.867 9.066 1.598 11.984.359 1.426 2.41 7.762 2.754 8.504a36.37 36.37 0 0 1 .465 1.203 44.29 44.29 0 0 0 .723 1.836l.578 1.371c.535 1.289 1.988 4.109 2.109 4.109.016 0-.09-.23-.227-.516zm2.559-1.648c.258-.23.789-.687 1.176-1.012l.852-.715.145-.125-.207-.559c-.117-.309-.285-.723-.375-.918s-.285-.727-.441-1.176-.289-.828-.309-.848c-.035-.035-.23.078-1.281.762l-1.16.73-.379.23-.77-.332a24.59 24.59 0 0 1-1.023-.465c-.453-.238-1.086-.504-1.086-.453 0 .059.582 2.086.719 2.496.051.156.25.664.445 1.133.285.703.379.867.512.914a37.69 37.69 0 0 0 2.617.75c.051 0 .305-.184.566-.414zm21.984.121c-.129-.348-.859-1.258-1.012-1.258-.027 0-.062.113-.078.254-.02.238-.008.262.289.492.172.133.41.34.527.461.215.223.344.246.273.051zm-16.488-.863c.211-.102.215-.105.258-.625.023-.285.031-.535.012-.551-.062-.062-.98.418-.98.516 0 .125.367.766.438.766.031 0 .152-.047.273-.105zm15.18-.379c.047-.355-.016-.422-.836-.879l-.539-.301-.039.184c-.02.105-.035.281-.035.398 0 .203.02.219.668.547.367.188.688.344.707.344s.055-.129.074-.293zm-14.133-.129a6.83 6.83 0 0 1 .625-.254c.125-.043.242-.098.258-.121s.047-.25.07-.5l.047-.457-.211.047c-.117.027-.449.137-.742.25l-.527.207-.02.5c-.012.313.008.504.047.504s.238-.078.453-.176zm12.434-.789c0-.387-.008-.41-.172-.473-.25-.098-1.215-.371-1.23-.352a4.75 4.75 0 0 0-.039.449l-.02.434.555.141c.305.078.57.156.586.176s.098.031.176.031c.137 0 .145-.023.145-.406zm-10.539.184l.813-.184c.109-.023.141-.09.172-.328l.066-.434c.039-.18-.043-.18-.949-.012l-.695.133-.039.289-.059.492c-.016.109-.016.203-.004.203a20.91 20.91 0 0 0 .695-.16zm8.824-.594c.023-.242.031-.457.012-.473-.043-.047-1.461-.238-1.492-.203-.016.016-.035.211-.051.43-.023.473-.07.441.852.582l.598.098c.023.004.059-.191.082-.434zm-6.453.188l.637-.078v-.191c.004-.105.02-.297.043-.43l.039-.234-.43.039c-.594.051-1.277.141-1.305.168-.012.012-.039.211-.066.449l-.043.426.242-.035a21.56 21.56 0 0 1 .883-.113zm2.641-.398a3.22 3.22 0 0 0 .047-.422l.004-.172-.449.004c-.25 0-.641.016-.867.039l-.41.039-.043.363c-.023.203-.027.387-.012.414s.398.031.855.016l.832-.031zm1.949.266c.016-.043.035-.234.047-.422l.016-.344-.809-.043c-.445-.023-.824-.023-.848 0s-.055.211-.066.418l-.02.375.371.035c.727.07 1.281.063 1.309-.02zm-16.461-2.133c.426-.281 1.023-.66 1.328-.844s.566-.375.586-.422c.027-.074-.219-1.004-.461-1.75-.051-.152-.164-.754-.25-1.34l-.176-1.082c-.031-.027-1.039.684-1.348.949-.219.188-1.238.949-1.406 1.055-.094.055-1.32-.434-2.812-1.121-.254-.117-.473-.199-.488-.18-.031.031.633 2.77.855 3.52l.121.406.422.18c.23.102.672.301.98.449 1.035.488 1.68.75 1.773.719.055-.016.445-.262.875-.539zm-1.18-4.191l1.086-.809c1.172-.879 1.211-.91 1.211-1.07 0-.086-.109-.867-.246-1.742l-.473-3.223c-.02-.145-.051-.262-.066-.262s-.219.137-.449.309l-1.484 1.09-.996.73c-.312.23-.609.422-.652.422s-.18-.066-.301-.148a14.59 14.59 0 0 0-.902-.504c-.914-.48-1.176-.625-1.605-.887-.199-.117-.379-.199-.402-.176s.035.34.129.707.324 1.449.512 2.41l.402 1.93c.059.168.168.234 1.074.633l2.48 1.07c.02.004.328-.211.684-.48zM3.547 80.5l-.285-1.328c-.887-3.926-1.703-9.309-2.059-13.539-.141-1.656-.16-6.723-.031-7.902.215-2 .488-3.293 1.016-4.855.406-1.191.695-1.895.992-2.418.102-.176.27-.488.379-.695.445-.859 1.445-2.219 2.324-3.152 1.457-1.551 3.016-2.73 6.145-4.641.953-.586 2.992-1.594 4.188-2.074l1.152-.48c.512-.23 2.625-1.016 4.449-1.648l3.516-1.176 1.617-.5c.953-.309 4.148-1.168 5.531-1.488a101.85 101.85 0 0 1 4.41-.84l.199-.031-.211-.277c-.117-.156-.262-.434-.32-.621s-.133-.344-.164-.344c-.102 0-1.922.313-2.41.414l-2.43.492c-1.875.379-5.496 1.25-6.812 1.637-1.945.574-3.461 1.047-4.203 1.316l-1.727.621c-.484.172-1.059.398-1.273.496s-.687.297-1.047.441c-.711.281-3.027 1.398-3.434 1.656-.141.09-.523.316-.852.504-1.098.633-3.512 2.273-4.957 3.375-2.703 2.063-4.602 4.367-5.477 6.664-.332.867-.746 2.352-.926 3.305-.297 1.563-.562 4.043-.664 6.121-.148 3.172.285 8.301 1 11.891l.438 2.211c.207 1.043.496 2.359.648 2.926l.563 2.113c.293 1.109.844 2.82.887 2.777.016-.016-.062-.441-.172-.949zm121.434-1.172a78.22 78.22 0 0 0 1.801-7.687c.176-1 .496-3.07.629-4.047.715-5.348.555-10.551-.473-15.168-.5-2.242-1.457-4.219-2.844-5.852-1.105-1.312-2.918-2.906-4.648-4.094-.301-.207-.641-.449-.758-.543-.285-.23-1.695-1.148-2.422-1.578l-.91-.559c-.465-.301-2.684-1.402-3.645-1.809l-1.016-.434a31.44 31.44 0 0 0-2.145-.824l-1.434-.508a30.45 30.45 0 0 0-1.586-.508l-1.727-.527c-.371-.117-1.406-.395-2.305-.617l-1.82-.461c-.828-.258-7.574-1.625-8.031-1.625-.043 0-.113.145-.156.316s-.168.441-.285.598-.195.305-.176.336.133.059.254.059.336.027.477.059.754.148 1.352.254c1.176.211 2.762.551 3.457.738l1.398.375c2.188.582 1.871.488 5.445 1.609a112.28 112.28 0 0 1 6.535 2.313c2.105.82 4.879 2.082 5.496 2.504a20.84 20.84 0 0 0 .82.508c.945.559 2.441 1.543 3.176 2.086 1.238.922 2.738 2.313 3.266 3.031.102.137.289.363.414.504.262.293 1.047 1.473 1.402 2.109a22.61 22.61 0 0 1 1.234 2.758c.406 1.199.813 2.957.996 4.305.336 2.504.285 6.832-.125 10.645-.156 1.445-.629 4.828-.902 6.484-.219 1.297-1.137 5.977-1.309 6.66-.152.598-.152.895 0 .426.063-.187.313-1.016.563-1.836zm-68.914 1.715c.484-.406.43-1.238-.18-2.68-.266-.629-.398-.855-.734-1.246l-1.277-1.531-1.152-1.387-.625-.754c-.637-.785-.965-1.176-.996-1.18-.02 0-.023.23-.012.516.035.902-.219 1.551-.797 2.039-.234.195-.332.336-.414.578-.055.176-.141.398-.191.496-.086.172-.082.184.238.527.793.859 2.766 2.844 3.211 3.227l.57.504c.176.172 1.027.719 1.336.859.398.18.828.191 1.023.031zm18.391-.059c.809-.371 1.75-1.156 3.387-2.816 1.383-1.402 1.984-2.055 1.969-2.133-.004-.039-.086-.266-.176-.504a1.51 1.51 0 0 0-.5-.734 2.33 2.33 0 0 1-.539-.715c-.187-.383-.207-.473-.227-1.117l-.023-.703-.242.273c-.133.152-.426.508-.656.793s-.457.563-.504.609-.242.285-.434.527a13.41 13.41 0 0 1-.582.691 9.04 9.04 0 0 0-.504.602c-.148.195-.543.676-.879 1.074-.477.563-.68.863-.918 1.363-.348.727-.555 1.469-.555 1.973 0 .379.168.82.348.926.211.121.633.078 1.035-.109zm-17.41-1.348l.926-1.328.844-1.234.277-.402-.18-.34c-.441-.848-1.367-1.824-2.055-2.168-.465-.238-1.121-.402-1.211-.309s-1.062 1.992-1.062 2.078c0 .043.266.398.594.789.363.43.664.859.77 1.098l.313.688c.172.375.379 1.172.379 1.461 0 .121.012.207.023.191s.188-.25.383-.523zm15.934-.539c.188-.734.648-1.59 1.25-2.316l.613-.758c.047-.062-.078-.359-.461-1.098-.285-.559-.543-1.039-.57-1.07-.082-.094-.742.07-1.215.305-.719.352-1.695 1.41-2.113 2.289l-.109.23.348.512.977 1.418.859 1.242.234.336.043-.348c.023-.191.09-.527.145-.742zm-27.754.277c.016-.914.203-1.617.625-2.328.188-.32.434-.695.543-.832.164-.207.211-.336.273-.777.07-.504.34-1.195.59-1.523.086-.109.082-.141-.062-.328-.234-.309-.656-.535-.984-.531-.645.012-1.492.652-1.863 1.406-.121.246-.293.727-.379 1.07-.145.555-.156.707-.129 1.43.031.867.109 1.211.426 1.883.164.344.801 1.137.914 1.137.02 0 .043-.273.047-.605zm39.449.199c.695-.789 1.004-1.742.953-2.953-.055-1.285-.387-2.18-1.051-2.855-.832-.84-1.555-.941-2.184-.312l-.297.297.184.293c.219.344.402.883.512 1.488.063.363.125.5.371.824.406.535.844 1.445.965 2.008.055.254.102.727.109 1.043s.023.574.043.574.199-.184.395-.406zm-35.922-.246c-.113-.539-.078-1.711.07-2.211.059-.187.258-.648.441-1.027.516-1.035.645-1.699.418-2.145-.078-.16-.215-.273-.504-.422-.215-.109-.465-.203-.551-.203-.453 0-1.102.574-1.418 1.262-.465 1.012-.449 2.363.031 3.402.355.758.672 1.176 1.141 1.504.238.168.441.301.449.289s-.027-.211-.078-.449zm32.602-.094c.461-.43.746-.914.992-1.668.172-.527.191-.668.191-1.348 0-.672-.02-.82-.184-1.301-.203-.605-.422-.957-.75-1.219-.547-.43-.848-.465-1.387-.16-.398.223-.488.348-.531.727-.047.465.102.984.551 1.945.207.441.414.988.461 1.211.137.648.105 1.617-.074 2.242l-.043.148.203-.117c.113-.066.371-.273.57-.461zm-39.281-.238c0-.242.203-.91.422-1.391.215-.473.496-.898.828-1.266.16-.176.25-.352.305-.59.195-.891.488-1.582.82-1.965l.195-.223-.176-.18a1.14 1.14 0 0 0-.367-.25c-.242-.078-.738.008-1.094.188-.367.188-1 .848-1.191 1.246-.457.945-.543 1.191-.605 1.758-.105.93.078 1.773.559 2.566.215.355.305.387.305.105zm45.781-.395c.25-.484.402-1.168.402-1.82 0-.48-.031-.687-.184-1.145-.285-.875-.59-1.422-1.02-1.836-.437-.426-.684-.562-1.18-.664-.336-.07-.383-.066-.648.07-.441.219-.457.293-.156.672.297.371.598 1.105.738 1.793.086.43.125.508.367.746.148.145.387.457.531.695.246.418.617 1.426.617 1.68 0 .066.031.199.066.293l.066.172.125-.184c.066-.102.191-.312.273-.473zM3.789 74.516c.453-2.187 1.25-4.766 1.992-6.453.398-.906.82-1.785.887-1.836.02-.02.199-.309.402-.645a9.28 9.28 0 0 1 1.977-2.312 4.93 4.93 0 0 1 1.645-.902c1.168-.387 2.434-.305 3.777.246.586.238.707.301 1.508.813l.387.242.031-.199c.02-.109.047-.48.07-.824.07-1.187.348-2.879.625-3.824.418-1.414.863-2.699 1.121-3.207.469-.934.961-1.809 1.105-1.965.074-.086.281-.352.465-.598.395-.539 1.363-1.477 1.895-1.836.215-.145.402-.277.422-.301.117-.148.973-.605 2.176-1.168.758-.355 2.363-.855 3.766-1.176 1.238-.281 2.828-.41 4.078-.332.426.027.789.039.801.027.039-.039.301-1.18.633-2.766l.844-3.734c.391-1.504.844-3.074 1-3.488.086-.223.32-.867.52-1.43.355-1.02.672-1.852.84-2.211.215-.469.223-.527.086-.527-.375 0-3.176.504-4.164.746l-4.074 1.059c-1.289.371-4.117 1.238-4.328 1.328-.117.051-.832.293-1.586.539l-1.555.531c-.105.047-.832.32-1.621.609-1.523.563-3.758 1.457-4.168 1.668-.137.074-.672.324-1.184.566s-1.074.523-1.246.637-.66.414-1.09.668c-1.02.605-2.344 1.453-2.469 1.578-.055.055-.215.172-.355.258-.746.465-2.34 1.875-3.137 2.777-1.805 2.039-2.949 4.184-3.75 7.043l-.148.547c-.164.586-.395 2.059-.508 3.25-.301 3.129.004 9.215.652 13.105.035.188.145.914.254 1.617.336 2.215.738 4.473.828 4.656.063.125.094.008.234-.84l.363-1.937zm121.25 1.605l.254-1.367c.297-1.41.727-4.465.992-7.035l.188-1.68c.207-1.535.281-5.969.133-7.687-.176-2.012-.77-4.824-1.254-5.941l-.434-1.031c-.25-.598-.652-1.387-.871-1.711-.059-.086-.262-.402-.453-.703-1.16-1.867-3.266-3.867-5.738-5.453l-1.121-.723c-.273-.18-.598-.375-.715-.434s-.434-.25-.699-.418c-.555-.355-3.027-1.539-3.937-1.887a21.35 21.35 0 0 1-1.496-.621c-.102-.043-.703-.27-1.336-.496l-1.559-.574c-.664-.266-3.207-1.105-5.133-1.699l-2.117-.652a131.6 131.6 0 0 0-3.641-.949 1.24 1.24 0 0 1-.281-.082c-.441-.195-4.699-.953-4.699-.836 0 .055.34.969.777 2.086.609 1.559 1.391 4.051 1.715 5.457l.406 1.73c.199.777.531 2.285.781 3.547l.27 1.32c.008.008.359-.012.785-.039 1.957-.141 4.18.211 6.719 1.055.656.219.703.238 1.797.754 1.875.891 3.328 2.137 4.438 3.82a10.08 10.08 0 0 1 1.145 2.125l.34.809c.512 1.121 1.293 4.945 1.305 6.375 0 .227.02.426.039.445s.266-.109.551-.293c2.121-1.371 4.152-1.578 5.941-.613.289.152.637.387.777.52s.355.316.477.414c.273.215.48.453 1.016 1.16s.809 1.152 1.34 2.207c.477.941.574 1.172.98 2.316a19.98 19.98 0 0 0 .328.902c.133.316.191.488.379 1.156.473 1.66 1.02 4.047 1.215 5.316.07.438.133.684.16.613a28.18 28.18 0 0 0 .238-1.203zm-60.367.242l1.73-1.27a36.59 36.59 0 0 0 .992-.742c.438-.344.438-.332.324-1.281-.156-1.289-.309-2.855-.309-3.125-.004-.164-.016-.297-.035-.297a8.69 8.69 0 0 0-.605.355l-1.305.797-1.336.801-.605.367-.312-.176c-.176-.094-.48-.289-.684-.426s-.605-.406-.902-.59l-.992-.641c-.254-.172-.473-.293-.492-.277-.039.043.215 3.77.305 4.484l.07.539.383.191c.207.105.449.242.539.305s.645.367 1.238.68a28.65 28.65 0 0 1 1.207.656c.07.051.16.082.195.07s.301-.203.594-.422zm5.824-.477c.523-.867 1.313-1.652 2-1.992.281-.141.645-.273.809-.297a4.01 4.01 0 0 0 .402-.066c.262-.074.188-1.277-.117-1.977-.324-.734-.977-1.445-1.621-1.773a3.24 3.24 0 0 0-1.715-.367l-.574.016-.02 1.496c-.027 2.094.137 5.023.301 5.43.031.078.082.141.113.141s.219-.273.422-.609zm-11.328-.117c-.152-.59-.508-2.324-.777-3.785a31.58 31.58 0 0 1-.234-1.543c-.082-.645-.129-.855-.195-.855-.051 0-.281.102-.52.227-.621.324-1.281 1.043-1.543 1.668-.223.535-.352 1.316-.289 1.738.035.246.066.293.211.324 1.051.215 1.57.5 2.293 1.262a7.99 7.99 0 0 1 .84 1.074c.184.297.34.531.352.523s-.051-.293-.137-.633zm-9.004-1.785c.004-.418-.246-.934-.512-1.055-.473-.215-2.656-.418-4.496-.418-1.891 0-2.414.113-2.707.594-.078.121-.125.234-.109.25s.188-.066.387-.184c.352-.215.801-.348 1.152-.348.246 0 .477.094.75.313l.234.184.355-.215c.477-.289.941-.391 1.285-.289.336.102.676.32.883.566l.168.195.238-.187c.43-.344.836-.406 1.328-.215.582.227.922.629.922 1.094 0 .152.016.18.063.109.031-.051.059-.227.059-.395zm29.41-.266c.145-.219.57-.512.867-.59a1.29 1.29 0 0 1 1.152.215l.316.227.23-.254c.574-.637 1.359-.703 2.148-.184l.289.195.285-.219c.352-.27.637-.328 1.121-.23.414.086.68.207.891.398.188.172.25.082.117-.168-.258-.477-.824-.605-2.672-.602-1.828 0-3.992.199-4.512.414-.355.145-.664.969-.504 1.348l.082.203.043-.301c.023-.164.09-.367.145-.453zm-.426-.441c.18-.375.461-.609.836-.691.91-.203 2.59-.34 4.23-.348 1.629-.004 2.004.031 2.508.242.109.047.109.035-.008-.195-.164-.312-.434-.586-.898-.895-.77-.516-1.723-.66-3.656-.559-1.461.078-2.039.172-2.609.438-.746.344-.906.637-.902 1.648 0 .52.027.676.152.965l.152.34.043-.359c.027-.199.094-.461.152-.586zm-28.469.559c.113-.246.137-.414.137-.949 0-.977-.148-1.242-.895-1.621-.898-.453-4.148-.625-5.402-.285-.691.188-1.543.809-1.781 1.301-.051.109-.09.203-.082.211s.141-.027.293-.082c.484-.168 1.34-.227 2.676-.187 1.793.055 3.516.242 4.078.445.379.141.656.578.746 1.18.023.156.055.281.07.281s.09-.133.16-.293zm13.492-2.59l.965-.582.965-.582 1.074-.645.418-.254v-.625c0-1.059.18-2.684.441-4.004.07-.355.113-.656.098-.676-.031-.027-.91.457-.973.539-.02.023-.453.293-.965.602l-2.41 1.473c-.508.324-.508.324-.828.039l-1.121-.992-1.398-1.246c-.574-.516-.809-.703-.844-.672-.035.039.074 3.027.168 4.457l.074 1.199.988.637 1.453.949c.711.473 1.176.77 1.211.773.02 0 .328-.176.684-.391zm-6.445-4.941a22.26 22.26 0 0 0-.082-.832l-.039-.293-.652.215c-.984.316-2.527.578-3.422.582-.219 0-.316.02-.277.059.09.094 1.836.539 2.547.648.844.137.922.145 1.484.156l.484.008zm14.047.449c.691-.09 2.824-.594 3.031-.715.074-.043-.016-.059-.301-.062-1.059-.004-2.523-.27-3.652-.652-.437-.152-.82-.254-.844-.23a16.94 16.94 0 0 0-.234 1.707c0 .082 1.25.051 2-.047zm-7.168-1.102l1.371-.836 1.719-1.047.875-.523.078-.395c.043-.215.199-.883.348-1.484s.328-1.43.398-1.836c.129-.773.316-2.367.316-2.707v-.191l-.172.117a28.82 28.82 0 0 1-.855.539l-1.617 1.004-1.465.906-1.203.754-.672.434-2.219-2.223c-1.805-1.812-2.223-2.199-2.258-2.105-.023.063-.078.457-.121.879-.094.875-.059 2.496.094 4.621l.094 1.336.297.281a181.19 181.19 0 0 0 1.992 1.781l1.695 1.504.387-.246c.215-.137.629-.391.918-.562zm-10.187-.02c.863-.09 1.609-.25 2.641-.562l.516-.156v-.262c0-.535-.082-1.289-.141-1.289-.031 0-.203.121-.379.273-.625.531-1.961 1.258-3.262 1.777-.863.34-.785.367.625.219zm20.363.043c-.035-.027-.48-.227-.992-.445s-1.09-.473-1.277-.578c-.508-.273-1.547-.973-1.824-1.223-.176-.156-.254-.195-.289-.137a16.2 16.2 0 0 0-.336 1.395c-.027.16 1.91.73 2.977.879 1.082.152 1.844.199 1.742.109zm-21.336-.449c1.121-.453 1.828-.777 2.082-.953.145-.102.141-.102-.133-.102-.969-.004-3.168-.527-4.23-1.012a19.49 19.49 0 0 0-.687-.297c-.605-.242-1.605-.754-1.641-.836-.023-.066.031-.102.199-.133.391-.066 1.57-.336 2.191-.5.637-.168.828-.23 1.211-.398.137-.059.355-.141.484-.184s.234-.094.234-.121-.137-.078-.297-.121c-1.035-.27-2.645-1.117-4.109-2.168l-.949-.695c-.016-.023-.203-.199-.414-.398L47 56.949h1.266c1.355 0 2.543-.074 3.336-.211.484-.086 1.645-.461 1.59-.516-.016-.016-.668-.164-1.449-.328-1.336-.285-2.238-.492-3.102-.715-.203-.051-1.059-.25-1.898-.441l-1.992-.473a203.63 203.63 0 0 0-2.363-.613l-2.18-.582c-.152-.051-.852-.254-1.555-.445-.992-.273-3.848-1.184-4.488-1.43-.074-.027-.285.164-.773.695a33.15 33.15 0 0 0-.996 1.137l-.492.594c-.156.172-.582.797-.98 1.434-.203.32-.441.988-.637 1.758-.187.746-.551 2.508-.523 2.535.012.012.203-.117.426-.285a6 6 0 0 1 2.023-.973c.57-.148.727-.164 1.867-.168 1.32-.004 1.824.059 2.996.367.844.223 2.805 1 3.57 1.418l.59.313a37.68 37.68 0 0 1 1.914 1.094c.23.141.434.258.453.258.039 0 .813.434 1.492.84a21.61 21.61 0 0 0 1.09.594c.359.184.668.352.684.375.063.086 3.168 1.613 3.828 1.883l1.059.441c.207.086.426.16.488.16s.605-.195 1.203-.437zm22.93.23a51.93 51.93 0 0 1 1.148-.5c1.254-.535 4.91-2.418 5.758-2.965.238-.156 1.762-.996 1.805-.996.027 0 .059-.023.07-.051.027-.07 1.168-.703 2.355-1.312.922-.473 2.758-1.18 3.578-1.379 1.984-.488 3.773-.484 5.262.008.395.133.695.266.832.375.039.031.301.207.582.395l.508.34-.035-.199c-.148-.812-.469-2.309-.516-2.395-.027-.059-.09-.23-.133-.387-.176-.621-.465-1.297-.711-1.68-.293-.461-1.266-1.711-1.617-2.086-.129-.137-.375-.418-.547-.621a7.39 7.39 0 0 0-.566-.602l-.254-.223-1.52.516c-.836.285-2.082.676-2.77.871l-1.687.492a60.7 60.7 0 0 1-2.211.59l-2.148.566c-.203.066-1.043.273-1.867.465l-2.023.473-3.594.836c-.625.133-1.137.258-1.137.281s.121.078.266.121a1.73 1.73 0 0 1 .328.133c.125.09 1.316.297 2.102.363.441.035 1.379.066 2.086.066l1.289.004-.406.375c-.367.34-.707.605-1.773 1.387-.832.609-2.492 1.469-3.379 1.754-.316.098-.574.199-.574.223 0 .156 2.344.859 3.824 1.148l.555.109-.152.121c-.195.156-.809.465-1.578.785l-.891.383c-.875.398-2.828.859-4.02.941l-.238.02.344.203c.184.113.824.402 1.418.641l1.27.531c.238.117.543.082.969-.121zm-59.512-1.066c.418-2.27.91-4.211 1.449-5.695.531-1.465.715-1.875 1.266-2.84.957-1.68 1.734-2.418 3.824-3.637a25.62 25.62 0 0 1 1.059-.59c.5-.254 2.184-.883 2.832-1.059 1.184-.32 2.754-.527 4.004-.527.398-.004.758-.023.797-.047s.133-.172.207-.324.277-.418.445-.598.305-.352.305-.387c0-.062-.195-.09-1.246-.172-1.406-.109-4.086.336-6.129 1.016a22.77 22.77 0 0 0-2.43 1.027c-.672.352-1.73 1.121-2.328 1.691-.609.586-1.422 1.625-1.852 2.367-1.059 1.824-1.93 4.637-2.23 7.191-.094.809-.195 3.48-.129 3.328.016-.035.086-.371.156-.746zm94.465-.492c-.023-.551-.07-1.199-.102-1.437l-.16-1.184c-.414-3.094-1.453-5.895-2.918-7.844-.73-.973-1.82-1.926-3.07-2.68-.473-.289-2.078-1.008-2.691-1.207-1.586-.52-3.398-.898-4.793-1-.852-.062-2.57 0-2.668.098-.016.02.125.191.316.387a3.92 3.92 0 0 1 .531.668l.18.313 1.055.039c1.996.074 3.355.34 5.129 1 1.539.574 2.871 1.305 4.199 2.313.641.484.844.695 1.379 1.41.586.789.809 1.172 1.301 2.238.871 1.902 1.59 4.359 2.059 7.047.227 1.297.309 1.242.254-.16zm-54.941-.109c.121-.07.41-.289.645-.48l.43-.352-.039-.621c-.059-.961-.109-1.34-.172-1.34-.031 0-.148.066-.258.148-.426.324-1.172.625-2.211.906-1.059.285-1.637.344-3.23.34-.797-.004-1.445-.004-1.445.004s.414.176.918.375l1.105.441c.313.133 1.582.453 2.207.551 1.23.199 1.762.207 2.051.027zm17.289-.027c.996-.172 1.57-.332 2.723-.766 1.734-.656 1.734-.613.031-.609-1.598.004-2.113-.051-3.211-.336-.785-.203-1.137-.332-1.762-.648-.242-.121-.457-.219-.488-.219s-.141.379-.258.844l-.207.844.148.16c.082.09.371.328.645.535l.5.371.664-.039c.367-.023.914-.082 1.215-.137zm-55.383-1.551c1.094-1.168 2.68-2.395 3.699-2.863a10.65 10.65 0 0 1 1.879-.637c.523-.117 1.902-.176 2.207-.09.121.035.508.137.855.23s.914.305 1.246.469l.605.301.043-1.008c.043-1.074.184-2.266.34-2.875.371-1.449 1.328-3.191 2.586-4.711.199-.238.238-.336.227-.52l-.016-.23-.84.043c-1.645.082-2.887.273-3.926.605l-.934.297c-.422.137-1.637.629-1.68.684-.02.02-.242.148-.5.277-1.312.676-2.531 1.551-3.16 2.277-1.074 1.234-1.809 2.695-2.645 5.262a30.78 30.78 0 0 0-.828 3.23l-.043.246.207-.246c.117-.133.422-.469.676-.742zm92.188.34c-.336-1.453-.98-3.652-1.246-4.262-.062-.137-.215-.496-.336-.797-.242-.586-.691-1.504-.793-1.621-.031-.039-.227-.324-.426-.633a7.81 7.81 0 0 0-.828-1.027c-.941-.953-3.191-2.309-4.746-2.859-1.93-.68-3.211-.937-5.203-1.031l-.848-.043-.027.227c-.023.207.012.277.375.707 1.063 1.273 2.086 3.195 2.457 4.621.172.648.289 1.633.332 2.746l.039 1.039.418-.211a9.03 9.03 0 0 1 1.77-.664c.578-.152 1.98-.223 2.34-.117a20.56 20.56 0 0 0 .715.184c.961.234 1.68.559 2.641 1.199 1.023.68 2.047 1.578 2.898 2.543.305.348.566.609.574.582s-.035-.289-.105-.582zm-56.969-.531c1.41-.199 2.516-.566 3.332-1.109l.48-.316-.035-.5a19.81 19.81 0 0 0-.066-.824l-.031-.324-.754.379c-.414.207-1.176.617-1.691.91s-1.258.668-1.648.836c-.641.277-2.102.727-2.703.832-.25.047-.242.121.02.172.406.07 2.469.035 3.098-.055zm24.223.008c.066-.039-.145-.117-.715-.266-1.672-.43-2.738-.883-3.949-1.684-1.379-.91-1.57-1.023-1.617-.949-.055.09-.453 1.688-.426 1.711.012.012.16.105.328.211.617.391 1.906.793 3.105.973.645.098 3.125.102 3.273.004zm-48.418-2.09c.156-.953.598-2.937.836-3.75.215-.742.434-1.262.641-1.531.094-.125.273-.379.395-.566.297-.461 1.023-1.332 2.035-2.449.461-.504.828-.93.813-.945s-.383-.008-.82.012l-.793.039-.371.402c-.203.223-.473.563-.602.754a21.46 21.46 0 0 1-.492.711c-.379.516-1.242 2.285-1.418 2.91-.211.746-.344 1.742-.402 2.965-.043.902-.027 2.367.023 2.309.008-.008.078-.395.156-.859zm69.566-.793c-.02-1.059-.066-1.883-.129-2.27-.172-1.074-.598-2.348-1.074-3.219a20.12 20.12 0 0 0-1.676-2.445l-.414-.484-.477-.004a9.81 9.81 0 0 1-.832-.051c-.199-.023-.359-.02-.359.012s.23.297.516.598c.684.723 1.426 1.551 1.535 1.711a21.24 21.24 0 0 0 .441.566c.871 1.074 1.328 2.016 1.676 3.465.199.816.688 3.41.688 3.648 0 .074.031.137.07.137.051 0 .063-.453.035-1.664zm-44.363.906c.43-.25 1.203-.668 1.715-.93l.934-.477.02-.504.016-.5-.328.152c-.676.309-2.031.527-3.246.527-1.461-.004-3.527-.297-5.039-.715-.199-.055-.395-.09-.43-.078-.078.027.387.387 1.355 1.059 1.238.848 3.301 1.797 4.16 1.906.031.004.41-.195.844-.441zm20.781.16c.68-.223 2.078-.902 2.758-1.344 1.152-.746 1.871-1.293 1.699-1.293a4.82 4.82 0 0 0-.52.156c-.5.172-1.637.414-2.496.531-.773.109-2.402.098-3.117-.023-.578-.098-1.621-.422-1.785-.555-.043-.035-.09-.055-.102-.043-.062.066-.227 1.074-.184 1.109.234.172 2.004 1.32 2.32 1.508.457.27.461.27 1.426-.047zm-10.715-.707l1.453-.902 2.93-1.809c.363-.219.746-.477.855-.578l.199-.184v-1.703c0-.937-.031-2.012-.066-2.387-.09-.941-.25-2.297-.277-2.324-.012-.012-.227.109-.477.27l-2.605 1.648-1.156.742-.691.445-.402-.422c-.5-.523-1.859-2-2.473-2.68-.25-.281-.477-.512-.5-.512s-.109.188-.191.418a6.76 6.76 0 0 1-.25.641c-.059.117-.27.664-.465 1.211l-.656 1.777a19.91 19.91 0 0 0-.496 1.508l-.195.73 2.324 2.301L63.777 60c.008 0 .371-.223.809-.496zm-10.059-1.496c.906-.062 1.852-.27 2.305-.5.371-.184.422-.23.391-.359-.023-.086-.242-.148-1.051-.312-.562-.113-1.301-.277-1.648-.359l-.625-.156-.48.184c-1.277.488-2.141.648-3.957.734-.883.039-.914.047-.687.125.656.23 2.566.551 3.82.641 1.031.074 1.012.07 1.934.004zm22.848-.113c.734-.133 1.941-.441 2.137-.555.074-.039-.195-.07-.879-.102-1.875-.082-2.605-.203-3.711-.613l-.672-.25-1.043.23c-1.406.316-1.34.289-1.387.52-.035.176-.016.211.168.32.297.172 1.121.438 1.688.535.719.129 2.758.078 3.699-.086zM57.27 56.266c.02-.258.043-.727.047-1.047.008-.535 0-.578-.102-.539-.391.164-.941.293-1.496.355-.812.09-1.512.012-2.969-.32-2.039-.465-3.602-.98-4.793-1.582a28.42 28.42 0 0 0-1.215-.535c-.875-.371-.914-.395-1.43-.902-.527-.516-.531-.52-.965-.586l-.777-.137a482.54 482.54 0 0 0-6.984-1.098c-.242-.027-.328-.004-.496.125-.199.148-1.059.477-1.258.477-.121 0-.477.301-.477.406 0 .051.512.246 1.387.527l1.621.535c.129.047.953.289 1.836.531l1.785.496c.504.152 4.809 1.23 6.32 1.586l2.895.68a173.06 173.06 0 0 0 3.234.723l2.832.617c.43.098.82.168.871.164.07-.008.105-.129.133-.477zm15.758.098l1.352-.309c1.156-.238 3.105-.672 3.656-.816a157.32 157.32 0 0 1 2.32-.555c1.797-.418 6.016-1.48 6.441-1.625.137-.047.922-.27 1.742-.496s2.266-.664 3.207-.977l1.82-.594c.313-.09-.047-.418-.637-.578-.488-.137-.895-.32-1.094-.496-.102-.094-.172-.102-.402-.062-.156.031-1.57.254-3.145.496l-2.863.445-.5.496c-.547.543-.621.582-2.363 1.332l-1.586.703c-.391.18-.934.344-2.023.605-3.035.727-3.082.734-4.109.738-1.125.008-1.637-.082-2.121-.371-.191-.117-.371-.211-.395-.211-.039 0-.254 1.297-.355 2.16-.035.281-.023.328.066.328.059 0 .504-.098.988-.215zm-16.824-1.66c.664-.102.98-.285 1.477-.848 1.18-1.34 1.582-2.48 1.492-4.25-.035-.687-.137-1.352-.219-1.441-.039-.039-2.656.398-3.59.602-.492.105-2.559.441-4.23.688l-2.117.313a63.05 63.05 0 0 1-2.281.289l-1.285.141-.195.434-.191.438.449.453c.43.43.492.473 1.355.832l1.434.629c.953.449 1.652.727 2.184.859l1.27.344c2.02.574 3.219.715 4.449.52zm9.418-2.871l1.828-1.168c2.012-1.262 2.332-1.488 2.297-1.602-.07-.234-.16-1.262-.16-1.871 0-.699.082-1.566.211-2.18l.082-.395-.191.035c-.18.035-.762.211-2.809.844-.445.137-.84.25-.871.25s-.16-.172-.277-.387a13.14 13.14 0 0 0-.375-.641c-.094-.137-.305-.504-.473-.82-.293-.555-.309-.57-.422-.469-.121.109-1.766 2.574-2.152 3.23-.371.625-.937 1.766-.937 1.887 0 .102 3.336 3.797 3.426 3.797.02 0 .391-.23.824-.512zM34.25 50.32c.586-.102 1.281-.301 1.555-.441.191-.098.359-.379.359-.605 0-.164-.242-.836-.445-1.242-.152-.305-.355-.539-.465-.539-.043 0-.367.188-.719.422-1.133.746-2.09 1.777-2.211 2.379l-.039.207.707-.043c.391-.02.957-.082 1.258-.137zm61.488.004c-.105-.664-1.469-2.027-2.691-2.695l-.312-.168-.133.152c-.223.27-.691 1.32-.719 1.625-.023.242-.004.309.117.43.445.426 2.047.813 3.293.789.43-.008.465-.02.445-.133zM33.75 48.125c.215-.164.543-.395.73-.512s.363-.238.383-.27.113-.437.207-.902.188-.918.219-1.004.109-.422.184-.746c.395-1.793.836-3.66.965-4.09a104.32 104.32 0 0 0 .535-1.855c.379-1.355.996-3.117 1.406-4.008a7.4 7.4 0 0 0 .211-.484c0-.012-.113-.02-.25-.02s-.41-.059-.605-.125-.371-.105-.398-.078c-.074.078-.953 2.352-1.316 3.406-.711 2.063-.855 2.52-1.125 3.609l-.473 1.898c-.109.43-.348 1.508-.531 2.398l-.465 2.191c-.125.563-.16.891-.094.891.016 0 .207-.137.418-.301zm61.004.25c.031-.051-.406-2.25-.637-3.187l-.469-2.059c-.437-1.961-.566-2.465-.992-3.793-.676-2.133-1.883-5.352-2-5.352a3.43 3.43 0 0 0-.344.125c-.172.066-.441.125-.594.125s-.277.008-.277.02.125.289.277.609c.25.527 1.066 2.84 1.395 3.945.414 1.398 1.191 4.547 1.563 6.348l.355 1.695.094.453.664.457c.367.254.703.504.75.563.09.109.168.129.215.051zM3.293 46.84c.039-.023.188-.203.332-.398.48-.652 2.508-2.578 3.191-3.035a4.9 4.9 0 0 0 .5-.395 5.37 5.37 0 0 1 .594-.453c.184-.121.539-.371.785-.555.738-.543 2.113-1.441 3.629-2.363 1.352-.824 4.496-2.273 6.535-3.012l1.152-.43c.242-.109 1.336-.461 2.426-.781l.996-.312c.352-.145.664-.234 2.895-.805l1.434-.379c.309-.094 1.66-.395 3.359-.75l1.898-.398a45.68 45.68 0 0 1 1.703-.312l1.293-.238c.02-.02-1.531-.48-2.684-.797-.242-.066-1.137-.277-1.992-.469l-1.93-.437c-1.703-.41-4.687-.953-6.941-1.266-.656-.09-2.441-.242-3.578-.309l-1.059-.059-.434.293c-.238.16-.469.32-.504.355s-.273.199-.52.371-.57.402-.711.516l-.82.629c-.309.234-.645.508-.746.609a4.35 4.35 0 0 1-.406.34c-.223.168-.781.672-1.918 1.727-.91.848-2.816 2.895-3.332 3.578a10.84 10.84 0 0 1-.508.629 6.95 6.95 0 0 0-.426.547 31.61 31.61 0 0 1-.586.809c-.238.316-.566.785-.73 1.043l-.625.965c-.531.82-1.301 2.242-1.77 3.27l-.527 1.125c-.141.246-.906 2.359-.902 2.48 0 .02.191-.262.43-.629s.457-.68.496-.703zm63.836-1.719a83.55 83.55 0 0 1 1.836-.559c.598-.176 1.105-.336 1.129-.359.031-.031.613-3.844.613-4.012 0-.031-.445.133-2.273.852l-.871.34-.395 1.004c-.219.551-.402.996-.414.984s.109-.496.266-1.078l.285-1.055-.305-.59c-.285-.543-.461-.754-.465-.555 0 .129-.59 1.32-.816 1.652a12.65 12.65 0 0 1-.535.703c-.184.223-.348.465-.367.539-.031.129.094.383.852 1.703.449.785.426.754.578.711l.883-.281zm6.879-3.383c-.141-.332-.551-.969-.621-.969-.086 0-.184.711-.105.758.039.023.238.152.445.289s.379.238.391.23-.039-.148-.109-.309zm-5.75-.914c.27-.105.602-.242.738-.305a33.68 33.68 0 0 1 1.047-.406c.91-.344.855-.277 1.016-1.199a3.23 3.23 0 0 0 .027-1.227c-.078-.637-.238-1.395-.316-1.48-.023-.027-.586.313-1.258.754l-1.215.805-.012.387a10.55 10.55 0 0 1-.348 1.957c-.187.566-.27.906-.223.906.027 0 .273-.086.543-.191zm-.77-.195a11.45 11.45 0 0 1 .191-.668c.188-.633.352-1.523.348-1.926 0-.309.063-.246-.816-.836l-.172-.113v.645c0 .371-.043.84-.102 1.109l-.137.633c-.02.117.059.328.273.73.168.309.32.563.344.566s.051-.062.07-.141zm-16.887-1.945c0-.016-.125-.203-.277-.41s-.281-.426-.281-.477c0-.062-.035-.078-.109-.047-.059.023-.262.102-.453.172s-.352.141-.367.152c-.09.066.641.441 1.086.551.371.094.402.098.402.059zm24.758-.668c-.187-.289-.699-.812-1.148-1.176l-.461-.379.039 1.156.809.289c.445.16.82.281.832.273s-.02-.086-.07-.164zm-23.512-.602c-.137-.199-.25-.395-.25-.434 0-.105-.098-.086-.437.086-.168.086-.309.191-.309.238 0 .109.387.285.84.383l.391.09c.008 0-.098-.16-.234-.363zm16.977-.34c.266-.18.664-.441.887-.574a13.02 13.02 0 0 0 .727-.484c.367-.266.367-.199.016-.941-.305-.656-1.148-1.48-1.863-1.828-.473-.227-.922-.379-.937-.312-.004.012-.098.652-.211 1.422l-.203 1.402.359.375c.383.402.504.598.613 1.012.039.145.086.262.102.262s.246-.148.512-.332zm-.855.207c0-.145-.203-.535-.41-.785-.254-.309-.672-.617-.93-.684l-.176-.043.133.191c.078.105.188.309.25.453.086.203.219.34.574.59.488.348.559.383.559.277zm-14.629-.734c0-.031-.055-.105-.121-.164a.65.65 0 0 1-.16-.23c-.047-.148-.133-.145-.395.016-.254.152-.215.207.242.332.414.109.434.113.434.047zm6.898-.754l1.121-.258.391-.074-.039-.391c-.023-.215-.102-.723-.176-1.125l-.137-.738-.191.012c-.172.016-2.078.559-2.129.609-.012.012.039.133.109.27.164.324.27 1.055.215 1.52-.027.258-.02.359.035.359.039 0 .398-.082.801-.184zm6.738-.207c.215-1.301.402-2.887.348-2.922-.121-.078-.934-.121-1.91-.102l-.961.02.004.188c0 .102.051.57.109 1.039s.102.961.102 1.094v.238l.668.078c.371.043.852.145 1.074.227.375.133.559.18.566.141zm-3.926-.324c.426-.051.91-.094 1.074-.094h.305l-.035-.266c-.02-.145-.062-.574-.098-.949s-.098-.832-.133-1.012l-.062-.324-.293.039c-.855.109-2.129.41-2.129.5 0 .043.055.359.125.703a16.49 16.49 0 0 1 .16 1.016c.051.508.07.555.199.516.059-.02.457-.078.887-.129zm-24.086-1.383c.609-.195 1.117-.871 1.117-1.477 0-.254-.309-.895-.535-1.105-.633-.594-1.77-.602-2.348-.016-.293.297-.426.621-.422 1.039.004.625.344 1.164.941 1.48.375.195.793.223 1.246.078zm51.285-.059c.66-.32 1.016-.836 1.02-1.48.004-.73-.383-1.227-1.133-1.445-1.293-.375-2.547.93-2.039 2.125a1.66 1.66 0 0 0 2.152.801zm-14.937-.426c-.117-.141-1.035-.715-1.422-.887-.516-.23-1.516-.586-1.551-.555-.016.02.043.125.129.238a4.5 4.5 0 0 1 .352.594c.105.215.211.391.238.395s.508.07 1.07.148c1.324.184 1.277.18 1.184.066zm-21.492-.199c.16-.078.262-.191.367-.422.195-.418.195-.492-.023-.445-.273.066-1.023.457-1.258.664-.18.152-.203.195-.125.246.152.098.801.07 1.039-.043zm-1.535-.062c.086-.055.238-.18.34-.273.316-.297.91-.617 1.434-.777.816-.25 1.328-.672 2.199-1.832 1.445-1.926 3.426-3.535 4.941-4.016 1.148-.363-.57-.133-2.316.309a19.42 19.42 0 0 0-2.957 1.012c-1.336.555-3.484 1.684-3.703 1.945a7.2 7.2 0 0 1-.312.25c-.773.582-1.391 1.383-1.512 1.965-.086.41.008.852.227 1.074.363.363 1.328.566 1.66.344zM38.836 30.52c.02-.016-.133-.352-.332-.742-.906-1.789-2.516-3.422-4.488-4.559l-.641-.371c-.148-.082-.285-.137-.305-.117s.195.254.48.523a45.3 45.3 0 0 1 2.004 2.039c.617.68 1.5 1.938 1.836 2.617l.336.691.539-.027c.297-.016.555-.039.57-.055zm51.578-.262c.168-.379.727-1.297 1.102-1.805.57-.773 1.238-1.52 2.375-2.648l1.184-1.176-.555.297c-1.141.617-2.375 1.492-2.992 2.125-.941.961-1.727 2.035-2.145 2.918-.312.664-.336.625.32.617l.57-.008zm-27.77-1.004a17.74 17.74 0 0 1 .977-.117c.492-.047.563-.07.82-.281.32-.266 1.32-.828 2.094-1.184a9.39 9.39 0 0 0 .648-.324c.109-.074.105-.082-.059-.125-.527-.129-2.824.328-3.723.746-.449.207-1.191.766-1.598 1.195l-.215.23.309-.039c.172-.02.508-.066.746-.102zm23.293 74.707c.113-.016.309-.016.438 0s.039.023-.203.023-.344-.012-.234-.023zm39.305-56.984c-.039-.078-.242-.574-.453-1.105-.465-1.172-1.426-3.051-2.223-4.355-.633-1.035-.699-1.16-.543-.965.406.496 1.633 2.676 2.32 4.121.391.828 1.027 2.387.992 2.426-.012.012-.055-.043-.094-.121zm-3.449-6.691c-.055-.059-.285-.387-.516-.727-.336-.5-1.695-2.273-1.836-2.398-.02-.02-.289-.328-.602-.687s-.629-.707-.703-.777-.254-.266-.391-.434c-.152-.195-.047-.109.289.234.992 1.012 2.512 2.883 3.484 4.293.398.57.492.742.273.496zm-74.16-3.469c.02-.07.066-.285.105-.48.125-.629.48-.973.84-.809.316.145.145.543-.312.734-.113.047-.285.191-.379.32-.246.336-.293.379-.254.234zm.762-.969c.07-.109-.004-.125-.129-.027-.105.078-.109.09-.016.09.059.004.121-.027.145-.062zm67.762-2.168c-.641-.613-1.383-1.293-1.648-1.504s-.469-.406-.453-.422c.055-.055 3.332 2.91 3.332 3.012 0 .078-.16-.062-1.23-1.086zM66.18 29.398a.5.5 0 0 1 .215 0c.063.016.012.027-.105.027s-.172-.012-.109-.027zm6.039-.5c.059-.016.145-.016.184.004s-.008.027-.109.027-.137-.016-.074-.031zm-57.328-.687a3.13 3.13 0 0 1 .469 0c.129.016.023.027-.234.027s-.363-.012-.234-.027zm1.246-.062a7.27 7.27 0 0 1 .715 0c.195.012.035.023-.359.023s-.555-.012-.355-.023zm0 0" fill="#000000"/></svg>
```